<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
  <meta charset="utf-8" /><meta name="generator" content="Docutils 0.18.1: http://docutils.sourceforge.net/" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>Use-case 2: data are provided by the user &mdash; ODAMNet 1 documentation</title>
      <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/custom.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/tabs.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/design-style.4045f2051d55cab465a707391d5b2007.min.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/custom.css" type="text/css" />
      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" type="text/css" />
  <!--[if lt IE 9]>
    <script src="../../_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
        <script data-url_root="../../" id="documentation_options" src="../../_static/documentation_options.js"></script>
        <script src="../../_static/jquery.js"></script>
        <script src="../../_static/underscore.js"></script>
        <script src="../../_static/_sphinx_javascript_frameworks_compat.js"></script>
        <script src="../../_static/doctools.js"></script>
        <script src="../../_static/sphinx_highlight.js"></script>
        <script src="../../_static/design-tabs.js"></script>
    <script src="../../_static/js/theme.js"></script>
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="Input files" href="../formats/Input.html" />
    <link rel="prev" title="Use-case 1: data are retrieved automatically by queries" href="usecase1.html" /> 
</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >

          
          
          <a href="../../index.html" class="icon icon-home">
            ODAMNet
          </a>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <p class="caption" role="heading"><span class="caption-text">Quickstart</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../Installation.html">Installation</a></li>
<li class="toctree-l1"><a class="reference internal" href="../Installation.html#usage">Usage</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Approaches</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_overview.html">Overview of the approaches</a></li>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_overlap.html">Overlap analysis</a></li>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_AMI.html">Active Module Identification</a></li>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_RWR.html">Random Walk with Restart</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Networks</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../network/NetworkDownloading.html">Networks downloading</a></li>
<li class="toctree-l1"><a class="reference internal" href="../network/NetworkCreation.html">Networks creation</a></li>
<li class="toctree-l1"><a class="reference internal" href="../network/NetworkUsed.html">Networks used</a></li>
<li class="toctree-l1"><a class="reference internal" href="../network/NetworkVisualisation.html">Networks visualisation</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Use-cases</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="usecase1.html">Use-case 1: data are retrieved automatically by queries</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Use-case 2: data are provided by the user</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#context">Context</a></li>
<li class="toctree-l2"><a class="reference internal" href="#overlap-analysis">Overlap analysis</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#running-overlap-analysis-with-data-provided-by-user">Running Overlap analysis with data provided by user</a></li>
<li class="toctree-l3"><a class="reference internal" href="#results-of-overlap-analysis-with-data-provided-by-user">Results of Overlap analysis with data provided by user</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#data-provided-by-user-overview">Data provided by user overview</a></li>
<li class="toctree-l4"><a class="reference internal" href="#overlap-analysis-results">Overlap analysis results</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#active-module-identification-ami">Active Module Identification (AMI)</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#running-ami-with-data-provided-by-user">Running AMI with data provided by user</a></li>
<li class="toctree-l3"><a class="reference internal" href="#results-of-ami-with-data-provided-by-user">Results of AMI with data provided by user</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#id18">Data provided by user overview</a></li>
<li class="toctree-l4"><a class="reference internal" href="#ami-results">AMI results</a></li>
<li class="toctree-l4"><a class="reference internal" href="#id24">Overlap analysis results</a></li>
<li class="toctree-l4"><a class="reference internal" href="#visualisation-of-ami-results">Visualisation of AMI results</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#random-walk-with-restart-rwr">Random Walk with Restart (RWR)</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#running-rwr-with-data-provided-by-user">Running RWR with data provided by user</a></li>
<li class="toctree-l3"><a class="reference internal" href="#results-of-rwr-with-data-provided-by-user">Results of RWR with data provided by user</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#id28">Data provided by user overview</a></li>
<li class="toctree-l4"><a class="reference internal" href="#rwr-results">RWR results</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#overlap-ami-and-rwr-results-comparison">Overlap, AMI and RWR results comparison</a></li>
<li class="toctree-l2"><a class="reference internal" href="#references">References</a></li>
</ul>
</li>
</ul>
<p class="caption" role="heading"><span class="caption-text">File formats</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../formats/Input.html">Input files</a></li>
<li class="toctree-l1"><a class="reference internal" href="../formats/Output.html">Output files</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Code description</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../functions/fcts_arguments.html">Input parameters for each approach</a></li>
<li class="toctree-l1"><a class="reference internal" href="../functions/fcts_description.html">Code functions in details</a></li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../index.html">ODAMNet</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="../../index.html" class="icon icon-home" aria-label="Home"></a></li>
      <li class="breadcrumb-item active">Use-case 2: data are provided by the user</li>
      <li class="wy-breadcrumbs-aside">
            <a href="../../_sources/pages/analysis/usecase2.rst.txt" rel="nofollow"> View page source</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section id="use-case-2-data-are-provided-by-the-user">
<h1>Use-case 2: data are provided by the user<a class="headerlink" href="#use-case-2-data-are-provided-by-the-user" title="Permalink to this heading"></a></h1>
<section id="context">
<h2>Context<a class="headerlink" href="#context" title="Permalink to this heading"></a></h2>
<p>In Ozisik <em>et al.</em>, 2021 <a class="footnote-reference brackets" href="#id33" id="id1" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a> paper, an overlap analysis is performed between vitamin A and D target genes and processes
related to Congenital Anomalies of the Kidney and Urinary Tract (CAKUT).</p>
<p>In this use-case, we illustrate how to use ODAMNet with data provided by user. We use data from Ozisik <em>et al.</em>,
2021 <a class="footnote-reference brackets" href="#id33" id="id2" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a> paper. Results are compared with those found in the Ozisik <em>et al.</em>, 2021 <a class="footnote-reference brackets" href="#id33" id="id3" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a> paper.</p>
<p><strong>Vitamin A target genes</strong> are coming from Balmer and Blomhoff <a class="footnote-reference brackets" href="#id34" id="id4" role="doc-noteref"><span class="fn-bracket">[</span>2<span class="fn-bracket">]</span></a> paper and <strong>pathways/processes</strong> related to CAKUT
are coming from <a class="reference external" href="https://www.wikipathways.org/">WikiPathways</a> <a class="footnote-reference brackets" href="#id35" id="id5" role="doc-noteref"><span class="fn-bracket">[</span>3<span class="fn-bracket">]</span></a>, <a class="reference external" href="https://reactome.org/">Reactome</a> <a class="footnote-reference brackets" href="#id36" id="id6" role="doc-noteref"><span class="fn-bracket">[</span>4<span class="fn-bracket">]</span></a> and <a class="reference external" href="http://geneontology.org/">Gene Ontology</a> (GO) <a class="footnote-reference brackets" href="#id37" id="id7" role="doc-noteref"><span class="fn-bracket">[</span>5<span class="fn-bracket">]</span></a><sup>,</sup> <a class="footnote-reference brackets" href="#id38" id="id8" role="doc-noteref"><span class="fn-bracket">[</span>6<span class="fn-bracket">]</span></a>. Biological networks used are presented in
the <a class="reference internal" href="../network/NetworkUsed.html"><span class="doc">Networks used</span></a> page and are coming from the <a class="reference external" href="https://www.ndexbio.org/">Network Data Exchange</a> (NDEx) <a class="footnote-reference brackets" href="#id39" id="id9" role="doc-noteref"><span class="fn-bracket">[</span>7<span class="fn-bracket">]</span></a>. Input data are available in <a class="reference external" href="https://github.com/MOohTus/ODAMNet/tree/main/useCases/InputData">GitHub</a>.</p>
</section>
<section id="overlap-analysis">
<span id="usecase2-overlap"></span><h2>Overlap analysis<a class="headerlink" href="#overlap-analysis" title="Permalink to this heading"></a></h2>
<p>The Overlap analysis searches intersecting genes between vitamin A target genes and genes involved in pathways/processes
related to CAKUT pathways (see <a class="reference internal" href="../approaches/methods_overlap.html"><span class="doc">Overlap analysis</span></a> page for more details).</p>
<section id="running-overlap-analysis-with-data-provided-by-user">
<h3>Running Overlap analysis with data provided by user<a class="headerlink" href="#running-overlap-analysis-with-data-provided-by-user" title="Permalink to this heading"></a></h3>
<p>Target genes file is given using the <code class="docutils literal notranslate"><span class="pre">--targetGenesFile</span></code> parameter. It contains the list of vitamin A target genes
[<a class="reference internal" href="../formats/Input.html#targetgenesfile"><span class="std std-ref">FORMAT</span></a>].</p>
<p>Pathways/processes related to CAKUT are given using the <code class="docutils literal notranslate"><span class="pre">--GMT</span></code> parameter [<a class="reference internal" href="../formats/Input.html#pathways"><span class="std std-ref">FORMAT</span></a>]. Background genes
of each source of pathways/processes is required (<code class="docutils literal notranslate"><span class="pre">--backgroundFile</span></code> parameter). The file contains the list of
background file names.</p>
<p>Results files are saved into <code class="docutils literal notranslate"><span class="pre">useCases/OutputResults_useCase2/</span></code> folder.</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>odamnet<span class="w"> </span>overlap<span class="w"> </span>--targetGenesFile<span class="w"> </span>useCases/InputData/VitA-Balmer2002-Genes.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--GMT<span class="w"> </span>useCases/InputData/PathwaysOfInterest.gmt<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--backgroundFile<span class="w"> </span>useCases/InputData/PathwaysOfInterestBackground.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--outputPath<span class="w"> </span>useCases/OutputResults_useCase2
</pre></div>
</div>
<p><code class="docutils literal notranslate"><span class="pre">Overlap_genesList_withpathOfInterest.csv</span></code> file is created. It contains results of the overlap analysis between
vitamin A target genes and CAKUT related pathways/processes.</p>
<p class="italic">For more details about this output file, see <a class="reference internal" href="../formats/Output.html#overlapoutput"><span class="std std-ref">Overlap analysis output files</span></a> section.</p>
</section>
<section id="results-of-overlap-analysis-with-data-provided-by-user">
<h3>Results of Overlap analysis with data provided by user<a class="headerlink" href="#results-of-overlap-analysis-with-data-provided-by-user" title="Permalink to this heading"></a></h3>
<section id="data-provided-by-user-overview">
<h4>Data provided by user overview<a class="headerlink" href="#data-provided-by-user-overview" title="Permalink to this heading"></a></h4>
<p>Target genes are coming from Balmer and Blomhoff <a class="footnote-reference brackets" href="#id34" id="id10" role="doc-noteref"><span class="fn-bracket">[</span>2<span class="fn-bracket">]</span></a>. Pathways of interests are coming from Reactome <a class="footnote-reference brackets" href="#id36" id="id11" role="doc-noteref"><span class="fn-bracket">[</span>4<span class="fn-bracket">]</span></a> and
WikiPathways <a class="footnote-reference brackets" href="#id35" id="id12" role="doc-noteref"><span class="fn-bracket">[</span>3<span class="fn-bracket">]</span></a> and processes of interest are coming from Biological Process (GO) <a class="footnote-reference brackets" href="#id37" id="id13" role="doc-noteref"><span class="fn-bracket">[</span>5<span class="fn-bracket">]</span></a><sup>,</sup> <a class="footnote-reference brackets" href="#id38" id="id14" role="doc-noteref"><span class="fn-bracket">[</span>6<span class="fn-bracket">]</span></a>.
Data are presented in the <a class="reference internal" href="#usecase2-overlapdataoverview"><span class="std std-ref">Table 13</span></a>.</p>
<span id="usecase2-overlapdataoverview"></span><table class="docutils align-center" id="id44">
<caption><span class="caption-number">Table 13 </span><span class="caption-text">- Overview of data provided by user</span><a class="headerlink" href="#id44" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"></th>
<th class="head"><p>Number</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Vitamin A target genes</p></td>
<td><p>521</p></td>
</tr>
<tr class="row-odd"><td><p>Pathways/processes related to CAKUT</p></td>
<td><p>13</p></td>
</tr>
<tr class="row-even"><td><p>Pathways/processes sources</p></td>
<td><p>3</p></td>
</tr>
</tbody>
</table>
</section>
<section id="overlap-analysis-results">
<h4>Overlap analysis results<a class="headerlink" href="#overlap-analysis-results" title="Permalink to this heading"></a></h4>
<p>We performed an Overlap analysis between vitamin A target genes (521) and pathways/processes related to CAKUT (13). We
obtained significant overlap between target genes and <strong>7 pathways/processes</strong> related to CAKUT (pAdjusted &lt;= 0.05).
Results are presented in the <a class="reference internal" href="#usecase2-overlapresults"><span class="std std-ref">Table 14</span></a>.</p>
<p>Top 5 of the significant overlaps between the vitamin A target genes and rare disease pathways</p>
<span id="usecase2-overlapresults"></span><table class="docutils align-center" id="id45">
<caption><span class="caption-number">Table 14 </span><span class="caption-text">- Significant overlap results between vitamin A target genes and pathways/processes related tp CAKUT</span><a class="headerlink" href="#id45" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"><p>PathwayIDs</p></th>
<th class="head"><p>PathwayNames</p></th>
<th class="head"><p>pAdjusted</p></th>
<th class="head"><p>IntersectionSize</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p><a class="reference external" href="GO:0072001">GO:0072001</a></p></td>
<td><p>renal system development</p></td>
<td><p>8.64e-17</p></td>
<td><p>43</p></td>
</tr>
<tr class="row-odd"><td><p><a class="reference external" href="GO:0001822">GO:0001822</a></p></td>
<td><p>kidney development</p></td>
<td><p>5.14e-16</p></td>
<td><p>41</p></td>
</tr>
<tr class="row-even"><td><p><a class="reference external" href="GO:0060993">GO:0060993</a></p></td>
<td><p>kidney morphogenesis</p></td>
<td><p>8.33e-12</p></td>
<td><p>20</p></td>
</tr>
<tr class="row-odd"><td><p>WP:WP5053</p></td>
<td><p>Development of ureteric collection …</p></td>
<td><p>7.37e-08</p></td>
<td><p>15</p></td>
</tr>
<tr class="row-even"><td><p>WP:WP4823</p></td>
<td><p>Genes controlling nephrogenesis</p></td>
<td><p>2.88e-04</p></td>
<td><p>10</p></td>
</tr>
<tr class="row-odd"><td><p>PMC5748921-PMC6115658</p></td>
<td><p>CAKUT causal genes</p></td>
<td><p>2.30e-03</p></td>
<td><p>6</p></td>
</tr>
<tr class="row-even"><td><p>WP:WP4830</p></td>
<td><p>GDNF/RET signalling axis</p></td>
<td><p>1.54e-02</p></td>
<td><p>5</p></td>
</tr>
</tbody>
</table>
<p>Ozisik <em>et al.,</em> <a class="footnote-reference brackets" href="#id33" id="id15" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a> identified 7 pathways/processes related to CAKUT disease over 13. ODAMNet found the same 7
pathways/processes.</p>
</section>
</section>
</section>
<section id="active-module-identification-ami">
<span id="usecase2-ami"></span><h2>Active Module Identification (AMI)<a class="headerlink" href="#active-module-identification-ami" title="Permalink to this heading"></a></h2>
<p>The Active Module Identification (AMI) approach identifies active module that contains high number of vitamin A target
genes using a protein-protein interaction (PPI) network. AMI is performed using DOMINO <a class="footnote-reference brackets" href="#id40" id="id16" role="doc-noteref"><span class="fn-bracket">[</span>8<span class="fn-bracket">]</span></a>. Then, an Overlap analysis
is applied between identified active modules and CAKUT pathways/processes. See <a class="reference internal" href="../approaches/methods_AMI.html"><span class="doc">Active Module Identification</span></a> page for
more details.</p>
<section id="running-ami-with-data-provided-by-user">
<h3>Running AMI with data provided by user<a class="headerlink" href="#running-ami-with-data-provided-by-user" title="Permalink to this heading"></a></h3>
<div class="admonition warning">
<p class="admonition-title">Warning</p>
<p><svg version="1.1" width="2.0em" height="2.0em" class="sd-octicon sd-octicon-alert" viewBox="0 0 24 24" aria-hidden="true"><path d="M13 17.5a1 1 0 11-2 0 1 1 0 012 0zm-.25-8.25a.75.75 0 00-1.5 0v4.5a.75.75 0 001.5 0v-4.5z"></path><path fill-rule="evenodd" d="M9.836 3.244c.963-1.665 3.365-1.665 4.328 0l8.967 15.504c.963 1.667-.24 3.752-2.165 3.752H3.034c-1.926 0-3.128-2.085-2.165-3.752L9.836 3.244zm3.03.751a1 1 0 00-1.732 0L2.168 19.499A1 1 0 003.034 21h17.932a1 1 0 00.866-1.5L12.866 3.994z"></path></svg> When using DOMINO server, <strong>results cannot be identically reproduced</strong>. Indeed, DOMINO server doesn’t allow to set the random seed. This random seed changes every new analysis.</p>
</div>
<p>Target genes file is given using the <code class="docutils literal notranslate"><span class="pre">--targetGenesFile</span></code> parameter. It contains the list of vitamin A target genes
[<a class="reference internal" href="../formats/Input.html#targetgenesfile"><span class="std std-ref">FORMAT</span></a>].</p>
<p>Pathways/processes related to CAKUT are given using the <code class="docutils literal notranslate"><span class="pre">--GMT</span></code> parameter [<a class="reference internal" href="../formats/Input.html#pathways"><span class="std std-ref">FORMAT</span></a>]. Background genes
of each source of pathways/processes is required (<code class="docutils literal notranslate"><span class="pre">--backgroundFile</span></code> parameter). The file contains the list of
background file names.</p>
<p>We used a PPI network [<a class="reference internal" href="../formats/Input.html#sif"><span class="std std-ref">FORMAT</span></a>] previously downloaded from NDEx <a class="footnote-reference brackets" href="#id39" id="id17" role="doc-noteref"><span class="fn-bracket">[</span>7<span class="fn-bracket">]</span></a>. The PPI network file is provided
using <code class="docutils literal notranslate"><span class="pre">--networkFile</span></code> parameter. See <a class="reference internal" href="../network/NetworkDownloading.html"><span class="doc">Networks downloading</span></a> section. Network name should have
<strong>.sif</strong> extension.</p>
<p>Results files are saved into <code class="docutils literal notranslate"><span class="pre">useCases/OutputResults_useCase2/</span></code> folder.</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>odamnet<span class="w"> </span>domino<span class="w">  </span>--targetGenesFile<span class="w"> </span>useCases/InputData/VitA-Balmer2002-Genes.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--GMT<span class="w"> </span>useCases/InputData/PathwaysOfInterest.gmt<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--backgroundFile<span class="w"> </span>useCases/InputData/PathwaysOfInterestBackground.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--networkFile<span class="w"> </span>useCases/InputData/PPI_HiUnion_LitBM_APID_gene_names_190123.sif<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--outputPath<span class="w"> </span>useCases/OutputResults_useCase2
</pre></div>
</div>
<p>Several files are generated:</p>
<ul class="simple">
<li><p><code class="docutils literal notranslate"><span class="pre">DOMINO_inputGeneList_genesList.txt</span></code>: vitamin A target genes list used for the active module identification.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">Overlap_AM_*_genesList_withpathOfInterest.csv</span></code>: results of the Overlap analysis between identified active modules
genes and pathways/processes of related to CAKUT. There is one file per active module.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_activeModulesNetwork.txt</span></code>, <code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_overlapAMresults4Cytoscape.txt</span></code>,
<code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_activeModules.txt</span></code>, <code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_activeModulesMetrics.txt</span></code> and
<code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_signOverlap.txt</span></code>: some statistics are calculated and saved into files. Theses files are useful
for visualisation.</p></li>
</ul>
<p class="italic">For more details about these files, see <a class="reference internal" href="../formats/Output.html"><span class="doc">Output files</span></a> page (<a class="reference internal" href="../formats/Output.html#overlapoutput"><span class="std std-ref">Overlap analysis output files</span></a> and <a class="reference internal" href="../formats/Output.html#amioutput"><span class="std std-ref">AMI output files</span></a>)</p>
</section>
<section id="results-of-ami-with-data-provided-by-user">
<h3>Results of AMI with data provided by user<a class="headerlink" href="#results-of-ami-with-data-provided-by-user" title="Permalink to this heading"></a></h3>
<section id="id18">
<h4>Data provided by user overview<a class="headerlink" href="#id18" title="Permalink to this heading"></a></h4>
<p>Target genes are coming from Balmer and Blomhoff <a class="footnote-reference brackets" href="#id34" id="id19" role="doc-noteref"><span class="fn-bracket">[</span>2<span class="fn-bracket">]</span></a>. Pathways of interests are coming from Reactome <a class="footnote-reference brackets" href="#id36" id="id20" role="doc-noteref"><span class="fn-bracket">[</span>4<span class="fn-bracket">]</span></a> and
WikiPathways <a class="footnote-reference brackets" href="#id35" id="id21" role="doc-noteref"><span class="fn-bracket">[</span>3<span class="fn-bracket">]</span></a> and processes of interest are coming from Biological Process (GO) <a class="footnote-reference brackets" href="#id37" id="id22" role="doc-noteref"><span class="fn-bracket">[</span>5<span class="fn-bracket">]</span></a><sup>,</sup> <a class="footnote-reference brackets" href="#id38" id="id23" role="doc-noteref"><span class="fn-bracket">[</span>6<span class="fn-bracket">]</span></a>.
Data are presented in the <a class="reference internal" href="#usecase2-amidataoverview"><span class="std std-ref">Table 15</span></a>.</p>
<span id="usecase2-amidataoverview"></span><table class="docutils align-center" id="id46">
<caption><span class="caption-number">Table 15 </span><span class="caption-text">- Overview of data provided by user</span><a class="headerlink" href="#id46" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"></th>
<th class="head"><p>Number</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Vitamin A target genes</p></td>
<td><p>521</p></td>
</tr>
<tr class="row-odd"><td><p>Pathways/processes related to CAKUT</p></td>
<td><p>13</p></td>
</tr>
<tr class="row-even"><td><p>Pathways/processes sources</p></td>
<td><p>3</p></td>
</tr>
</tbody>
</table>
<p>The PPI network is downloaded from <a class="reference external" href="https://www.ndexbio.org/viewer/networks/bfac0486-cefe-11ed-a79c-005056ae23aa">NDEx</a> (see <a class="reference internal" href="../network/NetworkUsed.html#ppinet"><span class="std std-ref">Protein-Protein Interaction (PPI) network</span></a>). It was build from 3 datasets: Lit-BM, Hi-Union and
APID. It contains 15,390 nodes and 131,087 edges.</p>
</section>
<section id="ami-results">
<h4>AMI results<a class="headerlink" href="#ami-results" title="Permalink to this heading"></a></h4>
<p>DOMINO defines vitamin A target genes as active genes and searches active modules enriched in active genes. Over the
521 target genes, 468 are found in the PPI and used as active genes by DOMINO. DOMINO identified
<strong>21 active modules</strong> enriched in vitamin A target genes (<a class="reference internal" href="#usecase2-amiresults"><span class="std std-ref">Table 16</span></a>).</p>
<span id="usecase2-amiresults"></span><table class="docutils align-center" id="id47">
<caption><span class="caption-number">Table 16 </span><span class="caption-text">- Composition of the active modules identified enriched in vitamin A target genes by DOMINO</span><a class="headerlink" href="#id47" title="Permalink to this table"></a></caption>
<colgroup>
<col style="width: 55%" />
<col style="width: 23%" />
<col style="width: 23%" />
</colgroup>
<thead>
<tr class="row-odd"><th class="head"></th>
<th class="head"><p>Min number</p></th>
<th class="head"><p>Max number</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Edges</p></td>
<td><p>3</p></td>
<td><p>223</p></td>
</tr>
<tr class="row-odd"><td><p>Nodes</p></td>
<td><p>4</p></td>
<td><p>99</p></td>
</tr>
<tr class="row-even"><td><p>Target genes</p></td>
<td><p>3</p></td>
<td><p>19</p></td>
</tr>
</tbody>
</table>
<p class="italic">See <code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_activeModulesNetworkMetrics.txt</span></code> file for more details.</p>
</section>
<section id="id24">
<h4>Overlap analysis results<a class="headerlink" href="#id24" title="Permalink to this heading"></a></h4>
<p>Then, we perform an Overlap analysis between identified active modules (21) and pathways/processes related to CAKUT
(13). We obtained significant overlap between <strong>6 identified active modules</strong> and <strong>6 pathways/processes</strong>
(pAdjusted &lt;= 0.05). Results are presented in <a class="reference internal" href="#usecase2-amioverlap"><span class="std std-ref">Table 17</span></a></p>
<p>Duplicates between active modules results are removed and we keep the more significant ones.</p>
<span id="usecase2-amioverlap"></span><table class="docutils align-center" id="id48">
<caption><span class="caption-number">Table 17 </span><span class="caption-text">- Significant overlaps between identified active modules and pathways/processes related to CAKUT</span><a class="headerlink" href="#id48" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"><p>Pathway IDs</p></th>
<th class="head"><p>Pathway Names</p></th>
<th class="head"><p>pAdjusted</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p><a class="reference external" href="GO:0072001">GO:0072001</a></p></td>
<td><p>renal system development</p></td>
<td><p>2.66e-03</p></td>
</tr>
<tr class="row-odd"><td><p><a class="reference external" href="GO:0001822">GO:0001822</a></p></td>
<td><p>kidney development</p></td>
<td><p>2.66e-03</p></td>
</tr>
<tr class="row-even"><td><p><a class="reference external" href="GO:0060993">GO:0060993</a></p></td>
<td><p>kidney morphogenesis</p></td>
<td><p>2.66e-03</p></td>
</tr>
<tr class="row-odd"><td><p><strong>REAC:R-HSA-8853659</strong></p></td>
<td><p><strong>RET signaling</strong></p></td>
<td><p><strong>5.19e-03</strong></p></td>
</tr>
<tr class="row-even"><td><p>WP:WP5053</p></td>
<td><p>Development of ureteric collection system</p></td>
<td><p>3.21e-02</p></td>
</tr>
<tr class="row-odd"><td><p><strong>REAC:R-HSA-195721</strong></p></td>
<td><p><strong>Signaling by WNT</strong></p></td>
<td><p><strong>3.47e-02</strong></p></td>
</tr>
</tbody>
</table>
<p>The <em>RET signaling</em> and <em>Signaling by WNT</em> reactome pathways were not identified with the overlap approach in the
previous study <a class="footnote-reference brackets" href="#id33" id="id25" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a> neither with the ODAMNet Overlap analysis.</p>
<p class="italic">See <code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_signOverlap.txt</span></code> file for more details.</p>
</section>
<section id="visualisation-of-ami-results">
<h4>Visualisation of AMI results<a class="headerlink" href="#visualisation-of-ami-results" title="Permalink to this heading"></a></h4>
<p>We created a visualisation of AMI results (<a class="reference internal" href="usecase1.html#usecase1-amifig"><span class="std std-numref">Fig. 19</span></a>) using Cytoscape <a class="footnote-reference brackets" href="#id41" id="id26" role="doc-noteref"><span class="fn-bracket">[</span>9<span class="fn-bracket">]</span></a>.</p>
<p>We found a significant overlap between <strong>6 active modules</strong> and <strong>6 pathways/processes</strong> related to CAKUT. For sake of
visualisation, we selected only three of them (<a class="reference internal" href="#usecase2-amifig"><span class="std std-numref">Fig. 23</span></a>). You can find the entire visualisation in the
cytoscape project called <code class="docutils literal notranslate"><span class="pre">AMI_visualisation.cys</span></code> in <a class="reference external" href="https://github.com/MOohTus/ODAMNet/tree/main/useCases/">GitHub</a>.</p>
<figure class="align-center" id="id49">
<span id="usecase2-amifig"></span><a class="reference internal image-reference" href="../../_images/UseCase2_AMI.png"><img alt="useCase2_AMIFig" src="../../_images/UseCase2_AMI.png" style="width: 674.0px; height: 389.5px;" /></a>
<figcaption>
<p><span class="caption-number">Fig. 23 </span><span class="caption-text">: Visualisation of 3 active modules and their associated pathways/processes related to CAKUT</span><a class="headerlink" href="#id49" title="Permalink to this image"></a></p>
<div class="legend">
<p>Genes are represented by nodes. Grey nodes are target genes, white nodes are non-target genes. Overlap results
between active modules and rare disease pathways are displayed using donuts color around nodes. Each color
corresponds to a rare disease pathways. Creation steps are explained in the <a class="reference internal" href="../network/NetworkVisualisation.html#cytoscape-ami"><span class="std std-ref">Active module identification results visualisation</span></a> section.</p>
</div>
</figcaption>
</figure>
<p>Module topology is different between modules and associated rare diseases pathways also vary
(<a class="reference internal" href="#usecase2-amifig"><span class="std std-numref">Fig. 23</span></a>).</p>
<p>The first active module (left in the <a class="reference internal" href="#usecase2-amifig"><span class="std std-numref">Fig. 23</span></a>) is very connected and contains genes involved in
several pathways and processes related to CAKUT. Association between vitamin A and <em>RET signaling</em> reactome pathway is
indirect. Indeed, genes involved in this pathway are not genes targeted by vitamin A. That explains why we didn’t found
this pathways with the Overlap analysis (see <a class="reference internal" href="#usecase2-overlap"><span class="std std-ref">Use-case 2 overlap results</span></a>).</p>
<p>The second active module (middle in the <a class="reference internal" href="#usecase2-amifig"><span class="std std-numref">Fig. 23</span></a>) is also highly connected but its contains genes
involved only in one pathways/processes related to CAKUT (<em>Development of ureteric collection system</em>).</p>
<p>The third active module (right in the <a class="reference internal" href="#usecase2-amifig"><span class="std std-numref">Fig. 23</span></a>) is sparser. It contains target genes and non-target
genes involved in <em>kidney morphogenesis</em> and <em>Signaling by WNT</em>.</p>
</section>
</section>
</section>
<section id="random-walk-with-restart-rwr">
<span id="usecase2-rwr"></span><h2>Random Walk with Restart (RWR)<a class="headerlink" href="#random-walk-with-restart-rwr" title="Permalink to this heading"></a></h2>
<p>The Random Walk with Restart (RWR) approach mesures proximities between vitamin A target genes and pathways/processes
related to CAKUT. To calculate these proximities (RWR scores), we used multiXrank <a class="footnote-reference brackets" href="#id42" id="id27" role="doc-noteref"><span class="fn-bracket">[</span>10<span class="fn-bracket">]</span></a> and multilayer networks. See
<a class="reference internal" href="../approaches/methods_RWR.html"><span class="doc">Random Walk with Restart</span></a> page for more details.</p>
<p>The multilayer network is composed of three gene networks and one pathways/processes related to CAKUT network. Genes
nodes are connected to pathways/processes nodes if they are involved in. See <a class="reference internal" href="../network/NetworkUsed.html"><span class="doc">Networks used</span></a> page for
more details.</p>
<section id="running-rwr-with-data-provided-by-user">
<h3>Running RWR with data provided by user<a class="headerlink" href="#running-rwr-with-data-provided-by-user" title="Permalink to this heading"></a></h3>
<p>Target genes file is given using <code class="docutils literal notranslate"><span class="pre">--targetGenesFile</span></code> parameter. IT contains list of vitamin A target genes
[<a class="reference internal" href="../formats/Input.html#targetgenesfile"><span class="std std-ref">FORMAT</span></a>].</p>
<p>MultiXrank needs as input a configuration file (<code class="docutils literal notranslate"><span class="pre">--configPath</span></code>) that contains path of networks and analysis
parameters. We used multiXrank with default parameters.</p>
<p>We provided a name file to save vitamin A target genes (i.e. seeds) <code class="docutils literal notranslate"><span class="pre">--sifFileName</span> <span class="pre">useCases/InputData/seeds.txt</span></code> and
also a SIF file name (<code class="docutils literal notranslate"><span class="pre">--sifFileName</span></code>) to save the top nodes based on RWR score (<code class="docutils literal notranslate"><span class="pre">--top</span> <span class="pre">20</span></code>).</p>
<p>Results files are saved into <code class="docutils literal notranslate"><span class="pre">useCases/OutputResults_useCase2/</span></code> folder.</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>odamnet<span class="w"> </span>multixrank<span class="w">      </span>--targetGenesFile<span class="w"> </span>useCases/InputData/VitA-Balmer2002-Genes.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                        </span>--configPath<span class="w"> </span>useCases/InputData/config_minimal_useCase2.yml<span class="w"> </span><span class="se">\</span>
<span class="w">                        </span>--networksPath<span class="w"> </span>useCases/InputData/<span class="w"> </span><span class="se">\</span>
<span class="w">                        </span>--seedsFile<span class="w"> </span>useCases/InputData/seeds.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                        </span>--sifFileName<span class="w"> </span>UseCase2_RWR_network.sif<span class="w"> </span><span class="se">\</span>
<span class="w">                        </span>--top<span class="w"> </span><span class="m">5</span><span class="w"> </span><span class="se">\</span>
<span class="w">                        </span>--outputPath<span class="w"> </span>useCases/OutputResults_useCase2
</pre></div>
</div>
<div class="admonition tip">
<p class="admonition-title">Tip</p>
<div class="line-block">
<div class="line">- Downloading of multiplex network from NDEx: <a class="reference internal" href="../network/NetworkUsed.html"><span class="doc">Networks used</span></a> + <a class="reference internal" href="../network/NetworkDownloading.html"><span class="doc">Networks downloading</span></a></div>
<div class="line">- Creation of the pathways/processes network: <a class="reference internal" href="../network/NetworkUsed.html"><span class="doc">Networks used</span></a> +  <a class="reference internal" href="../network/NetworkCreation.html"><span class="doc">Networks creation</span></a></div>
<div class="line">- Configuration file explanation and example: <a class="reference internal" href="../formats/Input.html#configfile"><span class="std std-ref">Configuration file</span></a> section</div>
</div>
</div>
<p>Several files are generated into <code class="docutils literal notranslate"><span class="pre">RWR_genesList/</span></code> folder:</p>
<ul class="simple">
<li><p><code class="docutils literal notranslate"><span class="pre">config_minimal_useCase2.yml</span></code> and <code class="docutils literal notranslate"><span class="pre">seeds.txt</span></code>: copies of the input files</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">multiplex_1.tsv</span></code> and <code class="docutils literal notranslate"><span class="pre">multiplex_2.tsv</span></code>: RWR scores for each multilayer. 1 is the genes multilayer network RWR
scores and 2 is the pathways/processes related to CAKUT network RWR scores.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">UseCase2_RWR_network.sif</span></code>: SIF file name that contains the network result</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">RWR_topX.txt</span></code>: Top X of pathways/processes related to CAKUT</p></li>
</ul>
<p class="italic">For more details about these file, see <a class="reference internal" href="../formats/Output.html"><span class="doc">Output files</span></a> page.</p>
</section>
<section id="results-of-rwr-with-data-provided-by-user">
<h3>Results of RWR with data provided by user<a class="headerlink" href="#results-of-rwr-with-data-provided-by-user" title="Permalink to this heading"></a></h3>
<section id="id28">
<h4>Data provided by user overview<a class="headerlink" href="#id28" title="Permalink to this heading"></a></h4>
<p>Target genes are coming from Balmer and Blomhoff <a class="footnote-reference brackets" href="#id34" id="id29" role="doc-noteref"><span class="fn-bracket">[</span>2<span class="fn-bracket">]</span></a>. There are <strong>521 vitamin A target genes</strong>.</p>
</section>
<section id="rwr-results">
<h4>RWR results<a class="headerlink" href="#rwr-results" title="Permalink to this heading"></a></h4>
<section id="analysis-with-pathways-processes-related-to-cakut-network">
<h5>Analysis with pathways/processes related to CAKUT network<a class="headerlink" href="#analysis-with-pathways-processes-related-to-cakut-network" title="Permalink to this heading"></a></h5>
<p>We used a multilayer network composed of three gene networks and one pathways/processes related to CAKUT network
(<a class="reference internal" href="../network/NetworkUsed.html#multilayercompo"><span class="std std-numref">Fig. 6</span></a> - left, <a class="reference internal" href="../network/NetworkUsed.html#genesmultilayernet"><span class="std std-ref">Genes multilayer network</span></a> and <a class="reference internal" href="../network/NetworkUsed.html#pathwaysofinterestnet"><span class="std std-ref">Rare disease pathways network</span></a>).</p>
<p>multiXrank defined vitamin A target genes as seeds. Over the 521 target genes, 480 are found in the multilayer and
used as seeds. Using the RWR scores (i.e. proximity score with the target genes), pathways/processes related to CAKUT
are prioritized. We selected the top 5 (<a class="reference internal" href="#usecase2-rwrpathofint"><span class="std std-ref">Table 18</span></a>).</p>
<span id="usecase2-rwrpathofint"></span><table class="docutils align-center" id="id50">
<caption><span class="caption-number">Table 18 </span><span class="caption-text">- Pathways/processes related to CAKUT prioritization using RWR score. The top 5 is displayed.</span><a class="headerlink" href="#id50" title="Permalink to this table"></a></caption>
<colgroup>
<col style="width: 25%" />
<col style="width: 50%" />
<col style="width: 25%" />
</colgroup>
<thead>
<tr class="row-odd"><th class="head"><p>Nodes (pathway IDs)</p></th>
<th class="head"><p>Pathway Names</p></th>
<th class="head"><p>RWR scores</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p><a class="reference external" href="GO:0072001">GO:0072001</a></p></td>
<td><p>renal system development</p></td>
<td><p>2.12e-03</p></td>
</tr>
<tr class="row-odd"><td><p><a class="reference external" href="GO:0001822">GO:0001822</a></p></td>
<td><p>kidney development</p></td>
<td><p>1.86e-03</p></td>
</tr>
<tr class="row-even"><td><p><strong>REAC:R-HSA-195721</strong></p></td>
<td><p><strong>Signaling by WNT</strong></p></td>
<td><p><strong>1.69e-03</strong></p></td>
</tr>
<tr class="row-odd"><td><p><strong>REAC:R-HSA-157118</strong></p></td>
<td><p><strong>Signaling by NOTCH</strong></p></td>
<td><p><strong>1.19e-03</strong></p></td>
</tr>
<tr class="row-even"><td><p>WP:WP5053</p></td>
<td><p>Development of ureteric collection system</p></td>
<td><p>6.46e-04</p></td>
</tr>
</tbody>
</table>
<p>We created a visualisation of the results (<a class="reference internal" href="#usecase2-rwrpathofintfig"><span class="std std-numref">Fig. 24</span></a>) using Cytoscape <a class="footnote-reference brackets" href="#id41" id="id30" role="doc-noteref"><span class="fn-bracket">[</span>9<span class="fn-bracket">]</span></a>. You can retrieved
it in the cytoscape project called <code class="docutils literal notranslate"><span class="pre">RWR_visualisation.cys</span></code> in <a class="reference external" href="https://github.com/MOohTus/ODAMNet/tree/main/useCases/">GitHub</a>. The <a class="reference internal" href="#usecase2-rwrpathofintfig"><span class="std std-numref">Fig. 24</span></a>
presents the top 5 of pathways/processes related to CAKUT, ordered by RWR score.</p>
<figure class="align-center" id="id51">
<span id="usecase2-rwrpathofintfig"></span><a class="reference internal image-reference" href="../../_images/UseCase2_RWR_top5.png"><img alt="useCase2_RWRpathOfIntFig" src="../../_images/UseCase2_RWR_top5.png" style="width: 721.0px; height: 545.3px;" /></a>
<figcaption>
<p><span class="caption-number">Fig. 24 </span><span class="caption-text">: Top 5 of the pathways/processes related to CAKUT prioritized using RWR score using a (disconnected)
pathways/processes related to CAKUT network</span><a class="headerlink" href="#id51" title="Permalink to this image"></a></p>
<div class="legend">
<p>Pathways/processes related to CAKUT are in pink triangles. Target genes are in grey and non-target genes are in
white. Creation steps are explained in the <a class="reference internal" href="../network/NetworkVisualisation.html#cytoscape-rwr"><span class="std std-ref">Random walk with restart results visualisation</span></a> section.</p>
</div>
</figcaption>
</figure>
</section>
<section id="extra-analysis-with-disease-disease-similarity-network">
<h5>Extra : analysis with disease-disease similarity network<a class="headerlink" href="#extra-analysis-with-disease-disease-similarity-network" title="Permalink to this heading"></a></h5>
<div class="admonition tip">
<p class="admonition-title">Tip</p>
<p><svg version="1.1" width="1.5em" height="1.5em" class="sd-octicon sd-octicon-alert" viewBox="0 0 24 24" aria-hidden="true"><path d="M13 17.5a1 1 0 11-2 0 1 1 0 012 0zm-.25-8.25a.75.75 0 00-1.5 0v4.5a.75.75 0 001.5 0v-4.5z"></path><path fill-rule="evenodd" d="M9.836 3.244c.963-1.665 3.365-1.665 4.328 0l8.967 15.504c.963 1.667-.24 3.752-2.165 3.752H3.034c-1.926 0-3.128-2.085-2.165-3.752L9.836 3.244zm3.03.751a1 1 0 00-1.732 0L2.168 19.499A1 1 0 003.034 21h17.932a1 1 0 00.866-1.5L12.866 3.994z"></path></svg> Same command line, but you need to change <a class="reference internal" href="../formats/Input.html#configfile"><span class="std std-ref">configuration file</span></a>.</p>
</div>
<p>We also propose to run an RWR approach using a <strong>disease-disease similarity network</strong>. In this network, rare diseases
are linked together according their phenotype similarity whereas in the previous network they were not at all connected.</p>
<p>We used a multilayer network composed of three genes network and one disease-disease similarity network
(<a class="reference internal" href="../network/NetworkUsed.html#multilayercompo"><span class="std std-numref">Fig. 6</span></a> - right, <a class="reference internal" href="../network/NetworkUsed.html#genesmultilayernet"><span class="std std-ref">Genes multilayer network</span></a>, <a class="reference internal" href="../network/NetworkUsed.html#similaritynet"><span class="std std-ref">Disease-disease similarity network</span></a>).</p>
<p>multiXrank defined vitamin A target genes as seeds. Over the 521 target genes retrieved from CTD, 480 are found in
the multilayer and used as seeds. Using the RWR scores (i.e. proximity score with the target genes), rare disease are
prioritized. We selected the top 20 and presented the top 5 (<a class="reference internal" href="#usecase2-rwrsim"><span class="std std-ref">Table 19</span></a>).</p>
<span id="usecase2-rwrsim"></span><table class="docutils align-center" id="id52">
<caption><span class="caption-number">Table 19 </span><span class="caption-text">- Rare disease prioritization using RWR score. The top 5 is displayed.</span><a class="headerlink" href="#id52" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"><p>Nodes (disease IDs)</p></th>
<th class="head"><p>Disease Names</p></th>
<th class="head"><p>RWR scores</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>OMIM:178500</p></td>
<td><p>Pulmonary fibrosis, idiopathic</p></td>
<td><p>3.47e-04</p></td>
</tr>
<tr class="row-odd"><td><p>OMIM:125853</p></td>
<td><p>Diabetes mellitus, noninsulin-dependent</p></td>
<td><p>3.13e-04</p></td>
</tr>
<tr class="row-even"><td><p>OMIM:215600</p></td>
<td><p>Cirrhosis, familial</p></td>
<td><p>2.73e-04</p></td>
</tr>
<tr class="row-odd"><td><p>OMIM:613659</p></td>
<td><p>Gastric cancer, somatic</p></td>
<td><p>2.38e-04</p></td>
</tr>
<tr class="row-even"><td><p>OMIM:104300</p></td>
<td><p>Alzheimer disease</p></td>
<td><p>2.34e-04</p></td>
</tr>
</tbody>
</table>
<p>We created a visualisation of the results (<a class="reference internal" href="#usecase2-rwrsimfig"><span class="std std-numref">Fig. 25</span></a>) using Cytoscape <a class="footnote-reference brackets" href="#id41" id="id31" role="doc-noteref"><span class="fn-bracket">[</span>9<span class="fn-bracket">]</span></a>. You can retrieved it in
the cytoscape project called <code class="docutils literal notranslate"><span class="pre">RWR_visualisation.cys</span></code> in <a class="reference external" href="https://github.com/MOohTus/ODAMNet/tree/main/useCases/">GitHub</a>. The <a class="reference internal" href="#usecase2-rwrsimfig"><span class="std std-numref">Fig. 25</span></a> presents the
top 5 of rare disease pathways, ordered by RWR score.</p>
<figure class="align-center" id="id53">
<span id="usecase2-rwrsimfig"></span><a class="reference internal image-reference" href="../../_images/UseCase2_RWR_top5_sim.png"><img alt="useCase2_RWRSimFig" src="../../_images/UseCase2_RWR_top5_sim.png" style="width: 765.8px; height: 545.3px;" /></a>
<figcaption>
<p><span class="caption-number">Fig. 25 </span><span class="caption-text">: Top 5 of the rare disease prioritized using RWR score using disease-disease similarity network</span><a class="headerlink" href="#id53" title="Permalink to this image"></a></p>
<div class="legend">
<blockquote>
<div><p>Rare disease are in pink triangles. Target genes are in grey and non-target genes are in white. Creation
steps are explained in the <a class="reference internal" href="../network/NetworkVisualisation.html#cytoscape-rwr"><span class="std std-ref">Random walk with restart results visualisation</span></a> section.</p>
</div></blockquote>
</div>
</figcaption>
</figure>
</section>
</section>
</section>
</section>
<section id="overlap-ami-and-rwr-results-comparison">
<h2>Overlap, AMI and RWR results comparison<a class="headerlink" href="#overlap-ami-and-rwr-results-comparison" title="Permalink to this heading"></a></h2>
<p>We compare results obtained with the three different approaches: Overlap analysis, Active Module Identification (AMI)
and Random Walk with Restart (RWR). We used orsum <a class="footnote-reference brackets" href="#id43" id="id32" role="doc-noteref"><span class="fn-bracket">[</span>11<span class="fn-bracket">]</span></a>, a Python package to filter and integrate enrichment analysis
from several analyses. The main result is a heatmap, presented in <a class="reference internal" href="#usecase2-orsum"><span class="std std-numref">Fig. 26</span></a>.</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>orsum.py<span class="w"> </span>--gmt<span class="w"> </span>00_Data/hsapiens_background.gmt<span class="w"> </span><span class="se">\</span>
<span class="w">         </span>--files<span class="w"> </span>00_Data/Overlap.4Orsum<span class="w"> </span>00_Data/DOMINO.4Orsum<span class="w"> </span>00_Data/RWR_top5.4Orsum<span class="w"> </span><span class="se">\</span>
<span class="w">         </span>--fileAliases<span class="w"> </span>Overlap<span class="w"> </span>AMI<span class="w"> </span>RWR<span class="w"> </span><span class="se">\</span>
<span class="w">         </span>--maxRepSize<span class="w"> </span><span class="m">0</span><span class="w"> </span><span class="se">\</span>
<span class="w">         </span>--outputFolder<span class="w"> </span>UseCase2_orsum
</pre></div>
</div>
<figure class="align-center" id="id54">
<span id="usecase2-orsum"></span><a class="reference internal image-reference" href="../../_images/UseCase2_orsum.png"><img alt="useCase2_orsum" src="../../_images/UseCase2_orsum.png" style="width: 438.9px; height: 469.2px;" /></a>
<figcaption>
<p><span class="caption-number">Fig. 26 </span><span class="caption-text">: Overlap analysis (Overlap), Active module identification (AMI) and Random walk with restart (RWR) results integration and comparison using orsum</span><a class="headerlink" href="#id54" title="Permalink to this image"></a></p>
</figcaption>
</figure>
<p>The <code class="docutils literal notranslate"><span class="pre">--maxRepSize</span></code> parameter is set to 0 to consider each term as is own representative term.</p>
<p>Some pathways/processes related to CAKUT are retrieved associated with vitamin A by all the three approaches such as
<em>renal system development</em> and <em>kidney development</em>. Some other are retrieved associated with vitamin A only by one
(<em>RET signaling</em>) or two (<em>Signaling by WNT</em>) approaches.</p>
</section>
<section id="references">
<h2>References<a class="headerlink" href="#references" title="Permalink to this heading"></a></h2>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id33" role="note">
<span class="label"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id1">1</a>,<a role="doc-backlink" href="#id2">2</a>,<a role="doc-backlink" href="#id3">3</a>,<a role="doc-backlink" href="#id15">4</a>,<a role="doc-backlink" href="#id25">5</a>)</span>
<p>Ozisik O, Ehrhart F, Evelo C <em>et al.</em>. Overlap of vitamin A and vitamin D target genes with CAKUT-related processes. F1000Research. 2021.</p>
</aside>
<aside class="footnote brackets" id="id34" role="note">
<span class="label"><span class="fn-bracket">[</span>2<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id4">1</a>,<a role="doc-backlink" href="#id10">2</a>,<a role="doc-backlink" href="#id19">3</a>,<a role="doc-backlink" href="#id29">4</a>)</span>
<p>Balmer JE &amp; Blomhoff R. Gene expression regulation by retinoic acid. Journal of lipid research. 2002.</p>
</aside>
<aside class="footnote brackets" id="id35" role="note">
<span class="label"><span class="fn-bracket">[</span>3<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id5">1</a>,<a role="doc-backlink" href="#id12">2</a>,<a role="doc-backlink" href="#id21">3</a>)</span>
<p>Martens M, Ammar A, Riutta A <em>et al.</em>. WikiPathways: connecting communities. Nucleic acids research. 2021.</p>
</aside>
<aside class="footnote brackets" id="id36" role="note">
<span class="label"><span class="fn-bracket">[</span>4<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id6">1</a>,<a role="doc-backlink" href="#id11">2</a>,<a role="doc-backlink" href="#id20">3</a>)</span>
<p>Jassal B, Matthews L, Viteri G <em>et al.</em>. The reactome pathway knowledgebase. Nucleic acids research. 2020.</p>
</aside>
<aside class="footnote brackets" id="id37" role="note">
<span class="label"><span class="fn-bracket">[</span>5<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id7">1</a>,<a role="doc-backlink" href="#id13">2</a>,<a role="doc-backlink" href="#id22">3</a>)</span>
<p>The Gene Ontology Consortium. The Gene Ontology resource: enriching a GOld mine. Nucleic acids research. 2021.</p>
</aside>
<aside class="footnote brackets" id="id38" role="note">
<span class="label"><span class="fn-bracket">[</span>6<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id8">1</a>,<a role="doc-backlink" href="#id14">2</a>,<a role="doc-backlink" href="#id23">3</a>)</span>
<p>Ashburner M, Ball CA, Blake JA <em>et al.</em>. Gene ontology: tool for the unification of biology. Nature Genetics. 2000.</p>
</aside>
<aside class="footnote brackets" id="id39" role="note">
<span class="label"><span class="fn-bracket">[</span>7<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id9">1</a>,<a role="doc-backlink" href="#id17">2</a>)</span>
<p>Pratt D, Chen J, Welker <em>et al.</em>. NDEx, the Network Data Exchange. Cell Systems. 2015.</p>
</aside>
<aside class="footnote brackets" id="id40" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id16">8</a><span class="fn-bracket">]</span></span>
<p>Levi H, Rahmanian N, Elkon R <em>et al.</em>. The DOMINO web-server for active module identification analysis. Bioinformatics. 2022.</p>
</aside>
<aside class="footnote brackets" id="id41" role="note">
<span class="label"><span class="fn-bracket">[</span>9<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id26">1</a>,<a role="doc-backlink" href="#id30">2</a>,<a role="doc-backlink" href="#id31">3</a>)</span>
<p>Shannon P, Markiel A, Ozier O <em>et al.</em>. Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome research. 2003.</p>
</aside>
<aside class="footnote brackets" id="id42" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id27">10</a><span class="fn-bracket">]</span></span>
<p>Baptista A, Gonzalez A &amp; Baudot A. Universal multilayer network exploration by random walk with restart. Communications Physics. 2022.</p>
</aside>
<aside class="footnote brackets" id="id43" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id32">11</a><span class="fn-bracket">]</span></span>
<p>Ozisik O, Térézol M &amp; Baudot A. orsum: a Python package for filtering and comparing enrichment analyses using a simple principle. BMC bioinformatics. 2022.</p>
</aside>
</aside>
</section>
</section>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="usecase1.html" class="btn btn-neutral float-left" title="Use-case 1: data are retrieved automatically by queries" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
        <a href="../formats/Input.html" class="btn btn-neutral float-right" title="Input files" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright 2022, Térézol.</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>