<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
  <meta charset="utf-8" /><meta name="generator" content="Docutils 0.18.1: http://docutils.sourceforge.net/" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>Use-case 1: data are retrieved automatically by queries &mdash; ODAMNet 1 documentation</title>
      <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/custom.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/tabs.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/design-style.4045f2051d55cab465a707391d5b2007.min.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/custom.css" type="text/css" />
      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" type="text/css" />
  <!--[if lt IE 9]>
    <script src="../../_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
        <script data-url_root="../../" id="documentation_options" src="../../_static/documentation_options.js"></script>
        <script src="../../_static/jquery.js"></script>
        <script src="../../_static/underscore.js"></script>
        <script src="../../_static/_sphinx_javascript_frameworks_compat.js"></script>
        <script src="../../_static/doctools.js"></script>
        <script src="../../_static/sphinx_highlight.js"></script>
        <script src="../../_static/design-tabs.js"></script>
    <script src="../../_static/js/theme.js"></script>
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="Use-case 2: data are provided by the user" href="usecase2.html" />
    <link rel="prev" title="Networks visualisation" href="../network/NetworkVisualisation.html" /> 
</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >

          
          
          <a href="../../index.html" class="icon icon-home">
            ODAMNet
          </a>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <p class="caption" role="heading"><span class="caption-text">Quickstart</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../Installation.html">Installation</a></li>
<li class="toctree-l1"><a class="reference internal" href="../Installation.html#usage">Usage</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Approaches</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_overview.html">Overview of the approaches</a></li>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_overlap.html">Overlap analysis</a></li>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_AMI.html">Active Module Identification</a></li>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_RWR.html">Random Walk with Restart</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Networks</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../network/NetworkDownloading.html">Networks downloading</a></li>
<li class="toctree-l1"><a class="reference internal" href="../network/NetworkCreation.html">Networks creation</a></li>
<li class="toctree-l1"><a class="reference internal" href="../network/NetworkUsed.html">Networks used</a></li>
<li class="toctree-l1"><a class="reference internal" href="../network/NetworkVisualisation.html">Networks visualisation</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Use-cases</span></p>
<ul class="current">
<li class="toctree-l1 current"><a class="current reference internal" href="#">Use-case 1: data are retrieved automatically by queries</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#overlap-analysis">Overlap analysis</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#running-overlap-analysis-with-data-retrieved-automatically-from-databases">Running Overlap analysis with data retrieved automatically from databases</a></li>
<li class="toctree-l3"><a class="reference internal" href="#results-of-overlap-analysis-with-data-retrieved-automatically-from-databases">Results of Overlap analysis with data retrieved automatically from databases</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#ctd-query-results">CTD query results</a></li>
<li class="toctree-l4"><a class="reference internal" href="#wikipathways-query-results">WikiPathways query results</a></li>
<li class="toctree-l4"><a class="reference internal" href="#overlap-analysis-results">Overlap analysis results</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#active-module-identification-ami">Active Module Identification (AMI)</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#running-ami-with-data-retrieved-automatically-from-databases">Running AMI with data retrieved automatically from databases</a></li>
<li class="toctree-l3"><a class="reference internal" href="#results-of-ami-with-data-retrieved-automatically-from-databases">Results of AMI with data retrieved automatically from databases</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#id9">CTD query results</a></li>
<li class="toctree-l4"><a class="reference internal" href="#id10">WikiPathways query results</a></li>
<li class="toctree-l4"><a class="reference internal" href="#ppi-network-information">PPI network information</a></li>
<li class="toctree-l4"><a class="reference internal" href="#ami-results">AMI results</a></li>
<li class="toctree-l4"><a class="reference internal" href="#id11">Overlap analysis results</a></li>
<li class="toctree-l4"><a class="reference internal" href="#visualisation-of-ami-results">Visualisation of AMI results</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#random-walk-with-restart-rwr">Random Walk with Restart (RWR)</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#running-rwr-with-data-retrieved-automatically-from-databases">Running RWR with data retrieved automatically from databases</a></li>
<li class="toctree-l3"><a class="reference internal" href="#results-of-rwr-with-data-retrieved-automatically-from-databases">Results of RWR with data retrieved automatically from databases</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#id14">CTD query results</a></li>
<li class="toctree-l4"><a class="reference internal" href="#rwr-results">RWR results</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#overlap-ami-and-rwr-results-comparison">Overlap, AMI and RWR results comparison</a></li>
<li class="toctree-l2"><a class="reference internal" href="#references">References</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="usecase2.html">Use-case 2: data are provided by the user</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">File formats</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../formats/Input.html">Input files</a></li>
<li class="toctree-l1"><a class="reference internal" href="../formats/Output.html">Output files</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Code description</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../functions/fcts_arguments.html">Input parameters for each approach</a></li>
<li class="toctree-l1"><a class="reference internal" href="../functions/fcts_description.html">Code functions in details</a></li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../index.html">ODAMNet</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="../../index.html" class="icon icon-home" aria-label="Home"></a></li>
      <li class="breadcrumb-item active">Use-case 1: data are retrieved automatically by queries</li>
      <li class="wy-breadcrumbs-aside">
            <a href="../../_sources/pages/analysis/usecase1.rst.txt" rel="nofollow"> View page source</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section id="use-case-1-data-are-retrieved-automatically-by-queries">
<h1>Use-case 1: data are retrieved automatically by queries<a class="headerlink" href="#use-case-1-data-are-retrieved-automatically-by-queries" title="Permalink to this heading"></a></h1>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>This analysis is inspired by the study carried out by Ozisik <em>et al.,</em> <a class="footnote-reference brackets" href="#id18" id="id1" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a>.</p>
</div>
<p>In this use-case, we want to study the molecular relationship between <strong>vitamin A</strong> and <strong>rare diseases</strong>.</p>
<p>Vitamin A <strong>target genes</strong> are retrieved <strong>directly</strong> from the <a class="reference external" href="http://ctdbase.org/"><strong>the Comparative Toxicogenomics Database</strong></a> <a class="footnote-reference brackets" href="#id19" id="id2" role="doc-noteref"><span class="fn-bracket">[</span>2<span class="fn-bracket">]</span></a> (CTD). <strong>Rare disease pathways</strong> are
retrieved from <a class="reference external" href="https://www.wikipathways.org/"><strong>WikiPathways</strong></a> <a class="footnote-reference brackets" href="#id20" id="id3" role="doc-noteref"><span class="fn-bracket">[</span>3<span class="fn-bracket">]</span></a>. Biological <strong>networks</strong> used are also downloaded automatically from <a class="reference external" href="https://www.ndexbio.org/"><strong>the Network Data Exchange</strong></a> <a class="footnote-reference brackets" href="#id21" id="id4" role="doc-noteref"><span class="fn-bracket">[</span>4<span class="fn-bracket">]</span></a>. Input data
are available in <a class="reference external" href="https://github.com/MOohTus/ODAMNet/tree/main/useCases/InputData">GitHub</a>.</p>
<p>This section presents you how to apply the three different approaches proposed.</p>
<section id="overlap-analysis">
<span id="usecase1-overlap"></span><h2>Overlap analysis<a class="headerlink" href="#overlap-analysis" title="Permalink to this heading"></a></h2>
<p>The Overlap analysis searches intersecting genes between vitamin A target genes and genes involved in rare disease
pathways (see <a class="reference internal" href="../approaches/methods_overlap.html"><span class="doc">Overlap analysis</span></a> page for more details).</p>
<section id="running-overlap-analysis-with-data-retrieved-automatically-from-databases">
<h3>Running Overlap analysis with data retrieved automatically from databases<a class="headerlink" href="#running-overlap-analysis-with-data-retrieved-automatically-from-databases" title="Permalink to this heading"></a></h3>
<p>The <strong>chemicalsFile.csv</strong> file [<a class="reference internal" href="../formats/Input.html#chemicalsfile"><span class="std std-ref">FORMAT</span></a>] contains the MeSH ID of vitamin A (D014801). We retrieved
from CTD, genes targeted by the vitamin A and its descendant chemicals (<code class="docutils literal notranslate"><span class="pre">--directAssociation</span> <span class="pre">FALSE</span></code>). We keep only
vitamin A - gene interactions which have at least two associated publications (<code class="docutils literal notranslate"><span class="pre">--nbPub</span> <span class="pre">2</span></code>).</p>
<p>Results files are saved into <code class="docutils literal notranslate"><span class="pre">useCases/OutputResults_useCase1/</span></code> folder.</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>odamnet<span class="w"> </span>overlap<span class="w"> </span>--chemicalsFile<span class="w"> </span>useCases/InputData/chemicalsFile.csv<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--directAssociation<span class="w"> </span>FALSE<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--nbPub<span class="w"> </span><span class="m">2</span><span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--outputPath<span class="w"> </span>useCases/OutputResults_useCase1/
</pre></div>
</div>
<p>Several files are generated:</p>
<ul class="simple">
<li><p><code class="docutils literal notranslate"><span class="pre">CTD_request_D014801_2022_09_07.tsv</span></code> and <code class="docutils literal notranslate"><span class="pre">CTD_requestFiltered_D014801_2022_09_07.tsv</span></code>:
the first file contains <strong>results from CTD</strong> query and the second one contains the results filtered using the
publication number.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">WP_RareDiseases_request_2022_09_07.gmt</span></code> and <code class="docutils literal notranslate"><span class="pre">WP_allPathways_request_2022_09_07.gmt</span></code>:
the first file contains <strong>human rare disease pathways</strong> and the second file contains <strong>all human pathways</strong> retrieved
in WikiPathways.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">Overlap_D014801_withRDWP.csv</span></code>: results of the Overlap analysis between vitamin A target genes and rare disease
pathways.</p></li>
</ul>
<p class="italic">For more details about these files, see the <a class="reference internal" href="../formats/Output.html"><span class="doc">Output files</span></a> page.</p>
</section>
<section id="results-of-overlap-analysis-with-data-retrieved-automatically-from-databases">
<h3>Results of Overlap analysis with data retrieved automatically from databases<a class="headerlink" href="#results-of-overlap-analysis-with-data-retrieved-automatically-from-databases" title="Permalink to this heading"></a></h3>
<p><em>Queries made on September 7th, 2022</em></p>
<section id="ctd-query-results">
<h4>CTD query results<a class="headerlink" href="#ctd-query-results" title="Permalink to this heading"></a></h4>
<p>We retrieved 7,765 genes targeted by 10 chemicals (vitamin A + nine descendant chemicals) in CTD
(<a class="reference internal" href="#usecase1-overlapctd"><span class="std std-ref">Table 2</span></a>). Chemical - gene associations are kept if they have at least two publications for
human. After filtering, we have 2,143 vitamin A target genes for 7 chemicals (vitamin A + its descendant molecules).</p>
<span id="usecase1-overlapctd"></span><table class="docutils align-center" id="id26">
<caption><span class="caption-number">Table 2 </span><span class="caption-text">- Vitamin A target genes retrieved from CTD</span><a class="headerlink" href="#id26" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"></th>
<th class="head"><p>Number of chemicals</p></th>
<th class="head"><p>Number of target genes</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Query result</p></td>
<td><p>10</p></td>
<td><p>7,765</p></td>
</tr>
<tr class="row-odd"><td><p>After filtering by associated publications number</p></td>
<td><p>7</p></td>
<td><p>2,143</p></td>
</tr>
</tbody>
</table>
</section>
<section id="wikipathways-query-results">
<h4>WikiPathways query results<a class="headerlink" href="#wikipathways-query-results" title="Permalink to this heading"></a></h4>
<p>All human pathways labeled as “rare disease” are retrieved from WikiPathways. We retrieved 104 rare disease pathways
(<a class="reference internal" href="#usecase1-overlapwp"><span class="std std-ref">Table 3</span></a>). All human pathways are also retrieved from WikiPathways
(<a class="reference internal" href="#usecase1-overlapwp"><span class="std std-ref">Table 3</span></a>). We use these pathways to create background gene sets used for statistical
analysis.</p>
<span id="usecase1-overlapwp"></span><table class="docutils align-center" id="id27">
<caption><span class="caption-number">Table 3 </span><span class="caption-text">- Pathways retrieval from WikiPathways</span><a class="headerlink" href="#id27" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"></th>
<th class="head"><p>Pathways number</p></th>
<th class="head"><p>Min genes number</p></th>
<th class="head"><p>Max genes number</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Rare Disease Pathways</p></td>
<td><p>104</p></td>
<td><p>3</p></td>
<td><p>436</p></td>
</tr>
<tr class="row-odd"><td><p>All Human WikiPathways</p></td>
<td><p>1,281</p></td>
<td><p>1</p></td>
<td><p>484</p></td>
</tr>
</tbody>
</table>
</section>
<section id="overlap-analysis-results">
<h4>Overlap analysis results<a class="headerlink" href="#overlap-analysis-results" title="Permalink to this heading"></a></h4>
<p>We performed an Overlap analysis between vitamin A target genes (2,143) and rare disease pathways (104). We obtained
significant overlap between target genes and <strong>28 rare disease pathways</strong> (pAdjusted &lt;= 0.05). The top 5 is presented in
<a class="reference internal" href="#usecase1-overlaptop5"><span class="std std-ref">Table 4</span></a>.</p>
<span id="usecase1-overlaptop5"></span><table class="docutils align-center" id="id28">
<caption><span class="caption-number">Table 4 </span><span class="caption-text">- Top 5 of the significant overlaps between the vitamin A target genes and rare disease pathways</span><a class="headerlink" href="#id28" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"><p>PathwayIDs</p></th>
<th class="head"><p>PathwayNames</p></th>
<th class="head"><p>pAdjusted</p></th>
<th class="head"><p>IntersectionSize</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>WP5087</p></td>
<td><p>Malignant pleural mesothelioma</p></td>
<td><p>3.77e-24</p></td>
<td><p>146</p></td>
</tr>
<tr class="row-odd"><td><p>WP4298</p></td>
<td><p>Acute viral myocarditis</p></td>
<td><p>9.38e-16</p></td>
<td><p>45</p></td>
</tr>
<tr class="row-even"><td><p>WP2447</p></td>
<td><p>Amyotrophic lateral sclerosis (ALS)</p></td>
<td><p>1.04e-11</p></td>
<td><p>25</p></td>
</tr>
<tr class="row-odd"><td><p>WP5053</p></td>
<td><p>Development of ureteric collection system</p></td>
<td><p>2.61e-08</p></td>
<td><p>28</p></td>
</tr>
<tr class="row-even"><td><p>WP4879</p></td>
<td><p>Overlap between signal transduction pathways …</p></td>
<td><p>7.80e-07</p></td>
<td><p>25</p></td>
</tr>
</tbody>
</table>
<p>In a previous analysis <a class="footnote-reference brackets" href="#id18" id="id5" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a>, an overlap analysis was performed between vitamin A and Congenital Anomalies of the Kidney
and Urinary Tract (CAKUT). Four pathways related to CAKUT were identified in WikiPathways. Significant overlaps were
identified between these four CAKUT pathways and vitamin A target genes.</p>
<p>With updated target genes data proposed here, we also retrieved significant overlap for 3 of these 4 CAKUT pathways
(<a class="reference internal" href="#usecase1-overlapcakut"><span class="std std-ref">Table 5</span></a>).</p>
<span id="usecase1-overlapcakut"></span><table class="docutils align-center" id="id29">
<caption><span class="caption-number">Table 5 </span><span class="caption-text">- Overlap analysis results between vitamin A target genes and CAKUT pathways</span><a class="headerlink" href="#id29" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"></th>
<th class="head"></th>
<th class="head" colspan="2"><p>Current analysis</p></th>
<th class="head" colspan="2"><p>Ozisik <em>et al.</em>, 2021</p></th>
</tr>
<tr class="row-even"><th class="head"><p>PathwayIDs</p></th>
<th class="head"><p>Pathway Names</p></th>
<th class="head"><p>pAdjusted</p></th>
<th class="head"><p>Inter</p></th>
<th class="head"><p>pAdjusted</p></th>
<th class="head"><p>Inter</p></th>
</tr>
</thead>
<tbody>
<tr class="row-odd"><td><p>WP5053</p></td>
<td><p>Development of ureteric collection …</p></td>
<td><p><strong>2.61e-08</strong></p></td>
<td><p>28</p></td>
<td><p><strong>1.59e-05</strong></p></td>
<td><p>16</p></td>
</tr>
<tr class="row-even"><td><p>WP4830</p></td>
<td><p>GDNF/RET signaling axis</p></td>
<td><p><strong>1.99e-05</strong></p></td>
<td><p>13</p></td>
<td><p><strong>1.57e-03</strong></p></td>
<td><p>8</p></td>
</tr>
<tr class="row-odd"><td><p>WP4823</p></td>
<td><p>Genes controlling nephrogenesis</p></td>
<td><p><strong>8.72e-05</strong></p></td>
<td><p>18</p></td>
<td><p><strong>1.84e-05</strong></p></td>
<td><p>15</p></td>
</tr>
<tr class="row-even"><td><p>WP5052</p></td>
<td><p>Nephrogenesis</p></td>
<td><p>0.09</p></td>
<td><p>6</p></td>
<td><p><strong>1.90e-04</strong></p></td>
<td><p>8</p></td>
</tr>
</tbody>
</table>
<p>The increase of the intersection size (<em>Inter column</em>) can be explained by the target gene size. In the previous work
<a class="footnote-reference brackets" href="#id18" id="id6" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a>, we retrieved 1,086 target genes and in this current work we retrieved 2,143 target genes.</p>
<p>The overlap between <em>Nephrogenesis</em> pathway and target genes is not found significant anymore. Number of target genes
shared with the pathway is smaller. It affects the p-value and decreases it below the 0.05 threshold.
It can be explained by the fact that one of the two missing genes is not related to human. So it cannot be retrieved by
the queri. And the other gene has only one publication that relates association with vitamin A. So it is not kept
during filtering.</p>
</section>
</section>
</section>
<section id="active-module-identification-ami">
<span id="usecase1-ami"></span><h2>Active Module Identification (AMI)<a class="headerlink" href="#active-module-identification-ami" title="Permalink to this heading"></a></h2>
<p>The Active Module Identification (AMI) approach identifies active module that contains high number of vitamin A target
genes using a protein-protein interaction (PPI) network. AMI is performed using DOMINO <a class="footnote-reference brackets" href="#id22" id="id7" role="doc-noteref"><span class="fn-bracket">[</span>5<span class="fn-bracket">]</span></a>. Then, an Overlap analysis
is applied between identified active modules and rare disease pathways. See <a class="reference internal" href="../approaches/methods_AMI.html"><span class="doc">Active Module Identification</span></a> page for
more details.</p>
<section id="running-ami-with-data-retrieved-automatically-from-databases">
<h3>Running AMI with data retrieved automatically from databases<a class="headerlink" href="#running-ami-with-data-retrieved-automatically-from-databases" title="Permalink to this heading"></a></h3>
<div class="admonition warning">
<p class="admonition-title">Warning</p>
<p><svg version="1.1" width="2.0em" height="2.0em" class="sd-octicon sd-octicon-alert" viewBox="0 0 24 24" aria-hidden="true"><path d="M13 17.5a1 1 0 11-2 0 1 1 0 012 0zm-.25-8.25a.75.75 0 00-1.5 0v4.5a.75.75 0 001.5 0v-4.5z"></path><path fill-rule="evenodd" d="M9.836 3.244c.963-1.665 3.365-1.665 4.328 0l8.967 15.504c.963 1.667-.24 3.752-2.165 3.752H3.034c-1.926 0-3.128-2.085-2.165-3.752L9.836 3.244zm3.03.751a1 1 0 00-1.732 0L2.168 19.499A1 1 0 003.034 21h17.932a1 1 0 00.866-1.5L12.866 3.994z"></path></svg> When using DOMINO server, <strong>results cannot be identically reproduced</strong>. Indeed, DOMINO server doesn’t allow to set the random seed. This random seed changes every new analysis.</p>
</div>
<p>The <strong>chemicalsFile.csv</strong> file [<a class="reference internal" href="../formats/Input.html#chemicalsfile"><span class="std std-ref">FORMAT</span></a>] contains the MeSH ID of vitamin A (D014801). We retrieved
from CTD, genes targeted by the vitamin A and its descendant chemicals (<code class="docutils literal notranslate"><span class="pre">--directAssociation</span> <span class="pre">FALSE</span></code>). We keep only
vitamin A - gene interactions which have at least two associated publications (<code class="docutils literal notranslate"><span class="pre">--nbPub</span> <span class="pre">2</span></code>).</p>
<p>We download automatically a PPI network [<a class="reference internal" href="../formats/Input.html#sif"><span class="std std-ref">FORMAT</span></a>] from NDEx <a class="footnote-reference brackets" href="#id21" id="id8" role="doc-noteref"><span class="fn-bracket">[</span>4<span class="fn-bracket">]</span></a> using the <code class="docutils literal notranslate"><span class="pre">--netUUID</span></code> parameter
(UUID bfac0486-cefe-11ed-a79c-005056ae23aa, version 1.0). We named the PPI network <cite>PPI_HiUnion_LitBM_APID_gene_names_190123.sif</cite>
(<code class="docutils literal notranslate"><span class="pre">--networkFile</span></code>). Network name should have <strong>.sif</strong> extension.</p>
<p>Results files are saved into <code class="docutils literal notranslate"><span class="pre">useCases/OutputResults_useCase1/</span></code> folder.</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>odamnet<span class="w"> </span>domino<span class="w">  </span>--chemicalsFile<span class="w"> </span>useCases/InputData/chemicalsFiles.csv<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--directAssociation<span class="w"> </span>FALSE<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--nbPub<span class="w"> </span><span class="m">2</span><span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--networkFile<span class="w"> </span>useCases/InputData/PPI_HiUnion_LitBM_APID_gene_names_190123.sif<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--netUUID<span class="w"> </span>bfac0486-cefe-11ed-a79c-005056ae23aa<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--outputPath<span class="w"> </span>useCases/OutputResults_useCase1
</pre></div>
</div>
<p>Several files are generated:</p>
<ul class="simple">
<li><p><code class="docutils literal notranslate"><span class="pre">CTD_request_D014801_2022_09_07.tsv</span></code> and <code class="docutils literal notranslate"><span class="pre">CTD_requestFiltered_D014801_2022_09_07.tsv</span></code>:
the first file contains <strong>results from CTD</strong> query and the second one contains the results filtered using the
publication number.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">WP_RareDiseases_request_2022_09_07.gmt</span></code> and <code class="docutils literal notranslate"><span class="pre">WP_allPathways_request_2022_09_07.gmt</span></code>:
the first file contains <strong>human rare disease pathways</strong> and the second file contains <strong>all human pathways</strong> retrieved
in WikiPathways.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">DOMINO_inputGeneList_D014801.txt</span></code>: vitamin A target genes list used for the active module identification.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">Overlap_AM_*_D014801_withRDWP.csv</span></code>: results of the Overlap analysis between identified active modules genes and
rare disease pathways. There is one file per active module.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">DOMINO_D014801_activeModulesNetwork.txt</span></code>, <code class="docutils literal notranslate"><span class="pre">DOMINO_D014801_overlapAMresults4Cytoscape.txt</span></code>, <code class="docutils literal notranslate"><span class="pre">DOMINO_D014801_activeModules.txt</span></code>
, <code class="docutils literal notranslate"><span class="pre">DOMINO_D014801_activeModulesMetrics.txt</span></code> and <code class="docutils literal notranslate"><span class="pre">DOMINO_D014801_signOverlap.txt</span></code>: some statistics are
calculated and saved into files. Theses files are useful for visualisation.</p></li>
</ul>
<p class="italic">For more details about these files, see <a class="reference internal" href="../formats/Output.html"><span class="doc">Output files</span></a> page (<a class="reference internal" href="../formats/Output.html#queryoutput"><span class="std std-ref">Query output files</span></a>, <a class="reference internal" href="../formats/Output.html#overlapoutput"><span class="std std-ref">Overlap analysis output files</span></a>,
<a class="reference internal" href="../formats/Output.html#amioutput"><span class="std std-ref">AMI output files</span></a>)</p>
</section>
<section id="results-of-ami-with-data-retrieved-automatically-from-databases">
<h3>Results of AMI with data retrieved automatically from databases<a class="headerlink" href="#results-of-ami-with-data-retrieved-automatically-from-databases" title="Permalink to this heading"></a></h3>
<p><em>Queries made on September 7th, 2022</em></p>
<section id="id9">
<h4>CTD query results<a class="headerlink" href="#id9" title="Permalink to this heading"></a></h4>
<p>We retrieved 7,765 genes targeted by 10 chemicals (vitamin A + nine descendant chemicals) in CTD
(<a class="reference internal" href="#usecase1-amictd"><span class="std std-ref">Table 6</span></a>). Chemical - gene associations are kept if they have at least two publications for
human. After filtering, we have 2,143 vitamin A target genes for 7 chemicals (vitamin A + its descendant molecules).</p>
<span id="usecase1-amictd"></span><table class="docutils align-center" id="id30">
<caption><span class="caption-number">Table 6 </span><span class="caption-text">- Vitamin A target genes retrieved from CTD</span><a class="headerlink" href="#id30" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"></th>
<th class="head"><p>Number of chemicals</p></th>
<th class="head"><p>Number of genes</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Query result</p></td>
<td><p>10</p></td>
<td><p>7,765</p></td>
</tr>
<tr class="row-odd"><td><p>After filtering by publication number</p></td>
<td><p>7</p></td>
<td><p>2,143</p></td>
</tr>
</tbody>
</table>
</section>
<section id="id10">
<h4>WikiPathways query results<a class="headerlink" href="#id10" title="Permalink to this heading"></a></h4>
<p>All human pathways labeled as “rare disease” are retrieved from WikiPathways. We retrieved 104 rare disease pathways
(<a class="reference internal" href="#usecase1-amiwp"><span class="std std-ref">Table 7</span></a>). All human pathways are also retrieved from WikiPathways
(<a class="reference internal" href="#usecase1-amiwp"><span class="std std-ref">Table 7</span></a>). We use these pathways to create background gene sets used for statistical analysis.</p>
<span id="usecase1-amiwp"></span><table class="docutils align-center" id="id31">
<caption><span class="caption-number">Table 7 </span><span class="caption-text">- Pathways retrieval from WikiPathways</span><a class="headerlink" href="#id31" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"></th>
<th class="head"><p>Pathways number</p></th>
<th class="head"><p>Min genes number</p></th>
<th class="head"><p>Max genes number</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Rare Disease Pathways</p></td>
<td><p>104</p></td>
<td><p>3</p></td>
<td><p>436</p></td>
</tr>
<tr class="row-odd"><td><p>All Human WikiPathways</p></td>
<td><p>1,281</p></td>
<td><p>1</p></td>
<td><p>484</p></td>
</tr>
</tbody>
</table>
</section>
<section id="ppi-network-information">
<h4>PPI network information<a class="headerlink" href="#ppi-network-information" title="Permalink to this heading"></a></h4>
<p>The PPI network is automatically downloaded from <a class="reference external" href="https://www.ndexbio.org/viewer/networks/bfac0486-cefe-11ed-a79c-005056ae23aa">NDEx</a>. It was build from 3 datasets: Lit-BM, Hi-Union and APID. It
contains 15,390 nodes and 131,087 edges.</p>
<p class="italic">For more details about the PPI network, see <a class="reference internal" href="../network/NetworkUsed.html#ppinet"><span class="std std-ref">Protein-Protein Interaction (PPI) network</span></a> section.</p>
</section>
<section id="ami-results">
<h4>AMI results<a class="headerlink" href="#ami-results" title="Permalink to this heading"></a></h4>
<p>DOMINO defines vitamin A target genes as active genes and searches active modules enriched in active genes. Over the
2,143 target genes retrieved from CTD, 1,937 are found in the PPI and used as active genes by DOMINO. DOMINO identified
<strong>12 active modules</strong> enriched in vitamin A target genes (<a class="reference internal" href="#usecase1-amiresults"><span class="std std-ref">Table 8</span></a>).</p>
<span id="usecase1-amiresults"></span><table class="docutils align-center" id="id32">
<caption><span class="caption-number">Table 8 </span><span class="caption-text">- Composition of the active modules identified enriched in vitamin A target genes by DOMINO</span><a class="headerlink" href="#id32" title="Permalink to this table"></a></caption>
<colgroup>
<col style="width: 55%" />
<col style="width: 23%" />
<col style="width: 23%" />
</colgroup>
<thead>
<tr class="row-odd"><th class="head"></th>
<th class="head"><p>Min number</p></th>
<th class="head"><p>Max number</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Edges</p></td>
<td><p>20</p></td>
<td><p>357</p></td>
</tr>
<tr class="row-odd"><td><p>Nodes</p></td>
<td><p>17</p></td>
<td><p>93</p></td>
</tr>
<tr class="row-even"><td><p>Target genes</p></td>
<td><p>8</p></td>
<td><p>35</p></td>
</tr>
</tbody>
</table>
<p class="italic">See <code class="docutils literal notranslate"><span class="pre">DOMINO_D014801_activeModulesNetworkMetrics.txt</span></code> file for more details.</p>
</section>
<section id="id11">
<h4>Overlap analysis results<a class="headerlink" href="#id11" title="Permalink to this heading"></a></h4>
<p>Then, we perform an Overlap analysis between identified active modules (12) and rare disease pathways (104). We obtained
significant overlap between <strong>6 active modules</strong> and <strong>19 rare disease pathways</strong> (pAdjusted &lt;= 0.05). The top 5 is
presented in <a class="reference internal" href="#usecase1-amioverlap"><span class="std std-ref">Table 9</span></a>.</p>
<span id="usecase1-amioverlap"></span><table class="docutils align-center" id="id33">
<caption><span class="caption-number">Table 9 </span><span class="caption-text">- Top 5 of the significant overlaps between identified active modules and rare disease pathways</span><a class="headerlink" href="#id33" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"><p>Pathway IDs</p></th>
<th class="head"><p>Pathway Names</p></th>
<th class="head"><p>pAdjusted</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>WP5087</p></td>
<td><p>Malignant pleural mesothelioma</p></td>
<td><p>2.78e-25</p></td>
</tr>
<tr class="row-odd"><td><p>WP4541</p></td>
<td><p>Hippo-Merlin signaling dysregulation</p></td>
<td><p>4.37e-07</p></td>
</tr>
<tr class="row-even"><td><p>WP4577</p></td>
<td><p>Neurodegeneration with brain iron accumulation (NBIA) subtypes …</p></td>
<td><p>2.84e-06</p></td>
</tr>
<tr class="row-odd"><td><p>WP5053</p></td>
<td><p>Development of ureteric collection system</p></td>
<td><p>1.23e-05</p></td>
</tr>
<tr class="row-even"><td><p>WP4540</p></td>
<td><p>Hippo signaling regulation pathways</p></td>
<td><p>1.55e-05</p></td>
</tr>
</tbody>
</table>
<p>Duplicates between active modules results are removed and we keep the more significant ones.</p>
<p class="italic">See <code class="docutils literal notranslate"><span class="pre">DOMINO_D014801_signOverlap.txt</span></code> file for more details.</p>
</section>
<section id="visualisation-of-ami-results">
<h4>Visualisation of AMI results<a class="headerlink" href="#visualisation-of-ami-results" title="Permalink to this heading"></a></h4>
<p>We created a visualisation of AMI results (<a class="reference internal" href="#usecase1-amifig"><span class="std std-numref">Fig. 19</span></a>) using Cytoscape <a class="footnote-reference brackets" href="#id23" id="id12" role="doc-noteref"><span class="fn-bracket">[</span>6<span class="fn-bracket">]</span></a>.</p>
<p>We found a significant overlap between <strong>6 active modules</strong> and <strong>19 rare disease pathways</strong>. For sake of visualisation,
we selected only three of them (<a class="reference internal" href="#usecase1-amifig"><span class="std std-numref">Fig. 19</span></a>). You can find the entire visualisation in the cytoscape
project called <code class="docutils literal notranslate"><span class="pre">AMI_visualisation.cys</span></code> in <a class="reference external" href="https://github.com/MOohTus/ODAMNet/tree/main/useCases/">GitHub</a>.</p>
<figure class="align-center" id="id34">
<span id="usecase1-amifig"></span><a class="reference internal image-reference" href="../../_images/UseCase1_AMI.png"><img alt="useCase1_AMIFig" src="../../_images/UseCase1_AMI.png" style="width: 683.5500000000001px; height: 353.7px;" /></a>
<figcaption>
<p><span class="caption-number">Fig. 19 </span><span class="caption-text">: Visualisation of 3 active modules and their associated rare disease pathways</span><a class="headerlink" href="#id34" title="Permalink to this image"></a></p>
<div class="legend">
<blockquote>
<div><p>Genes are represented by nodes. Grey nodes are the target genes, white nodes are non-target genes. Overlap results
between active modules and rare disease pathways are displayed using donuts color around nodes. Each color
corresponds to a rare disease pathways. Creation steps are explained in the <a class="reference internal" href="../network/NetworkVisualisation.html#cytoscape-ami"><span class="std std-ref">Active module identification results visualisation</span></a> section.</p>
</div></blockquote>
</div>
</figcaption>
</figure>
<p>Module topology is different between modules and associated rare diseases pathways also vary (<a class="reference internal" href="#usecase1-amifig"><span class="std std-numref">Fig. 19</span></a>).
For instance, the module on the right is very connected and contains genes that are involved in a lot of rare disease
pathways. Genes, such as <em>PTEN</em>, are part of at least 5 pathways. The two other modules are sparser. The module in the
middle contains genes involved only in <em>Development of ureteric collection system</em>.</p>
</section>
</section>
</section>
<section id="random-walk-with-restart-rwr">
<span id="usecase1-rwr"></span><h2>Random Walk with Restart (RWR)<a class="headerlink" href="#random-walk-with-restart-rwr" title="Permalink to this heading"></a></h2>
<p>The Random Walk with Restart (RWR) approach mesures proximities between vitamin A target genes and rare disease pathways.
To calculate these proximities (RWR scores), we used multiXrank <a class="footnote-reference brackets" href="#id24" id="id13" role="doc-noteref"><span class="fn-bracket">[</span>7<span class="fn-bracket">]</span></a> and multilayer networks. See
<a class="reference internal" href="../approaches/methods_RWR.html"><span class="doc">Random Walk with Restart</span></a> page for more details.</p>
<p>The multilayer network is composed of three gene networks and one rare disease pathways network. Genes nodes are
connected to disease nodes if they are involved in. See <a class="reference internal" href="../network/NetworkUsed.html"><span class="doc">Networks used</span></a> page for more details.</p>
<section id="running-rwr-with-data-retrieved-automatically-from-databases">
<h3>Running RWR with data retrieved automatically from databases<a class="headerlink" href="#running-rwr-with-data-retrieved-automatically-from-databases" title="Permalink to this heading"></a></h3>
<p>The <strong>chemicalsFile.csv</strong> file [<a class="reference internal" href="../formats/Input.html#chemicalsfile"><span class="std std-ref">FORMAT</span></a>] contains the MeSH ID of vitamin A (D014801). We retrieved
from CTD, genes targeted by the vitamin A and its descendant chemicals (<code class="docutils literal notranslate"><span class="pre">--directAssociation</span> <span class="pre">FALSE</span></code>). We keep only
vitamin A - gene interactions which have at least two associated publications (<code class="docutils literal notranslate"><span class="pre">--nbPub</span> <span class="pre">2</span></code>).</p>
<p>MultiXrank needs as input a configuration file (<code class="docutils literal notranslate"><span class="pre">--configPath</span></code>) that contains path of networks and analysis parameters.
We used multiXrank with default parameters.</p>
<p>We provided a name file to save vitamin A target genes (i.e. seeds) <code class="docutils literal notranslate"><span class="pre">--seedsFile</span> <span class="pre">useCases/InputData/seeds.txt</span></code> and
also a SIF file name (<code class="docutils literal notranslate"><span class="pre">--sifFileName</span></code>) to save the top nodes based on RWR score (<code class="docutils literal notranslate"><span class="pre">--top</span> <span class="pre">20</span></code>).</p>
<p>Results files are saved into <code class="docutils literal notranslate"><span class="pre">useCases/OutputResults_useCase1/</span></code> folder.</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>odamnet<span class="w"> </span>multixrank<span class="w">  </span>--chemicalsFile<span class="w"> </span>useCases/InputData/chemicalsFile.csv<span class="w"> </span><span class="se">\</span>
<span class="w">                    </span>--directAssociation<span class="w"> </span>FALSE<span class="w"> </span><span class="se">\</span>
<span class="w">                    </span>--nbPub<span class="w"> </span><span class="m">2</span><span class="w"> </span><span class="se">\</span>
<span class="w">                    </span>--configPath<span class="w"> </span>useCases/InputData/config_minimal_useCase1.yml<span class="w"> </span><span class="se">\</span>
<span class="w">                    </span>--networksPath<span class="w"> </span>useCases/InputData/<span class="w"> </span><span class="se">\</span>
<span class="w">                    </span>--seedsFile<span class="w"> </span>useCases/InputData/seeds.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                    </span>--sifFileName<span class="w"> </span>UseCase1_RWR_network.sif<span class="w"> </span><span class="se">\</span>
<span class="w">                    </span>--top<span class="w"> </span><span class="m">20</span><span class="w"> </span><span class="se">\</span>
<span class="w">                    </span>--outputPath<span class="w"> </span>useCases/OutputResults_useCase1/
</pre></div>
</div>
<div class="admonition tip">
<p class="admonition-title">Tip</p>
<div class="line-block">
<div class="line">- Downloading of multiplex network from NDEx: <a class="reference internal" href="../network/NetworkUsed.html"><span class="doc">Networks used</span></a> + <a class="reference internal" href="../network/NetworkDownloading.html"><span class="doc">Networks downloading</span></a></div>
<div class="line">- Creation of the rare disease pathways network: <a class="reference internal" href="../network/NetworkUsed.html"><span class="doc">Networks used</span></a> +  <a class="reference internal" href="../network/NetworkCreation.html"><span class="doc">Networks creation</span></a></div>
<div class="line">- Configuration file explanation and example: <a class="reference internal" href="../formats/Input.html#configfile"><span class="std std-ref">Configuration file</span></a> section</div>
</div>
</div>
<p>Several files are generated:</p>
<ul>
<li><p><code class="docutils literal notranslate"><span class="pre">CTD_request_D014801_2022_09_07.tsv</span></code> and <code class="docutils literal notranslate"><span class="pre">CTD_requestFiltered_D014801_2022_09_07.tsv</span></code>:
the first file contains <strong>results from CTD</strong> query and the second one contains the filtered by publication number.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">RWR_D014801/</span></code> folder with the RWR results:</p>
<blockquote>
<div><ul class="simple">
<li><p><code class="docutils literal notranslate"><span class="pre">config_minimal_useCase1.yml</span></code> and <code class="docutils literal notranslate"><span class="pre">seeds.txt</span></code>: copies of the input files</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">multiplex_1.tsv</span></code> and <code class="docutils literal notranslate"><span class="pre">multiplex_2.tsv</span></code>: RWR scores for each multilayer. 1 is the genes multilayer network RWR
scores and 2 is the rare disease pathways network RWR scores.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">UseCase1_RWR_network.sif</span></code>: SIF file name that contains the network result</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">RWR_topX.txt</span></code>: Top X of rare disease pathways</p></li>
</ul>
</div></blockquote>
</li>
</ul>
<p class="italic">For more details about these file, see <a class="reference internal" href="../formats/Output.html"><span class="doc">Output files</span></a> page.</p>
</section>
<section id="results-of-rwr-with-data-retrieved-automatically-from-databases">
<h3>Results of RWR with data retrieved automatically from databases<a class="headerlink" href="#results-of-rwr-with-data-retrieved-automatically-from-databases" title="Permalink to this heading"></a></h3>
<p><em>Queries made on September 7th, 2022</em></p>
<section id="id14">
<h4>CTD query results<a class="headerlink" href="#id14" title="Permalink to this heading"></a></h4>
<p>We retrieved 7,765 genes targeted by 10 chemicals (vitamin A + nine descendant chemicals) in CTD
(<a class="reference internal" href="#usecase1-rwrctd"><span class="std std-ref">Table 10</span></a>). Chemical - gene associations are kept if they have at least two publications for
human. After filtering, we have 2,143 vitamin A target genes for 7 chemicals (vitamin A + its descendant molecules).</p>
<span id="usecase1-rwrctd"></span><table class="docutils align-center" id="id35">
<caption><span class="caption-number">Table 10 </span><span class="caption-text">- Vitamin A target genes retrieved from CTD</span><a class="headerlink" href="#id35" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"></th>
<th class="head"><p>Number of chemicals</p></th>
<th class="head"><p>Number of genes</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Request result</p></td>
<td><p>10</p></td>
<td><p>7,765</p></td>
</tr>
<tr class="row-odd"><td><p>After filtering by papers number</p></td>
<td><p>7</p></td>
<td><p>2,143</p></td>
</tr>
</tbody>
</table>
</section>
<section id="rwr-results">
<h4>RWR results<a class="headerlink" href="#rwr-results" title="Permalink to this heading"></a></h4>
<section id="analysis-with-rare-disease-pathways-network">
<h5>Analysis with rare disease pathways network<a class="headerlink" href="#analysis-with-rare-disease-pathways-network" title="Permalink to this heading"></a></h5>
<p>We used a multilayer network composed of three gene networks and one rare disease pathways network
(<a class="reference internal" href="../network/NetworkUsed.html#multilayercompo"><span class="std std-numref">Fig. 6</span></a> - left, <a class="reference internal" href="../network/NetworkUsed.html#genesmultilayernet"><span class="std std-ref">Genes multilayer network</span></a> and
<a class="reference internal" href="../network/NetworkUsed.html#pathwaysofinterestnet"><span class="std std-ref">Rare disease pathways network</span></a>).</p>
<p>multiXrank defined vitamin A target genes as seeds. Over the 2,143 target genes retrieved from CTD, 2,012 are found in
the multilayer and used as seeds. Using the RWR scores (i.e. proximity score with the target genes), rare disease
pathways are prioritized. We selected the top 20 and presented the top 5 (<a class="reference internal" href="#usecase1-rwrwp"><span class="std std-ref">Table 11</span></a>).</p>
<span id="usecase1-rwrwp"></span><table class="docutils align-center" id="id36">
<caption><span class="caption-number">Table 11 </span><span class="caption-text">- Rare disease pathways prioritization using RWR score. The top 5 is displayed.</span><a class="headerlink" href="#id36" title="Permalink to this table"></a></caption>
<colgroup>
<col style="width: 25%" />
<col style="width: 50%" />
<col style="width: 25%" />
</colgroup>
<thead>
<tr class="row-odd"><th class="head"><p>Nodes (pathway IDs)</p></th>
<th class="head"><p>Pathway Names</p></th>
<th class="head"><p>RWR scores</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>WP5087</p></td>
<td><p>Malignant pleural mesothelioma</p></td>
<td><p>2.85e-03</p></td>
</tr>
<tr class="row-odd"><td><p>WP4673</p></td>
<td><p>Male infertility</p></td>
<td><p>9.02e-04</p></td>
</tr>
<tr class="row-even"><td><p>WP2059</p></td>
<td><p>Alzheimer’s disease and miRNA effects</p></td>
<td><p>7.76e-04</p></td>
</tr>
<tr class="row-odd"><td><p>WP5124</p></td>
<td><p>Alzheimer’s disease</p></td>
<td><p>7.76e-04</p></td>
</tr>
<tr class="row-even"><td><p>WP4298</p></td>
<td><p>Acute viral myocarditis</p></td>
<td><p>0.000731</p></td>
</tr>
</tbody>
</table>
<p>We created a visualisation of the results (<a class="reference internal" href="#usecase1-rwrwpfig"><span class="std std-numref">Fig. 20</span></a>) using Cytoscape <a class="footnote-reference brackets" href="#id23" id="id15" role="doc-noteref"><span class="fn-bracket">[</span>6<span class="fn-bracket">]</span></a>. You can retrieved it in the
cytoscape project called <code class="docutils literal notranslate"><span class="pre">RWR_visualisation.cys</span></code> in <a class="reference external" href="https://github.com/MOohTus/ODAMNet/tree/main/useCases/">GitHub</a>. The <a class="reference internal" href="#usecase1-rwrwpfig"><span class="std std-numref">Fig. 20</span></a> presents the top 5 of rare
disease pathways, ordered by RWR score.</p>
<figure class="align-center" id="id37">
<span id="usecase1-rwrwpfig"></span><a class="reference internal image-reference" href="../../_images/UseCase1_RWR_top5.png"><img alt="useCase1_RWRWPFig" src="../../_images/UseCase1_RWR_top5.png" style="width: 692.3px; height: 543.9px;" /></a>
<figcaption>
<p><span class="caption-number">Fig. 20 </span><span class="caption-text">: Top 5 of the rare disease pathways prioritized using RWR score using a (disconnected) rare disease pathways network</span><a class="headerlink" href="#id37" title="Permalink to this image"></a></p>
<div class="legend">
<blockquote>
<div><p>Rare disease pathways are in pink triangles. Target genes are in grey and non-target genes are in white. Creation
steps are explained in the <a class="reference internal" href="../network/NetworkVisualisation.html#cytoscape-rwr"><span class="std std-ref">Random walk with restart results visualisation</span></a> section.</p>
</div></blockquote>
</div>
</figcaption>
</figure>
</section>
<section id="extra-analysis-with-disease-disease-similarity-network">
<h5>Extra : analysis with disease-disease similarity network<a class="headerlink" href="#extra-analysis-with-disease-disease-similarity-network" title="Permalink to this heading"></a></h5>
<div class="admonition tip">
<p class="admonition-title">Tip</p>
<p><svg version="1.1" width="1.5em" height="1.5em" class="sd-octicon sd-octicon-alert" viewBox="0 0 24 24" aria-hidden="true"><path d="M13 17.5a1 1 0 11-2 0 1 1 0 012 0zm-.25-8.25a.75.75 0 00-1.5 0v4.5a.75.75 0 001.5 0v-4.5z"></path><path fill-rule="evenodd" d="M9.836 3.244c.963-1.665 3.365-1.665 4.328 0l8.967 15.504c.963 1.667-.24 3.752-2.165 3.752H3.034c-1.926 0-3.128-2.085-2.165-3.752L9.836 3.244zm3.03.751a1 1 0 00-1.732 0L2.168 19.499A1 1 0 003.034 21h17.932a1 1 0 00.866-1.5L12.866 3.994z"></path></svg> Same command line, but you need to change <strong>configuration file</strong>.</p>
</div>
<p>We also propose to run an RWR approach using a <strong>disease-disease similarity network</strong>. In this network, rare diseases are
linked together according their phenotype similarity whereas in the previous network they were not at all connected.</p>
<p>We used a multilayer network composed of three genes network and one disease-disease similarity network
(<a class="reference internal" href="../network/NetworkUsed.html#multilayercompo"><span class="std std-numref">Fig. 6</span></a> - right, <a class="reference internal" href="../network/NetworkUsed.html#genesmultilayernet"><span class="std std-ref">Genes multilayer network</span></a>, <a class="reference internal" href="../network/NetworkUsed.html#similaritynet"><span class="std std-ref">Disease-disease similarity network</span></a>).</p>
<p>multiXrank defined vitamin A target genes as seeds. Over the 2,143 target genes retrieved from CTD, 2,012 are found in
the multilayer and used as seeds. Using the RWR scores (i.e. proximity score with the target genes), rare disease are
prioritized. We selected the top 20 and presented the top 5 (<a class="reference internal" href="#usecase1-rwrsim"><span class="std std-ref">Table 12</span></a>).</p>
<span id="usecase1-rwrsim"></span><table class="docutils align-center" id="id38">
<caption><span class="caption-number">Table 12 </span><span class="caption-text">- Rare disease prioritization using RWR score. The top 5 is displayed.</span><a class="headerlink" href="#id38" title="Permalink to this table"></a></caption>
<colgroup>
<col style="width: 25%" />
<col style="width: 50%" />
<col style="width: 25%" />
</colgroup>
<thead>
<tr class="row-odd"><th class="head"><p>Nodes (disease IDs)</p></th>
<th class="head"><p>Disease Names</p></th>
<th class="head"><p>RWR scores</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>OMIM:601626</p></td>
<td><p>Leukemia, acute myeloid</p></td>
<td><p>1.68e-04</p></td>
</tr>
<tr class="row-odd"><td><p>OMIM:114500</p></td>
<td><p>Colorectal cancer</p></td>
<td><p>1.61e-04</p></td>
</tr>
<tr class="row-even"><td><p>OMIM:125853</p></td>
<td><p>Diabetes mellitus, noninsulin-dependent</p></td>
<td><p>1.60e-04</p></td>
</tr>
<tr class="row-odd"><td><p>OMIM:114480</p></td>
<td><p>Breast cancer</p></td>
<td><p>1.20e-04</p></td>
</tr>
<tr class="row-even"><td><p>OMIM:211980</p></td>
<td><p>Lung cancer, susceptibility to</p></td>
<td><p>1.16e-04</p></td>
</tr>
</tbody>
</table>
<p>We created a visualisation of the results (<a class="reference internal" href="#usecase1-rwrsimfig"><span class="std std-numref">Fig. 21</span></a>) using Cytoscape <a class="footnote-reference brackets" href="#id23" id="id16" role="doc-noteref"><span class="fn-bracket">[</span>6<span class="fn-bracket">]</span></a>. You can retrieved it in
the cytoscape project called <code class="docutils literal notranslate"><span class="pre">RWR_visualisation.cys</span></code> in <a class="reference external" href="https://github.com/MOohTus/ODAMNet/tree/main/useCases/">GitHub</a>. The <a class="reference internal" href="#usecase1-rwrsimfig"><span class="std std-numref">Fig. 21</span></a> presents the
top 5 of rare disease pathways, ordered by RWR score.</p>
<figure class="align-center" id="id39">
<span id="usecase1-rwrsimfig"></span><a class="reference internal image-reference" href="../../_images/UseCase1_RWR_top5_sim.png"><img alt="useCase1_RWRSimFig" src="../../_images/UseCase1_RWR_top5_sim.png" style="width: 700.0px; height: 489.99999999999994px;" /></a>
<figcaption>
<p><span class="caption-number">Fig. 21 </span><span class="caption-text">: Top 5 of the rare disease prioritized using RWR score using disease-disease similarity network</span><a class="headerlink" href="#id39" title="Permalink to this image"></a></p>
<div class="legend">
<blockquote>
<div><p>Rare disease are in pink triangles. Target genes are in grey and non-target genes are in white. Creation
steps are explained in the <a class="reference internal" href="../network/NetworkVisualisation.html#cytoscape-rwr"><span class="std std-ref">Random walk with restart results visualisation</span></a> section.</p>
</div></blockquote>
</div>
</figcaption>
</figure>
</section>
</section>
</section>
</section>
<section id="overlap-ami-and-rwr-results-comparison">
<h2>Overlap, AMI and RWR results comparison<a class="headerlink" href="#overlap-ami-and-rwr-results-comparison" title="Permalink to this heading"></a></h2>
<p>We compare results obtained with the three different approaches: Overlap analysis, Active Module Identification (AMI)
and Random Walk with Restart (RWR). We used orsum <a class="footnote-reference brackets" href="#id25" id="id17" role="doc-noteref"><span class="fn-bracket">[</span>8<span class="fn-bracket">]</span></a>, a Python package to filter and integrate enrichment analysis
from several analyses. The main result is a heatmap, presented in <a class="reference internal" href="#usecase1-orsum"><span class="std std-numref">Fig. 22</span></a>.</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>orsum.py<span class="w"> </span>--gmt<span class="w"> </span>00_Data/WP_RareDiseases_request_2022_09_07.gmt<span class="w"> </span><span class="se">\</span>
<span class="w">         </span>--files<span class="w"> </span>00_Data/Overlap_D014801.4Orsum<span class="w"> </span>00_Data/DOMINO_D014801.4Orsum<span class="w"> </span>00_Data/RWR_D014801.4Orsum<span class="w"> </span><span class="se">\</span>
<span class="w">         </span>--fileAliases<span class="w"> </span>Overlap<span class="w"> </span>AMI<span class="w"> </span>RWR<span class="w"> </span><span class="se">\</span>
<span class="w">         </span>--maxRepSize<span class="w"> </span><span class="m">0</span><span class="w"> </span><span class="se">\</span>
<span class="w">         </span>--outputFolder<span class="w"> </span>UseCase1_D014801_orsum
</pre></div>
</div>
<figure class="align-center" id="id40">
<span id="usecase1-orsum"></span><a class="reference internal image-reference" href="../../_images/UseCase1_orsum.png"><img alt="useCase1_orsum" src="../../_images/UseCase1_orsum.png" style="width: 733.5px; height: 815.5px;" /></a>
<figcaption>
<p><span class="caption-number">Fig. 22 </span><span class="caption-text">: Overlap analysis (Overlap), Active module identification (AMI) and Random walk with restart (RWR) results integration and comparison using orsum</span><a class="headerlink" href="#id40" title="Permalink to this image"></a></p>
</figcaption>
</figure>
<p>The <code class="docutils literal notranslate"><span class="pre">--maxRepSize</span></code> parameter is set to 0 to consider each term as is own representative term.</p>
<p>Some rare disease pathways are retrieved associated with vitamin A by all the three approaches such as
<em>Malignant pleural mesothelioma</em> or <em>Acute viral myocarditis</em>. Some other rare disease pathways are retrieved associated
with vitamin A only by one (<em>NBIA subtypes pathway</em>) or two (<em>Male infertility</em>) approaches.</p>
</section>
<section id="references">
<h2>References<a class="headerlink" href="#references" title="Permalink to this heading"></a></h2>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id18" role="note">
<span class="label"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id1">1</a>,<a role="doc-backlink" href="#id5">2</a>,<a role="doc-backlink" href="#id6">3</a>)</span>
<p>Ozisik O, Ehrhart F, Evelo C <em>et al.</em>. Overlap of vitamin A and vitamin D target genes with CAKUT-related processes. F1000Research. 2021.</p>
</aside>
<aside class="footnote brackets" id="id19" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id2">2</a><span class="fn-bracket">]</span></span>
<p>Davis AP, Grondin CJ, Johnson RJ <em>et al.</em>. The Comparative Toxicogenomics Database: update 2021. Nucleic acids research. 2021.</p>
</aside>
<aside class="footnote brackets" id="id20" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id3">3</a><span class="fn-bracket">]</span></span>
<p>Martens M, Ammar A, Riutta A <em>et al.</em>. WikiPathways: connecting communities. Nucleic acids research. 2021.</p>
</aside>
<aside class="footnote brackets" id="id21" role="note">
<span class="label"><span class="fn-bracket">[</span>4<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id4">1</a>,<a role="doc-backlink" href="#id8">2</a>)</span>
<p>Pratt D, Chen J, Welker <em>et al.</em>. NDEx, the Network Data Exchange. Cell Systems. 2015.</p>
</aside>
<aside class="footnote brackets" id="id22" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id7">5</a><span class="fn-bracket">]</span></span>
<p>Levi H, Rahmanian N, Elkon R <em>et al.</em>. The DOMINO web-server for active module identification analysis. Bioinformatics. 2022.</p>
</aside>
<aside class="footnote brackets" id="id23" role="note">
<span class="label"><span class="fn-bracket">[</span>6<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id12">1</a>,<a role="doc-backlink" href="#id15">2</a>,<a role="doc-backlink" href="#id16">3</a>)</span>
<p>Shannon P, Markiel A, Ozier O <em>et al.</em>. Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome research. 2003.</p>
</aside>
<aside class="footnote brackets" id="id24" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id13">7</a><span class="fn-bracket">]</span></span>
<p>Baptista A, Gonzalez A &amp; Baudot A. Universal multilayer network exploration by random walk with restart. Communications Physics. 2022.</p>
</aside>
<aside class="footnote brackets" id="id25" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id17">8</a><span class="fn-bracket">]</span></span>
<p>Ozisik O, Térézol M &amp; Baudot A. orsum: a Python package for filtering and comparing enrichment analyses using a simple principle. BMC bioinformatics. 2022.</p>
</aside>
</aside>
</section>
</section>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="../network/NetworkVisualisation.html" class="btn btn-neutral float-left" title="Networks visualisation" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
        <a href="usecase2.html" class="btn btn-neutral float-right" title="Use-case 2: data are provided by the user" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright 2022, Térézol.</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>