<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
  <meta charset="utf-8" /><meta name="generator" content="Docutils 0.18.1: http://docutils.sourceforge.net/" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>Use-case 2: data are provided directly by users &mdash; ODAMNet 1 documentation</title>
      <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/custom.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/tabs.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/design-style.4045f2051d55cab465a707391d5b2007.min.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/custom.css" type="text/css" />
      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" type="text/css" />
  <!--[if lt IE 9]>
    <script src="../../_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
        <script data-url_root="../../" id="documentation_options" src="../../_static/documentation_options.js"></script>
        <script src="../../_static/jquery.js"></script>
        <script src="../../_static/underscore.js"></script>
        <script src="../../_static/_sphinx_javascript_frameworks_compat.js"></script>
        <script src="../../_static/doctools.js"></script>
        <script src="../../_static/sphinx_highlight.js"></script>
        <script src="../../_static/tabs.js"></script>
        <script src="../../_static/design-tabs.js"></script>
    <script src="../../_static/js/theme.js"></script>
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="Input files" href="../formats/Input.html" />
    <link rel="prev" title="Use-case 1: data are extracted automatically with requests" href="usecase1.html" /> 
</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >

          
          
          <a href="../../index.html" class="icon icon-home">
            ODAMNet
          </a>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <p class="caption" role="heading"><span class="caption-text">Quickstart</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../Installation.html">Installation</a></li>
<li class="toctree-l1"><a class="reference internal" href="../Installation.html#usage">Usage</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Approaches</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_overview.html">Overview of the approaches</a></li>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_overlap.html">Overlap analysis</a></li>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_AMI.html">Active Module Identification</a></li>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_RWR.html">Random Walk with Restart</a></li>
<li class="toctree-l1"><a class="reference internal" href="../approaches/Cytoscape.html">Network visualisation</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Use-cases</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="usecase1.html">Use-case 1: data are extracted automatically with requests</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Use-case 2: data are provided directly by users</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#context">Context</a></li>
<li class="toctree-l2"><a class="reference internal" href="#overlap-analysis">Overlap analysis</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#running-overlap-analysis-with-data-provided-by-users">Running overlap analysis with data provided by users</a></li>
<li class="toctree-l3"><a class="reference internal" href="#results-of-overlap-analysis-with-data-provided-by-users">Results of overlap analysis with data provided by users</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#data-provided-by-users-description">Data provided by users description</a></li>
<li class="toctree-l4"><a class="reference internal" href="#overlap-analysis-results">Overlap analysis results</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#active-modules-identification-ami">Active Modules Identification (AMI)</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#running-active-modules-identification-with-data-provided-by-users">Running active modules identification with data provided by users</a></li>
<li class="toctree-l3"><a class="reference internal" href="#results-of-active-module-identification-with-data-provided-by-users">Results of active module identification with data provided by users</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#id11">Data provided by users description</a></li>
<li class="toctree-l4"><a class="reference internal" href="#active-modules-identification-results">Active Modules Identification results</a></li>
<li class="toctree-l4"><a class="reference internal" href="#id13">Overlap analysis results</a></li>
<li class="toctree-l4"><a class="reference internal" href="#visualisation-of-am-results">Visualisation of AM results</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#random-walk-with-restart-rwr">Random Walk with Restart (RWR)</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#running-random-walk-analysis-with-data-provided-by-users">Running Random Walk analysis with data provided by users</a></li>
<li class="toctree-l3"><a class="reference internal" href="#results-of-random-walk-analysis-with-data-provided-by-users">Results of Random Walk analysis with data provided by users</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#pathways-of-interest-network-analysis">Pathways of interest network analysis</a></li>
<li class="toctree-l4"><a class="reference internal" href="#disease-disease-similarity-network">Disease-Disease similarity network</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#references">References</a></li>
</ul>
</li>
</ul>
<p class="caption" role="heading"><span class="caption-text">File formats</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../formats/Input.html">Input files</a></li>
<li class="toctree-l1"><a class="reference internal" href="../formats/Output.html">Output files</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Code description</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../functions/fcts_arguments.html">Input parameters for each approach</a></li>
<li class="toctree-l1"><a class="reference internal" href="../functions/fcts_description.html">Code functions in details</a></li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../index.html">ODAMNet</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="../../index.html" class="icon icon-home" aria-label="Home"></a></li>
      <li class="breadcrumb-item active">Use-case 2: data are provided directly by users</li>
      <li class="wy-breadcrumbs-aside">
            <a href="../../_sources/pages/analysis/usecase2.rst.txt" rel="nofollow"> View page source</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section id="use-case-2-data-are-provided-directly-by-users">
<span id="usecase2"></span><h1>Use-case 2: data are provided directly by users<a class="headerlink" href="#use-case-2-data-are-provided-directly-by-users" title="Permalink to this heading"></a></h1>
<section id="context">
<h2>Context<a class="headerlink" href="#context" title="Permalink to this heading"></a></h2>
<p>In the context of the EJP-RD project, an overlap analysis between genes targeted by vitamins A and D and pathways
related to CAKUT disease was performed (Ozisik <em>et al.</em>, 2021 <a class="footnote-reference brackets" href="#id15" id="id1" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a>).</p>
<p>For the vitamin A analysis, data was retrieved from several sources: target genes come from
the <a class="reference external" href="https://ctdbase.org/">Comparative Toxicogenomics Database (CTD)</a> <a class="footnote-reference brackets" href="#id16" id="id2" role="doc-noteref"><span class="fn-bracket">[</span>2<span class="fn-bracket">]</span></a> and the study of Balmer and
Blomhoff <a class="footnote-reference brackets" href="#id17" id="id3" role="doc-noteref"><span class="fn-bracket">[</span>3<span class="fn-bracket">]</span></a>, and pathways/processes of interest come from WikiPathways (WP) <a class="footnote-reference brackets" href="#id18" id="id4" role="doc-noteref"><span class="fn-bracket">[</span>4<span class="fn-bracket">]</span></a>, Reactome <a class="footnote-reference brackets" href="#id19" id="id5" role="doc-noteref"><span class="fn-bracket">[</span>5<span class="fn-bracket">]</span></a> and Gene Ontology (GO) <a class="footnote-reference brackets" href="#id20" id="id6" role="doc-noteref"><span class="fn-bracket">[</span>6<span class="fn-bracket">]</span></a><sup>,</sup> <a class="footnote-reference brackets" href="#id21" id="id7" role="doc-noteref"><span class="fn-bracket">[</span>7<span class="fn-bracket">]</span></a>.</p>
<p>For this use-case, we used data from this paper:</p>
<ul class="simple">
<li><p>target genes : genes targeted by vitamin A from the study of Balmer and Blomhoff <a class="footnote-reference brackets" href="#id17" id="id8" role="doc-noteref"><span class="fn-bracket">[</span>3<span class="fn-bracket">]</span></a>,</p></li>
<li><p>pathways of interest : list of pathways related to CAKUT disease.</p></li>
</ul>
</section>
<section id="overlap-analysis">
<span id="usecase2-overlap"></span><h2>Overlap analysis<a class="headerlink" href="#overlap-analysis" title="Permalink to this heading"></a></h2>
<p>This approach calculates the overlap between genes targeted by vitamin A, and the pathways of interest
(see <a class="reference internal" href="../approaches/methods_overlap.html"><span class="doc">Overlap analysis</span></a> section for more details).</p>
<section id="running-overlap-analysis-with-data-provided-by-users">
<h3>Running overlap analysis with data provided by users<a class="headerlink" href="#running-overlap-analysis-with-data-provided-by-users" title="Permalink to this heading"></a></h3>
<p>The target genes file is provided by the <code class="docutils literal notranslate"><span class="pre">--targetGenesFile</span></code> parameter and contains a list of genes targeted
by the vitamin A [<a class="reference internal" href="../formats/Input.html#targetgenesfile"><span class="std std-ref">FORMAT</span></a>]. Pathways of interest (here, pathways or processes related to CAKUT disease) are given by the <code class="docutils literal notranslate"><span class="pre">--GMT</span></code>
parameter [<a class="reference internal" href="../formats/Input.html#pathways"><span class="std std-ref">FORMAT</span></a>]. You need to provide the background genes of each source: give a file with the list of
those files using <code class="docutils literal notranslate"><span class="pre">--backgroundFile</span></code> parameter.</p>
<p>Results files are saved into <code class="docutils literal notranslate"><span class="pre">useCases/OutputResults_useCase2/</span></code> folder.</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>odamnet<span class="w"> </span>overlap<span class="w"> </span>--targetGenesFile<span class="w"> </span>useCases/InputData/VitA-Balmer2002-Genes.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                        </span>--GMT<span class="w"> </span>useCases/InputData/PathwaysOfInterest.gmt<span class="w"> </span><span class="se">\</span>
<span class="w">                        </span>--backgroundFile<span class="w"> </span>useCases/InputData/PathwaysOfInterestBackground.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                        </span>--outputPath<span class="w"> </span>useCases/OutputResults_useCase2/
</pre></div>
</div>
<p><code class="docutils literal notranslate"><span class="pre">Overlap_genesList_withpathOfInterest.csv</span></code> file is created. It contains results of the overlap analysis between
target genes and CAKUT related pathways/processes.</p>
<p>For more details about these files, see <a class="reference internal" href="../formats/Output.html"><span class="doc">Output files</span></a> page.</p>
</section>
<section id="results-of-overlap-analysis-with-data-provided-by-users">
<h3>Results of overlap analysis with data provided by users<a class="headerlink" href="#results-of-overlap-analysis-with-data-provided-by-users" title="Permalink to this heading"></a></h3>
<p><em>request on the 07th of September 2022</em></p>
<section id="data-provided-by-users-description">
<h4>Data provided by users description<a class="headerlink" href="#data-provided-by-users-description" title="Permalink to this heading"></a></h4>
<p>Data come from the Ozisik <em>et al.</em>, 2021 <a class="footnote-reference brackets" href="#id15" id="id9" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a> paper. Details of the data are presented in the <a class="reference internal" href="#usecase2dataoverview"><span class="std std-ref">Table 13</span></a>.</p>
<span id="usecase2dataoverview"></span><table class="docutils align-center" id="id22">
<caption><span class="caption-number">Table 13 </span><span class="caption-text">Input data overview</span><a class="headerlink" href="#id22" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"></th>
<th class="head"><p>Number</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Target genes</p></td>
<td><p>521</p></td>
</tr>
<tr class="row-odd"><td><p>Pathways of interest</p></td>
<td><p>12</p></td>
</tr>
<tr class="row-even"><td><p>Sources</p></td>
<td><p>3</p></td>
</tr>
</tbody>
</table>
<p>Pathways of interests come from Reactome and WP and processes of interest come from Biological Process (Gene Ontology).</p>
</section>
<section id="overlap-analysis-results">
<h4>Overlap analysis results<a class="headerlink" href="#overlap-analysis-results" title="Permalink to this heading"></a></h4>
<p>Target genes overlap significantly <code class="docutils literal notranslate"><span class="pre">8</span> <span class="pre">pathways/processes</span></code> related to CAKUT disease (pAdjusted &lt;= 0.05). Results are
presented in <a class="reference internal" href="#usecase2overlap"><span class="std std-ref">Table 14</span></a>.</p>
<span id="usecase2overlap"></span><table class="docutils align-center" id="id23">
<caption><span class="caption-number">Table 14 </span><span class="caption-text">Results of overlap between target genes and CAKUT disease</span><a class="headerlink" href="#id23" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"><p>PathwayIDs</p></th>
<th class="head"><p>PathwayNames</p></th>
<th class="head"><p>pValue</p></th>
<th class="head"><p>IntersectionSize</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p><a class="reference external" href="GO:0072001">GO:0072001</a></p></td>
<td><p>renal system development</p></td>
<td><p>6.65e-18</p></td>
<td><p>43</p></td>
</tr>
<tr class="row-odd"><td><p><a class="reference external" href="GO:0001822">GO:0001822</a></p></td>
<td><p>kidney development</p></td>
<td><p>7.90e-17</p></td>
<td><p>41</p></td>
</tr>
<tr class="row-even"><td><p><a class="reference external" href="GO:0060993">GO:0060993</a></p></td>
<td><p>kidney morphogenesis</p></td>
<td><p>1.92e-12</p></td>
<td><p>20</p></td>
</tr>
<tr class="row-odd"><td><p>WP:WP5053</p></td>
<td><p>Development of ureteric collection …</p></td>
<td><p>2.278e-08</p></td>
<td><p>15</p></td>
</tr>
<tr class="row-even"><td><p>WP:WP4823</p></td>
<td><p>Genes controlling nephrogenesis</p></td>
<td><p>1.11e-04</p></td>
<td><p>10</p></td>
</tr>
<tr class="row-odd"><td><p>PMC5748921-PMC6115658</p></td>
<td><p>CAKUT causal genes</p></td>
<td><p>1.06e-03</p></td>
<td><p>6</p></td>
</tr>
<tr class="row-even"><td><p>WP:WP4830</p></td>
<td><p>GDNF/RET signalling axis</p></td>
<td><p>8.30e-03</p></td>
<td><p>5</p></td>
</tr>
<tr class="row-odd"><td><p>REAC:R-HSA-2022377</p></td>
<td><p>Metabolism of Angiotensinogen to …</p></td>
<td><p>0.04</p></td>
<td><p>3</p></td>
</tr>
</tbody>
</table>
<p>Ozisik <em>et al.,</em> <a class="footnote-reference brackets" href="#id15" id="id10" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a> identified seven pathways/processes related to CAKUT disease. All of them have been retrieved
on this analysis and one more (REAC:R-HSA-2022377).</p>
</section>
</section>
</section>
<section id="active-modules-identification-ami">
<span id="usecase2-ami"></span><h2>Active Modules Identification (AMI)<a class="headerlink" href="#active-modules-identification-ami" title="Permalink to this heading"></a></h2>
<p>This approach identifies Active Modules (AM) through a Protein-Protein Interaction (PPI) network. Then it performs an
overlap analysis between each AM identified and Rare Diseases pathways from WP.
For more detail, see <a class="reference internal" href="../approaches/methods_AMI.html"><span class="doc">Active Module Identification</span></a> section.</p>
<section id="running-active-modules-identification-with-data-provided-by-users">
<h3>Running active modules identification with data provided by users<a class="headerlink" href="#running-active-modules-identification-with-data-provided-by-users" title="Permalink to this heading"></a></h3>
<div class="admonition warning">
<p class="admonition-title">Warning</p>
<p><svg version="1.1" width="2.0em" height="2.0em" class="sd-octicon sd-octicon-alert" viewBox="0 0 24 24" aria-hidden="true"><path d="M13 17.5a1 1 0 11-2 0 1 1 0 012 0zm-.25-8.25a.75.75 0 00-1.5 0v4.5a.75.75 0 001.5 0v-4.5z"></path><path fill-rule="evenodd" d="M9.836 3.244c.963-1.665 3.365-1.665 4.328 0l8.967 15.504c.963 1.667-.24 3.752-2.165 3.752H3.034c-1.926 0-3.128-2.085-2.165-3.752L9.836 3.244zm3.03.751a1 1 0 00-1.732 0L2.168 19.499A1 1 0 003.034 21h17.932a1 1 0 00.866-1.5L12.866 3.994z"></path></svg> Results of DOMINO cannot be reproduced when using the DOMINO’s server.</p>
</div>
<p>As before, users provide target genes (<code class="docutils literal notranslate"><span class="pre">--targetGenesFile</span></code>) and pathways of interest (<code class="docutils literal notranslate"><span class="pre">--GMT</span></code> and <code class="docutils literal notranslate"><span class="pre">--backgroundFile</span></code> - [<a class="reference internal" href="../formats/Input.html#pathways"><span class="std std-ref">FORMAT</span></a>]).</p>
<p>We will identify AM using a Protein-Protein Interaction (PPI) network named <code class="docutils literal notranslate"><span class="pre">PPI_network_2016.sif</span></code> [<a class="reference internal" href="../formats/Input.html#sif"><span class="std std-ref">FORMAT</span></a>].</p>
<p>Results files are saved into <code class="docutils literal notranslate"><span class="pre">useCases/OutputResults_useCase2/</span></code> folder.</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>odamnet<span class="w"> </span>domino<span class="w">  </span>--targetGenesFile<span class="w"> </span>useCases/InputData/VitA-Balmer2002-Genes.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                        </span>--GMT<span class="w"> </span>useCases/InputData/PathwaysOfInterest.gmt<span class="w"> </span><span class="se">\</span>
<span class="w">                        </span>--backgroundFile<span class="w"> </span>useCases/InputData/PathwaysOfInterestBackground.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                        </span>--networkFile<span class="w"> </span>useCases/InputData/PPI_network_2016.sif<span class="w"> </span><span class="se">\</span>
<span class="w">                        </span>--outputPath<span class="w"> </span>useCases/OutputResults_useCase2
</pre></div>
</div>
<p>Several files are generated :</p>
<ul class="simple">
<li><p><code class="docutils literal notranslate"><span class="pre">DOMINO_inputGeneList_D014801.txt</span></code>: genes targeted by vitamin A, used for the AM identification.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">Overlap_AM_*_genesList_withpathOfInterest.csv</span></code>: results of the overlap analysis between target genes and pathways of
interest provided by users. One file per AM.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_activeModulesNetwork.txt</span></code>, <code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_overlapAMresults4Cytoscape.txt</span></code>, <code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_activeModules.txt</span></code>
, <code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_activeModulesNetworkMetrics.txt</span></code> and <code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_signOverlap.txt</span></code>: some statistics are
calculated and saved into files. Theses files are useful for visualisation.</p></li>
</ul>
<p>For more details about these file, see <a class="reference internal" href="../formats/Output.html"><span class="doc">Output files</span></a> page (<a class="reference internal" href="../formats/Output.html#requestoutput"><span class="std std-ref">Request file results</span></a>, <a class="reference internal" href="../formats/Output.html#overlapoutput"><span class="std std-ref">Overlap analysis</span></a>, <a class="reference internal" href="../formats/Output.html#amioutput"><span class="std std-ref">Active Module Identification</span></a>).</p>
</section>
<section id="results-of-active-module-identification-with-data-provided-by-users">
<h3>Results of active module identification with data provided by users<a class="headerlink" href="#results-of-active-module-identification-with-data-provided-by-users" title="Permalink to this heading"></a></h3>
<p><em>request on the 07th of September 2022</em></p>
<section id="id11">
<h4>Data provided by users description<a class="headerlink" href="#id11" title="Permalink to this heading"></a></h4>
<p>Data come from the Ozisik <em>et al.</em>, 2021 <a class="footnote-reference brackets" href="#id15" id="id12" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a> paper. Details of the data are presented in the <a class="reference internal" href="#usecase2dataoverviewami"><span class="std std-ref">Table 15</span></a>.</p>
<span id="usecase2dataoverviewami"></span><table class="docutils align-center" id="id24">
<caption><span class="caption-number">Table 15 </span><span class="caption-text">Input data overview</span><a class="headerlink" href="#id24" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"></th>
<th class="head"><p>Number</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Target genes</p></td>
<td><p>521</p></td>
</tr>
<tr class="row-odd"><td><p>Pathways of interest</p></td>
<td><p>12</p></td>
</tr>
<tr class="row-even"><td><p>Sources</p></td>
<td><p>3</p></td>
</tr>
</tbody>
</table>
<p>Pathways of interests come from Reactome and WP and processes of interest come from Biological Process (Gene Ontology).</p>
</section>
<section id="active-modules-identification-results">
<h4>Active Modules Identification results<a class="headerlink" href="#active-modules-identification-results" title="Permalink to this heading"></a></h4>
<p>Target genes are defined as <strong>active genes</strong> by DOMINO. We give <strong>521 active genes</strong> as input. DOMINO found
<strong>18 Active Modules</strong> (<a class="reference internal" href="#usecase2amiresults"><span class="std std-ref">Table 16</span></a>).</p>
<span id="usecase2amiresults"></span><table class="docutils align-center" id="id25">
<caption><span class="caption-number">Table 16 </span><span class="caption-text">Composition of AMs found by DOMINO</span><a class="headerlink" href="#id25" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"></th>
<th class="head"><p>Min number</p></th>
<th class="head"><p>Max number</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Edges</p></td>
<td><p>5</p></td>
<td><p>157</p></td>
</tr>
<tr class="row-odd"><td><p>Nodes</p></td>
<td><p>5</p></td>
<td><p>79</p></td>
</tr>
<tr class="row-even"><td><p>Active Genes</p></td>
<td><p>3</p></td>
<td><p>21</p></td>
</tr>
</tbody>
</table>
<p><em>See DOMINO_genesList_activeModulesNetworkMetrics.txt file for more details.</em></p>
</section>
<section id="id13">
<h4>Overlap analysis results<a class="headerlink" href="#id13" title="Permalink to this heading"></a></h4>
<p>Then, we perform an overlap analysis between AM and pathways of interest. We found <strong>7 pathways</strong> that are significantly overlapped
by <strong>6 Active Modules</strong> (padjusted &lt;= 0.05).</p>
<span id="usecase2amioverlap"></span><table class="docutils align-center" id="id26">
<caption><span class="caption-number">Table 17 </span><span class="caption-text">Overlap analysis between AM and pathways of interest provided by users</span><a class="headerlink" href="#id26" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"><p>termID</p></th>
<th class="head"><p>termTitle</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p><a class="reference external" href="GO:0001822">GO:0001822</a></p></td>
<td><p>kidney development</p></td>
</tr>
<tr class="row-odd"><td><p><a class="reference external" href="GO:0060993">GO:0060993</a></p></td>
<td><p>kidney morphogenesis</p></td>
</tr>
<tr class="row-even"><td><p><a class="reference external" href="GO:0072001">GO:0072001</a></p></td>
<td><p>renal system development</p></td>
</tr>
<tr class="row-odd"><td><p><strong>REAC:R-HSA-8853659</strong></p></td>
<td><p><strong>RET signaling</strong></p></td>
</tr>
<tr class="row-even"><td><p>WP:WP4823</p></td>
<td><p>Genes controlling nephrogenesis</p></td>
</tr>
<tr class="row-odd"><td><p>WP:WP4830</p></td>
<td><p>GDNF/RET signalling axis</p></td>
</tr>
<tr class="row-even"><td><p>WP:WP5053</p></td>
<td><p>Development of ureteric collection system</p></td>
</tr>
</tbody>
</table>
<p>We found a pathway that was not found with the overlap approach (<strong>REAC:R-HSA-8853659</strong>).</p>
</section>
<section id="visualisation-of-am-results">
<h4>Visualisation of AM results<a class="headerlink" href="#visualisation-of-am-results" title="Permalink to this heading"></a></h4>
<p>We visualised the result using a network representation (<a class="reference internal" href="#dominousage2fig"><span class="std std-numref">Fig. 13</span></a>). To know how to create this figure,
see the <a class="reference internal" href="../approaches/Cytoscape.html#networkami"><span class="std std-ref">Active Modules Identification network visualisation</span></a> section.</p>
<figure class="align-center" id="id27">
<span id="dominousage2fig"></span><img alt="usecase2 AMI" src="../../_images/useCase2_DOMINO_network.png" />
<figcaption>
<p><span class="caption-number">Fig. 13 </span><span class="caption-text">: Network visualisation of Active modules which are enriched in CAKUT disease pathways</span><a class="headerlink" href="#id27" title="Permalink to this image"></a></p>
</figcaption>
</figure>
<p>Some network are enriched with the same pathways whereas other contain genes involved in different pathways. Target genes
(i.e. active genes, grey spheres) could be part of pathways as non-target genes (white spheres).</p>
</section>
</section>
</section>
<section id="random-walk-with-restart-rwr">
<span id="usecase2-rwr"></span><h2>Random Walk with Restart (RWR)<a class="headerlink" href="#random-walk-with-restart-rwr" title="Permalink to this heading"></a></h2>
<p>The third approach, Random Walk with Restart (RWR), is applied into two different multilayer compositions:</p>
<ol class="arabic simple">
<li><p>Multiplex (PPI + Complex + Reactome) and pathways of interest network connected to genes nodes</p></li>
<li><p>Multiplex (PPI + Complex + Reactome) and Disease-Disease similarity network linked with a bipartite</p></li>
</ol>
<p><em>For more details about RWR, see</em> <a class="reference internal" href="../approaches/methods_RWR.html"><span class="doc">Random Walk with Restart</span></a>.</p>
<section id="running-random-walk-analysis-with-data-provided-by-users">
<h3>Running Random Walk analysis with data provided by users<a class="headerlink" href="#running-random-walk-analysis-with-data-provided-by-users" title="Permalink to this heading"></a></h3>
<div class="line-block">
<div class="line">To know how to create the pathways of interest network: see <a class="reference internal" href="../approaches/methods_RWR.html#pathwaysofinterestnet"><span class="std std-ref">Pathways of interest network</span></a>.</div>
<div class="line">To know how to create the disease-disease similarity network: see <a class="reference internal" href="../approaches/methods_RWR.html#ddnet"><span class="std std-ref">Disease-disease similarity network</span></a>.</div>
</div>
<p>Whatever the network used, target genes file is provided by users using <code class="docutils literal notranslate"><span class="pre">--targetGenesFile</span></code> [<a class="reference internal" href="../formats/Input.html#targetgenesfile"><span class="std std-ref">FORMAT</span></a>].</p>
<p>MultiXrank needs a configuration file (<code class="docutils literal notranslate"><span class="pre">--configPath</span></code>) and the networks path (<code class="docutils literal notranslate"><span class="pre">--networksPath</span></code>). We run the analysis with
default parameters.</p>
<p>The target genes are set as seeds for the walk and saved into a file <code class="docutils literal notranslate"><span class="pre">--seedsFile</span> <span class="pre">examples/InputData/seeds.txt</span></code>.
You need to give the SIF name (<code class="docutils literal notranslate"><span class="pre">--sifFileName</span></code>) to save the network results and the top number of results too
(<code class="docutils literal notranslate"><span class="pre">--top</span> <span class="pre">10</span></code>).</p>
<p>Results files are saved into <code class="docutils literal notranslate"><span class="pre">useCases/OutputResults_useCase2/</span></code> folder.</p>
<p>If you need more details about the input format files, see <a class="reference internal" href="../formats/Input.html#gr"><span class="std std-ref">Random Walk with Restart input network</span></a> and <a class="reference internal" href="../formats/Input.html#configfile"><span class="std std-ref">Configuration file</span></a> parts.</p>
<div class="admonition tip">
<p class="admonition-title">Tip</p>
<p>Whatever the networks used, the <strong>command line is the same</strong>. But you have to <strong>change</strong> the network name inside the
<strong>configuration file</strong>.</p>
<div class="sphinx-tabs docutils container">
<div aria-label="Tabbed content" class="closeable" role="tablist"><button aria-controls="panel-0-UGF0aHdheXMgb2YgaW50ZXJlc3QgbmV0d29yaw==" aria-selected="true" class="sphinx-tabs-tab group-tab" id="tab-0-UGF0aHdheXMgb2YgaW50ZXJlc3QgbmV0d29yaw==" name="UGF0aHdheXMgb2YgaW50ZXJlc3QgbmV0d29yaw==" role="tab" tabindex="0">Pathways of interest network</button><button aria-controls="panel-0-RGlzZWFzZS1EaXNlYXNlIHNpbWlsYXJpdHkgbmV0d29yaw==" aria-selected="false" class="sphinx-tabs-tab group-tab" id="tab-0-RGlzZWFzZS1EaXNlYXNlIHNpbWlsYXJpdHkgbmV0d29yaw==" name="RGlzZWFzZS1EaXNlYXNlIHNpbWlsYXJpdHkgbmV0d29yaw==" role="tab" tabindex="-1">Disease-Disease similarity network</button></div><div aria-labelledby="tab-0-UGF0aHdheXMgb2YgaW50ZXJlc3QgbmV0d29yaw==" class="sphinx-tabs-panel group-tab" id="panel-0-UGF0aHdheXMgb2YgaW50ZXJlc3QgbmV0d29yaw==" name="UGF0aHdheXMgb2YgaW50ZXJlc3QgbmV0d29yaw==" role="tabpanel" tabindex="0"><div class="highlight-bash notranslate"><div class="highlight"><pre><span></span><span class="w"> </span>multiplex:
<span class="w">     </span><span class="m">1</span>:
<span class="w">         </span>layers:
<span class="w">             </span>-<span class="w"> </span>multiplex/1/Complexes_Nov2020.gr
<span class="w">             </span>-<span class="w"> </span>multiplex/1/PPI_Jan2021.gr
<span class="w">             </span>-<span class="w"> </span>multiplex/1/Reactome_Nov2020.gr
<span class="w">     </span><span class="m">2</span>:
<span class="w">         </span>layers:
<span class="hll"><span class="w">             </span>-<span class="w"> </span>multiplex/2/pathwaysOfInterestNetwork_fromPaper.sif
</span><span class="w"> </span>bipartite:
<span class="hll"><span class="w">     </span>bipartite/Bipartite_pathOfInterest_geneSymbols_fromPaper.tsv:
</span><span class="w">         </span>source:<span class="w"> </span><span class="m">2</span>
<span class="w">         </span>target:<span class="w"> </span><span class="m">1</span>
<span class="w"> </span>seed:
<span class="w">     </span>seeds.txt
</pre></div>
</div>
</div><div aria-labelledby="tab-0-RGlzZWFzZS1EaXNlYXNlIHNpbWlsYXJpdHkgbmV0d29yaw==" class="sphinx-tabs-panel group-tab" hidden="true" id="panel-0-RGlzZWFzZS1EaXNlYXNlIHNpbWlsYXJpdHkgbmV0d29yaw==" name="RGlzZWFzZS1EaXNlYXNlIHNpbWlsYXJpdHkgbmV0d29yaw==" role="tabpanel" tabindex="0"><div class="highlight-bash notranslate"><div class="highlight"><pre><span></span><span class="w"> </span>multiplex:
<span class="w">     </span><span class="m">1</span>:
<span class="w">         </span>layers:
<span class="w">             </span>-<span class="w"> </span>multiplex/1/Complexes_Nov2020.gr
<span class="w">             </span>-<span class="w"> </span>multiplex/1/PPI_Jan2021.gr
<span class="w">             </span>-<span class="w"> </span>multiplex/1/Reactome_Nov2020.gr
<span class="w">     </span><span class="m">2</span>:
<span class="w">         </span>layers:
<span class="hll"><span class="w">             </span>-<span class="w"> </span>multiplex/2/DiseaseSimilarity_network_2022_06_11.txt
</span><span class="w"> </span>bipartite:
<span class="hll"><span class="w">     </span>bipartite/Bipartite_genes_to_OMIM_2022_09_27.txt:
</span><span class="w">         </span>source:<span class="w"> </span><span class="m">2</span>
<span class="w">         </span>target:<span class="w"> </span><span class="m">1</span>
<span class="w"> </span>seed:
<span class="w">     </span>seeds.txt
</pre></div>
</div>
</div></div>
</div>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>odamnet<span class="w"> </span>multixrank<span class="w">  </span>--targetGenesFile<span class="w"> </span>useCases/InputData/VitA-Balmer2002-Genes.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                            </span>--configPath<span class="w"> </span>useCases/InputData/config_minimal_useCase2.yml<span class="w"> </span><span class="se">\</span>
<span class="w">                            </span>--networksPath<span class="w"> </span>useCases/InputData/<span class="w"> </span><span class="se">\</span>
<span class="w">                            </span>--seedsFile<span class="w"> </span>useCases/InputData/seeds.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                            </span>--sifFileName<span class="w"> </span>resultsNetwork_useCase2.sif<span class="w"> </span><span class="se">\</span>
<span class="w">                            </span>--top<span class="w"> </span><span class="m">10</span><span class="w"> </span><span class="se">\</span>
<span class="w">                            </span>--outputPath<span class="w"> </span>useCases/OutputResults_useCase2/
</pre></div>
</div>
<p>Several files are generated into <code class="docutils literal notranslate"><span class="pre">RWR_genesList/</span></code> folder:</p>
<blockquote>
<div><ul class="simple">
<li><p><code class="docutils literal notranslate"><span class="pre">config_minimal_useCase2.yml</span></code> and <code class="docutils literal notranslate"><span class="pre">seeds.txt</span></code>: copies of the input files</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">multiplex_1.tsv</span></code> and <code class="docutils literal notranslate"><span class="pre">multiplex_2.tsv</span></code>: score for each feature. 1 corresponds to the multiplex and 2 to
the disease network (depends of the network folder name).</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">resultsNetwork_useCase2.sif</span></code>: SIF file that contains the network result</p></li>
</ul>
</div></blockquote>
<p>For more details about these file, see <a class="reference internal" href="../formats/Output.html"><span class="doc">Output files</span></a> page.</p>
</section>
<section id="results-of-random-walk-analysis-with-data-provided-by-users">
<h3>Results of Random Walk analysis with data provided by users<a class="headerlink" href="#results-of-random-walk-analysis-with-data-provided-by-users" title="Permalink to this heading"></a></h3>
<p>We use the default parameters, whatever the networks used. For reminder, we have <strong>521 target genes</strong> provided by users.</p>
<section id="pathways-of-interest-network-analysis">
<h4>Pathways of interest network analysis<a class="headerlink" href="#pathways-of-interest-network-analysis" title="Permalink to this heading"></a></h4>
<p><em>In this part, we present results found for the first multiplex composition: multiplex + pathways of interest.</em></p>
<p>First, target genes are used as seed to start the walk: <code class="docutils literal notranslate"><span class="pre">483/521</span></code> genes are used.</p>
<p>The gene with the highest score is <code class="docutils literal notranslate"><span class="pre">ASMT</span></code> with <code class="docutils literal notranslate"><span class="pre">score</span> <span class="pre">=</span> <span class="pre">0.0006682735081574565</span></code> (it’s a seed). This score helps
us to select a list of pathways. All pathways with a score bigger than this score are extracted and considered as connected
with target genes (i.e. seeds).</p>
<p>According this highest score, <strong>4 pathways</strong> are selected (<a class="reference internal" href="#usecase2-pathwaysrwr"><span class="std std-ref">Table 18</span></a>).</p>
<span id="usecase2-pathwaysrwr"></span><table class="docutils align-center" id="id28">
<caption><span class="caption-number">Table 18 </span><span class="caption-text">Pathways linked to target genes</span><a class="headerlink" href="#id28" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"><p>node</p></th>
<th class="head"><p>pathway</p></th>
<th class="head"><p>score</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p><a class="reference external" href="GO:0072001">GO:0072001</a></p></td>
<td><p>renal system development</p></td>
<td><p>0.002101</p></td>
</tr>
<tr class="row-odd"><td><p><a class="reference external" href="GO:0001822">GO:0001822</a></p></td>
<td><p>kidney development</p></td>
<td><p>0.001847</p></td>
</tr>
<tr class="row-even"><td><p><strong>REAC:R-HSA-195721</strong></p></td>
<td><p><strong>Signaling by WNT</strong></p></td>
<td><p><strong>0.001660</strong></p></td>
</tr>
<tr class="row-odd"><td><p><strong>REAC:R-HSA-157118</strong></p></td>
<td><p><strong>Signaling by NOTCH</strong></p></td>
<td><p><strong>0.001140</strong></p></td>
</tr>
</tbody>
</table>
<p>Two pathways not found with the previous approaches, are link to target genes: <code class="docutils literal notranslate"><span class="pre">REAC:R-HSA-195721</span></code> and <code class="docutils literal notranslate"><span class="pre">REAC:R-HSA-157118</span></code>.
You can visualise the results with a network as shown on the <a class="reference internal" href="#usecase2-pathwaysnetworkrwr"><span class="std std-numref">Fig. 14</span></a>.</p>
<figure class="align-center" id="id29">
<span id="usecase2-pathwaysnetworkrwr"></span><img alt="usecase 2 pathwaysNetworkRWR" src="../../_images/RWR_pathwaysNet_useCase2.png" />
<figcaption>
<p><span class="caption-number">Fig. 14 </span><span class="caption-text">: Results from RWR through the molecular multilayer and pathways of interest network</span><a class="headerlink" href="#id29" title="Permalink to this image"></a></p>
<div class="legend">
<blockquote>
<div><p>Disease pathways are represented by triangle nodes in pink, genes are represented by white nodes and target genes by grey nodes.</p>
</div></blockquote>
</div>
</figcaption>
</figure>
</section>
<section id="disease-disease-similarity-network">
<h4>Disease-Disease similarity network<a class="headerlink" href="#disease-disease-similarity-network" title="Permalink to this heading"></a></h4>
<p><em>In this part, we present results found for the second multiplex composition: multiplex + disease-disease network.</em></p>
<p>First, target genes are used as seed to start the walk: <code class="docutils literal notranslate"><span class="pre">483/521</span></code> genes are used.</p>
<p>We selected the top 10 of diseases (<a class="reference internal" href="#usecase2-diseasesrwr"><span class="std std-ref">Table 19</span></a>).</p>
<span id="usecase2-diseasesrwr"></span><table class="docutils align-center" id="id30">
<caption><span class="caption-number">Table 19 </span><span class="caption-text">Diseases linked to target genes</span><a class="headerlink" href="#id30" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"><p>node</p></th>
<th class="head"><p>Diseases</p></th>
<th class="head"><p>score</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>OMIM:178500</p></td>
<td><p>Pulmonary fibrosis, idiopathic</p></td>
<td><p>0.000334</p></td>
</tr>
<tr class="row-odd"><td><p>OMIM:125853</p></td>
<td><p>Diabetes mellitus, noninsulin-dependent</p></td>
<td><p>0.000301</p></td>
</tr>
<tr class="row-even"><td><p>OMIM:215600</p></td>
<td><p>Cirrhosis, familial</p></td>
<td><p>0.000255</p></td>
</tr>
<tr class="row-odd"><td><p>OMIM:613659</p></td>
<td><p>Gastric cancer, somatic</p></td>
<td><p>0.000235</p></td>
</tr>
<tr class="row-even"><td><p>OMIM:211980</p></td>
<td><p>Lung cancer, susceptibility to</p></td>
<td><p>0.000230</p></td>
</tr>
<tr class="row-odd"><td><p>OMIM:104300</p></td>
<td><p>Alzheimer disease</p></td>
<td><p>0.000224</p></td>
</tr>
<tr class="row-even"><td><p>OMIM:168600</p></td>
<td><p>Parkinson disease, late-onset</p></td>
<td><p>0.000192</p></td>
</tr>
<tr class="row-odd"><td><p>OMIM:601859</p></td>
<td><p>Autoimmune lymphoproliferative syndrome</p></td>
<td><p>0.000182</p></td>
</tr>
<tr class="row-even"><td><p>OMIM:601665</p></td>
<td><p>OBESITY</p></td>
<td><p>0.000181</p></td>
</tr>
<tr class="row-odd"><td><p>OMIM:171300</p></td>
<td><p>PHEOCHROMOCYTOMA</p></td>
<td><p>0.000145</p></td>
</tr>
</tbody>
</table>
<p>You can represent the results with a network as shown in the <a class="reference internal" href="#usecase2-simnetworkrwr"><span class="std std-numref">Fig. 15</span></a>.</p>
<figure class="align-center" id="id31">
<span id="usecase2-simnetworkrwr"></span><img alt="usecase 2 simNetworkRWR" src="../../_images/RWR_pathwaysNet_useCase2_simNet.png" />
<figcaption>
<p><span class="caption-number">Fig. 15 </span><span class="caption-text">: Results from RWR through the molecular multilayer and disease-disease similarity network</span><a class="headerlink" href="#id31" title="Permalink to this image"></a></p>
<div class="legend">
<blockquote>
<div><p>Diseases are represented by triangle pink nodes, genes are represented by white nodes and target genes by grey nodes.</p>
</div></blockquote>
</div>
</figcaption>
</figure>
<p>Rare disease pathways identified</p>
<hr class="docutils" />
<p>Approaches give us a list of CAKUT pathways significantly connected to vitamin A target genes. To easily compare results,
we use orsum <a class="footnote-reference brackets" href="#id16" id="id14" role="doc-noteref"><span class="fn-bracket">[</span>2<span class="fn-bracket">]</span></a>. Results are displayed into a heatmap in the <a class="reference internal" href="#usecase2-orsum"><span class="std std-numref">Fig. 16</span></a>.</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>orsum.py<span class="w">    </span>--gmt<span class="w"> </span>00_Data/hsapiens_background.gmt<span class="w"> </span><span class="se">\</span>
<span class="w">            </span>--files<span class="w"> </span>Overlap_genesList_withpathOfInterest.4Orsum<span class="w"> </span>DOMINO_genesList_signOverlap.4Orsum<span class="w"> </span>pathwaysResults.4Orsum<span class="w"> </span><span class="se">\</span>
<span class="w">            </span>--fileAliases<span class="w"> </span>Overlap<span class="w"> </span>DOMINO<span class="w"> </span>multiXrank<span class="w"> </span><span class="se">\</span>
<span class="w">            </span>--outputFolder<span class="w"> </span>useCase2Comparison/
</pre></div>
</div>
<figure class="align-center" id="id32">
<span id="usecase2-orsum"></span><img alt="usecase2 orsum" src="../../_images/useCase2_orsum.png" />
<figcaption>
<p><span class="caption-number">Fig. 16 </span><span class="caption-text">: Comparison of use-case 2 results using orsum</span><a class="headerlink" href="#id32" title="Permalink to this image"></a></p>
</figcaption>
</figure>
</section>
</section>
</section>
<section id="references">
<h2>References<a class="headerlink" href="#references" title="Permalink to this heading"></a></h2>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id15" role="note">
<span class="label"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id1">1</a>,<a role="doc-backlink" href="#id9">2</a>,<a role="doc-backlink" href="#id10">3</a>,<a role="doc-backlink" href="#id12">4</a>)</span>
<p>Ozisik, O., Ehrhart, F., Evelo, C. T., Mantovani, A., &amp; Baudot, A. (2021). Overlap of vitamin A and vitamin D target genes with CAKUT-related processes. F1000Research, 10.</p>
</aside>
<aside class="footnote brackets" id="id16" role="note">
<span class="label"><span class="fn-bracket">[</span>2<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id2">1</a>,<a role="doc-backlink" href="#id14">2</a>)</span>
<p>Davis AP, Grondin CJ, Johnson RJ, Sciaky D, Wiegers J, Wiegers TC, Mattingly CJ The Comparative Toxicogenomics Database: update 2021. Nucleic Acids Res. 2021.</p>
</aside>
<aside class="footnote brackets" id="id17" role="note">
<span class="label"><span class="fn-bracket">[</span>3<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id3">1</a>,<a role="doc-backlink" href="#id8">2</a>)</span>
<p>Balmer, J. E., &amp; Blomhoff, R. (2002). Gene expression regulation by retinoic acid. Journal of lipid research, 43(11), 1773-1808.</p>
</aside>
<aside class="footnote brackets" id="id18" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id4">4</a><span class="fn-bracket">]</span></span>
<p>Martens, M., Ammar, A., Riutta, A., Waagmeester, A., Slenter, D. N., Hanspers, K., … &amp; Kutmon, M. (2021). WikiPathways: connecting communities. Nucleic acids research, 49(D1), D613-D621.</p>
</aside>
<aside class="footnote brackets" id="id19" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id5">5</a><span class="fn-bracket">]</span></span>
<p>Jassal, B., Matthews, L., Viteri, G., Gong, C., Lorente, P., Fabregat, A., … &amp; D’Eustachio, P. (2020). The reactome pathway knowledgebase. Nucleic acids research, 48(D1), D498-D503.</p>
</aside>
<aside class="footnote brackets" id="id20" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id6">6</a><span class="fn-bracket">]</span></span>
<p>Ashburner et al. Gene ontology: tool for the unification of biology. Nat Genet. May 2000;25(1):25-9</p>
</aside>
<aside class="footnote brackets" id="id21" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id7">7</a><span class="fn-bracket">]</span></span>
<p>The Gene Ontology resource: enriching a GOld mine. Nucleic Acids Res. Jan 2021;49(D1):D325-D334</p>
</aside>
</aside>
</section>
</section>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="usecase1.html" class="btn btn-neutral float-left" title="Use-case 1: data are extracted automatically with requests" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
        <a href="../formats/Input.html" class="btn btn-neutral float-right" title="Input files" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright 2022, Térézol.</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>