<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
  <meta charset="utf-8" /><meta name="generator" content="Docutils 0.18.1: http://docutils.sourceforge.net/" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>Use-case 2: data are provided by the user &mdash; ODAMNet 1 documentation</title>
      <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/custom.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/tabs.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/design-style.4045f2051d55cab465a707391d5b2007.min.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/custom.css" type="text/css" />
      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" type="text/css" />
  <!--[if lt IE 9]>
    <script src="../../_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
        <script data-url_root="../../" id="documentation_options" src="../../_static/documentation_options.js"></script>
        <script src="../../_static/jquery.js"></script>
        <script src="../../_static/underscore.js"></script>
        <script src="../../_static/_sphinx_javascript_frameworks_compat.js"></script>
        <script src="../../_static/doctools.js"></script>
        <script src="../../_static/sphinx_highlight.js"></script>
        <script src="../../_static/tabs.js"></script>
        <script src="../../_static/design-tabs.js"></script>
    <script src="../../_static/js/theme.js"></script>
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="Input files" href="../formats/Input.html" />
    <link rel="prev" title="Use-case 1: data are retrieved automatically by queries" href="usecase1.html" /> 
</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >

          
          
          <a href="../../index.html" class="icon icon-home">
            ODAMNet
          </a>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <p class="caption" role="heading"><span class="caption-text">Quickstart</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../Installation.html">Installation</a></li>
<li class="toctree-l1"><a class="reference internal" href="../Installation.html#usage">Usage</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Approaches</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_overview.html">Overview of the approaches</a></li>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_overlap.html">Overlap analysis</a></li>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_AMI.html">Active Module Identification</a></li>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_RWR.html">Random Walk with Restart</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Networks</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../network/NetworkDownloading.html">Networks downloading</a></li>
<li class="toctree-l1"><a class="reference internal" href="../network/NetworkCreation.html">Networks creation</a></li>
<li class="toctree-l1"><a class="reference internal" href="../network/NetworkUsed.html">Networks used</a></li>
<li class="toctree-l1"><a class="reference internal" href="../network/NetworkVisualisation.html">Networks visualisation</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Use-cases</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="usecase1.html">Use-case 1: data are retrieved automatically by queries</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Use-case 2: data are provided by the user</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#context">Context</a></li>
<li class="toctree-l2"><a class="reference internal" href="#overlap-analysis">Overlap analysis</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#running-overlap-analysis-with-data-provided-by-users">Running overlap analysis with data provided by users</a></li>
<li class="toctree-l3"><a class="reference internal" href="#results-of-overlap-analysis-with-data-provided-by-users">Results of overlap analysis with data provided by users</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#data-provided-by-users-overview">Data provided by users overview</a></li>
<li class="toctree-l4"><a class="reference internal" href="#overlap-analysis-results">Overlap analysis results</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#active-modules-identification-ami">Active Modules Identification (AMI)</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#running-active-modules-identification-with-data-provided-by-users">Running active modules identification with data provided by users</a></li>
<li class="toctree-l3"><a class="reference internal" href="#results-of-active-module-identification-with-data-provided-by-users">Results of active module identification with data provided by users</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#id16">Data provided by users overview</a></li>
<li class="toctree-l4"><a class="reference internal" href="#active-modules-identification-results">Active Modules Identification results</a></li>
<li class="toctree-l4"><a class="reference internal" href="#id24">Overlap analysis results</a></li>
<li class="toctree-l4"><a class="reference internal" href="#visualisation-of-active-module-identification-results">Visualisation of active module identification results</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#random-walk-with-restart-rwr">Random Walk with Restart (RWR)</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#running-random-walk-analysis-with-data-provided-by-users">Running Random Walk analysis with data provided by users</a></li>
<li class="toctree-l3"><a class="reference internal" href="#results-of-random-walk-analysis-with-data-provided-by-users">Results of Random Walk analysis with data provided by users</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#id29">Data provided by users overview</a></li>
<li class="toctree-l4"><a class="reference internal" href="#multilayer-1-pathways-processes-related-to-cakut-network">Multilayer 1 : Pathways/processes related to CAKUT network</a></li>
<li class="toctree-l4"><a class="reference internal" href="#disease-disease-similarity-network">Disease-Disease similarity network</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#rare-disease-pathways-identified">Rare disease pathways identified</a></li>
<li class="toctree-l2"><a class="reference internal" href="#references">References</a></li>
</ul>
</li>
</ul>
<p class="caption" role="heading"><span class="caption-text">File formats</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../formats/Input.html">Input files</a></li>
<li class="toctree-l1"><a class="reference internal" href="../formats/Output.html">Output files</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Code description</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../functions/fcts_arguments.html">Input parameters for each approach</a></li>
<li class="toctree-l1"><a class="reference internal" href="../functions/fcts_description.html">Code functions in details</a></li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../index.html">ODAMNet</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="../../index.html" class="icon icon-home" aria-label="Home"></a></li>
      <li class="breadcrumb-item active">Use-case 2: data are provided by the user</li>
      <li class="wy-breadcrumbs-aside">
            <a href="../../_sources/pages/analysis/usecase2.rst.txt" rel="nofollow"> View page source</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section id="use-case-2-data-are-provided-by-the-user">
<span id="usecase2"></span><h1>Use-case 2: data are provided by the user<a class="headerlink" href="#use-case-2-data-are-provided-by-the-user" title="Permalink to this heading"></a></h1>
<section id="context">
<h2>Context<a class="headerlink" href="#context" title="Permalink to this heading"></a></h2>
<p>In Ozisik <em>et al.</em>, 2021 <a class="footnote-reference brackets" href="#id33" id="id1" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a> paper, an overlap analysis is performed between vitamin A and D target genes and processes
related to Congenital Anomalies of the Kidney and Urinary Tract (CAKUT).</p>
<p>In this use-case, we illustrate how to use ODAMNet with data provided by user. We use data from Ozisik <em>et al.</em>,
2021 <a class="footnote-reference brackets" href="#id33" id="id2" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a> paper. Results are compared with those found in the Ozisik <em>et al.</em>, 2021 <a class="footnote-reference brackets" href="#id33" id="id3" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a> paper.</p>
<p><strong>Vitamin A target genes</strong> are coming from Balmer and Blomhoff <a class="footnote-reference brackets" href="#id35" id="id4" role="doc-noteref"><span class="fn-bracket">[</span>3<span class="fn-bracket">]</span></a> paper and <strong>pathways/processes</strong> related to CAKUT
are coming from <a class="reference external" href="https://www.wikipathways.org/">WikiPathways</a> <a class="footnote-reference brackets" href="#id36" id="id5" role="doc-noteref"><span class="fn-bracket">[</span>4<span class="fn-bracket">]</span></a>, <a class="reference external" href="https://reactome.org/">Reactome</a> <a class="footnote-reference brackets" href="#id37" id="id6" role="doc-noteref"><span class="fn-bracket">[</span>5<span class="fn-bracket">]</span></a> and <a class="reference external" href="http://geneontology.org/">Gene Ontology</a> (GO) <a class="footnote-reference brackets" href="#id38" id="id7" role="doc-noteref"><span class="fn-bracket">[</span>6<span class="fn-bracket">]</span></a><sup>,</sup> <a class="footnote-reference brackets" href="#id39" id="id8" role="doc-noteref"><span class="fn-bracket">[</span>7<span class="fn-bracket">]</span></a>. Input data are available in <a class="reference external" href="https://github.com/MOohTus/ODAMNet/tree/main/useCases/InputData">GitHub</a>.</p>
</section>
<section id="overlap-analysis">
<span id="usecase2-overlap"></span><h2>Overlap analysis<a class="headerlink" href="#overlap-analysis" title="Permalink to this heading"></a></h2>
<p>This approach calculates the overlap between genes targeted by vitamin A, and the pathways/processes related to CAKUT.</p>
<p class="italic">For more details, see <a class="reference internal" href="../approaches/methods_overlap.html"><span class="doc">Overlap analysis</span></a> section.</p>
<section id="running-overlap-analysis-with-data-provided-by-users">
<h3>Running overlap analysis with data provided by users<a class="headerlink" href="#running-overlap-analysis-with-data-provided-by-users" title="Permalink to this heading"></a></h3>
<p>Target genes file is given using the <code class="docutils literal notranslate"><span class="pre">--targetGenesFile</span></code> parameter. It contains the list of vitamin A target genes
[<a class="reference internal" href="../formats/Input.html#targetgenesfile"><span class="std std-ref">FORMAT</span></a>].</p>
<p>Pathways/processes related to CAKUT are given using the <code class="docutils literal notranslate"><span class="pre">--GMT</span></code> parameter [<a class="reference internal" href="../formats/Input.html#pathways"><span class="std std-ref">FORMAT</span></a>]. Background genes
of each source of pathways/processes is required (<code class="docutils literal notranslate"><span class="pre">--backgroundFile</span></code> parameter). The file contains the list of
background files.</p>
<p>Results files are saved into <code class="docutils literal notranslate"><span class="pre">useCases/OutputResults_useCase2/</span></code> folder.</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>odamnet<span class="w"> </span>overlap<span class="w"> </span>--targetGenesFile<span class="w"> </span>useCases/InputData/VitA-Balmer2002-Genes.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--GMT<span class="w"> </span>useCases/InputData/PathwaysOfInterest.gmt<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--backgroundFile<span class="w"> </span>useCases/InputData/PathwaysOfInterestBackground.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--outputPath<span class="w"> </span>useCases/OutputResults_useCase2/
</pre></div>
</div>
<p><code class="docutils literal notranslate"><span class="pre">Overlap_genesList_withpathOfInterest.csv</span></code> file is created. It contains results of the overlap analysis between
target genes and CAKUT related pathways/processes.</p>
<p class="italic">For more details about this output file, see <a class="reference internal" href="../formats/Output.html"><span class="doc">Output files</span></a> page.</p>
</section>
<section id="results-of-overlap-analysis-with-data-provided-by-users">
<h3>Results of overlap analysis with data provided by users<a class="headerlink" href="#results-of-overlap-analysis-with-data-provided-by-users" title="Permalink to this heading"></a></h3>
<section id="data-provided-by-users-overview">
<h4>Data provided by users overview<a class="headerlink" href="#data-provided-by-users-overview" title="Permalink to this heading"></a></h4>
<p>Target genes are coming from Balmer and Blomhoff <a class="footnote-reference brackets" href="#id35" id="id9" role="doc-noteref"><span class="fn-bracket">[</span>3<span class="fn-bracket">]</span></a>. Pathways of interests are coming from Reactome <a class="footnote-reference brackets" href="#id37" id="id10" role="doc-noteref"><span class="fn-bracket">[</span>5<span class="fn-bracket">]</span></a> and
WikiPathways <a class="footnote-reference brackets" href="#id36" id="id11" role="doc-noteref"><span class="fn-bracket">[</span>4<span class="fn-bracket">]</span></a> and processes of interest are coming from Biological Process (GO) <a class="footnote-reference brackets" href="#id38" id="id12" role="doc-noteref"><span class="fn-bracket">[</span>6<span class="fn-bracket">]</span></a><sup>,</sup> <a class="footnote-reference brackets" href="#id39" id="id13" role="doc-noteref"><span class="fn-bracket">[</span>7<span class="fn-bracket">]</span></a>.</p>
<p>Details of input data are presented in the <a class="reference internal" href="#usecase2dataoverview"><span class="std std-ref">Table 13</span></a>.</p>
<span id="usecase2dataoverview"></span><table class="docutils align-center" id="id43">
<caption><span class="caption-number">Table 13 </span><span class="caption-text">Overview of data provided by users</span><a class="headerlink" href="#id43" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"></th>
<th class="head"><p>Number</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Vitamin A target genes</p></td>
<td><p>521</p></td>
</tr>
<tr class="row-odd"><td><p>Pathways/processes related to CAKUT</p></td>
<td><p>13</p></td>
</tr>
<tr class="row-even"><td><p>Pathways/processes sources</p></td>
<td><p>3</p></td>
</tr>
</tbody>
</table>
</section>
<section id="overlap-analysis-results">
<h4>Overlap analysis results<a class="headerlink" href="#overlap-analysis-results" title="Permalink to this heading"></a></h4>
<p>ODAMNet found significant overlap between target genes and <code class="docutils literal notranslate"><span class="pre">7</span> <span class="pre">pathways/processes</span></code> related to CAKUT disease (pAdjusted &lt;= 0.05).
Results are presented in the <a class="reference internal" href="#usecase2overlap"><span class="std std-ref">Table 14</span></a>.</p>
<span id="usecase2overlap"></span><table class="docutils align-center" id="id44">
<caption><span class="caption-number">Table 14 </span><span class="caption-text">Results of overlap analysis between target genes and CAKUT disease</span><a class="headerlink" href="#id44" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"><p>PathwayIDs</p></th>
<th class="head"><p>PathwayNames</p></th>
<th class="head"><p>pValue</p></th>
<th class="head"><p>IntersectionSize</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p><a class="reference external" href="GO:0072001">GO:0072001</a></p></td>
<td><p>renal system development</p></td>
<td><p>8.64e-17</p></td>
<td><p>43</p></td>
</tr>
<tr class="row-odd"><td><p><a class="reference external" href="GO:0001822">GO:0001822</a></p></td>
<td><p>kidney development</p></td>
<td><p>5.14e-16</p></td>
<td><p>41</p></td>
</tr>
<tr class="row-even"><td><p><a class="reference external" href="GO:0060993">GO:0060993</a></p></td>
<td><p>kidney morphogenesis</p></td>
<td><p>8.33e-12</p></td>
<td><p>20</p></td>
</tr>
<tr class="row-odd"><td><p>WP:WP5053</p></td>
<td><p>Development of ureteric collection …</p></td>
<td><p>7.37e-08</p></td>
<td><p>15</p></td>
</tr>
<tr class="row-even"><td><p>WP:WP4823</p></td>
<td><p>Genes controlling nephrogenesis</p></td>
<td><p>2.89e-04</p></td>
<td><p>10</p></td>
</tr>
<tr class="row-odd"><td><p>PMC5748921-PMC6115658</p></td>
<td><p>CAKUT causal genes</p></td>
<td><p>2.30e-03</p></td>
<td><p>6</p></td>
</tr>
<tr class="row-even"><td><p>WP:WP4830</p></td>
<td><p>GDNF/RET signalling axis</p></td>
<td><p>1.54e-02</p></td>
<td><p>5</p></td>
</tr>
</tbody>
</table>
<p>Ozisik <em>et al.,</em> <a class="footnote-reference brackets" href="#id33" id="id14" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a> identified 7 pathways/processes related to CAKUT disease over 13. ODAMNet found the same 7
pathways/processes.</p>
</section>
</section>
</section>
<section id="active-modules-identification-ami">
<span id="usecase2-ami"></span><h2>Active Modules Identification (AMI)<a class="headerlink" href="#active-modules-identification-ami" title="Permalink to this heading"></a></h2>
<p>ODAMNet identifies active modules using a protein-protein interaction (PPI) network with DOMINO <a class="footnote-reference brackets" href="#id40" id="id15" role="doc-noteref"><span class="fn-bracket">[</span>8<span class="fn-bracket">]</span></a>. Then, it performs
an overlap analysis between each identified active modules and pathways/processes related to CAKUT.</p>
<p class="italic">For more detail, see <a class="reference internal" href="../approaches/methods_AMI.html"><span class="doc">Active Module Identification</span></a> section.</p>
<section id="running-active-modules-identification-with-data-provided-by-users">
<h3>Running active modules identification with data provided by users<a class="headerlink" href="#running-active-modules-identification-with-data-provided-by-users" title="Permalink to this heading"></a></h3>
<p>Target genes file is given using the <code class="docutils literal notranslate"><span class="pre">--targetGenesFile</span></code> parameter. It contains the list of vitamin A target genes
[<a class="reference internal" href="../formats/Input.html#targetgenesfile"><span class="std std-ref">FORMAT</span></a>].</p>
<p>Pathways/processes related to CAKUT are given using the <code class="docutils literal notranslate"><span class="pre">--GMT</span></code> parameter [<a class="reference internal" href="../formats/Input.html#pathways"><span class="std std-ref">FORMAT</span></a>]. Background genes
of each source of pathways/processes is required (<code class="docutils literal notranslate"><span class="pre">--backgroundFile</span></code> parameter). The file contains the list of
background files.</p>
<p>AMI is performed on a protein-protein interaction (PPI) network named <code class="docutils literal notranslate"><span class="pre">PPI_network_2016.sif</span></code> [<a class="reference internal" href="../formats/Input.html#sif"><span class="std std-ref">FORMAT</span></a>].
The PPI network file is provided using <code class="docutils literal notranslate"><span class="pre">--networkFile</span></code> parameter. The PPI network is available in <a class="reference external" href="https://github.com/MOohTus/ODAMNet/tree/main/useCases/InputData">GitHub</a>.</p>
<p>Results files are saved into <code class="docutils literal notranslate"><span class="pre">useCases/OutputResults_useCase2/</span></code> folder.</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>odamnet<span class="w"> </span>domino<span class="w">  </span>--targetGenesFile<span class="w"> </span>useCases/InputData/VitA-Balmer2002-Genes.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--GMT<span class="w"> </span>useCases/InputData/PathwaysOfInterest.gmt<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--backgroundFile<span class="w"> </span>useCases/InputData/PathwaysOfInterestBackground.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--networkFile<span class="w"> </span>useCases/InputData/PPI_network_2016.sif<span class="w"> </span><span class="se">\</span>
<span class="w">                </span>--outputPath<span class="w"> </span>useCases/OutputResults_useCase2
</pre></div>
</div>
<p>Several files are generated:</p>
<ul class="simple">
<li><p><code class="docutils literal notranslate"><span class="pre">DOMINO_inputGeneList_D014801.txt</span></code>: genes targeted by vitamin A, used for the active module identification.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">Overlap_AM_*_genesList_withpathOfInterest.csv</span></code>: results of the overlap analysis between target genes and pathways/processes of
related to CAKUT. One file per active module.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_activeModulesNetwork.txt</span></code>, <code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_overlapAMresults4Cytoscape.txt</span></code>,
<code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_activeModules.txt</span></code>, <code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_activeModulesNetworkMetrics.txt</span></code> and
<code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_signOverlap.txt</span></code>: some statistics are calculated and saved into files. Theses files are useful
for visualisation.</p></li>
</ul>
<p class="italic">For more details about output files, see <a class="reference internal" href="../formats/Output.html"><span class="doc">Output files</span></a> page (<a class="reference internal" href="../formats/Output.html#overlapoutput"><span class="std std-ref">Overlap analysis</span></a>, <a class="reference internal" href="../formats/Output.html#amioutput"><span class="std std-ref">Active Module Identification</span></a>).</p>
</section>
<section id="results-of-active-module-identification-with-data-provided-by-users">
<h3>Results of active module identification with data provided by users<a class="headerlink" href="#results-of-active-module-identification-with-data-provided-by-users" title="Permalink to this heading"></a></h3>
<section id="id16">
<h4>Data provided by users overview<a class="headerlink" href="#id16" title="Permalink to this heading"></a></h4>
<p>Target genes are coming from Balmer and Blomhoff <a class="footnote-reference brackets" href="#id35" id="id17" role="doc-noteref"><span class="fn-bracket">[</span>3<span class="fn-bracket">]</span></a>. Pathways of interests are coming from Reactome <a class="footnote-reference brackets" href="#id37" id="id18" role="doc-noteref"><span class="fn-bracket">[</span>5<span class="fn-bracket">]</span></a> and
WikiPathways <a class="footnote-reference brackets" href="#id36" id="id19" role="doc-noteref"><span class="fn-bracket">[</span>4<span class="fn-bracket">]</span></a> and processes of interest are coming from Biological Process (GO) <a class="footnote-reference brackets" href="#id38" id="id20" role="doc-noteref"><span class="fn-bracket">[</span>6<span class="fn-bracket">]</span></a><sup>,</sup> <a class="footnote-reference brackets" href="#id39" id="id21" role="doc-noteref"><span class="fn-bracket">[</span>7<span class="fn-bracket">]</span></a>.</p>
<p>Details of input data are presented in the <a class="reference internal" href="#usecase2dataoverviewami"><span class="std std-ref">Table 15</span></a>.</p>
<span id="usecase2dataoverviewami"></span><table class="docutils align-center" id="id45">
<caption><span class="caption-number">Table 15 </span><span class="caption-text">Overview of data provided by users</span><a class="headerlink" href="#id45" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"></th>
<th class="head"><p>Number</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Vitamin A target genes</p></td>
<td><p>521</p></td>
</tr>
<tr class="row-odd"><td><p>Pathways/processes related to CAKUT</p></td>
<td><p>13</p></td>
</tr>
<tr class="row-even"><td><p>Pathways/processes sources</p></td>
<td><p>3</p></td>
</tr>
</tbody>
</table>
</section>
<section id="active-modules-identification-results">
<h4>Active Modules Identification results<a class="headerlink" href="#active-modules-identification-results" title="Permalink to this heading"></a></h4>
<div class="admonition warning">
<p class="admonition-title">Warning</p>
<p><svg version="1.1" width="2.0em" height="2.0em" class="sd-octicon sd-octicon-alert" viewBox="0 0 24 24" aria-hidden="true"><path d="M13 17.5a1 1 0 11-2 0 1 1 0 012 0zm-.25-8.25a.75.75 0 00-1.5 0v4.5a.75.75 0 001.5 0v-4.5z"></path><path fill-rule="evenodd" d="M9.836 3.244c.963-1.665 3.365-1.665 4.328 0l8.967 15.504c.963 1.667-.24 3.752-2.165 3.752H3.034c-1.926 0-3.128-2.085-2.165-3.752L9.836 3.244zm3.03.751a1 1 0 00-1.732 0L2.168 19.499A1 1 0 003.034 21h17.932a1 1 0 00.866-1.5L12.866 3.994z"></path></svg> DOMINO <a class="footnote-reference brackets" href="#id40" id="id22" role="doc-noteref"><span class="fn-bracket">[</span>8<span class="fn-bracket">]</span></a> server doesn’t allow to set the random seed. That why, results cannot be reproduced identically.</p>
</div>
<p>Target genes (521 in total) are defined as <strong>active genes</strong> by DOMINO <a class="footnote-reference brackets" href="#id40" id="id23" role="doc-noteref"><span class="fn-bracket">[</span>8<span class="fn-bracket">]</span></a>. Using the PPI network, DOMINO found
<strong>18 active modules</strong> over-represented by target genes, presented in the <a class="reference internal" href="#usecase2amiresults"><span class="std std-ref">Table 16</span></a>.</p>
<span id="usecase2amiresults"></span><table class="docutils align-center" id="id46">
<caption><span class="caption-number">Table 16 </span><span class="caption-text">Composition of the 18 identified active modules by DOMINO</span><a class="headerlink" href="#id46" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"></th>
<th class="head"><p>Minimum number</p></th>
<th class="head"><p>Maximum number</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Edges</p></td>
<td><p>5</p></td>
<td><p>157</p></td>
</tr>
<tr class="row-odd"><td><p>Nodes</p></td>
<td><p>5</p></td>
<td><p>79</p></td>
</tr>
<tr class="row-even"><td><p>Target genes</p></td>
<td><p>3</p></td>
<td><p>21</p></td>
</tr>
</tbody>
</table>
<p class="italic">See <code class="docutils literal notranslate"><span class="pre">DOMINO_genesList_activeModulesNetworkMetrics.txt</span></code> file for more details.</p>
</section>
<section id="id24">
<h4>Overlap analysis results<a class="headerlink" href="#id24" title="Permalink to this heading"></a></h4>
<p>After identification of active modules, ODAMNet performs an overlap analysis between each identified active modules and
pathways/processes related to CAKUT. Significant overlaps are found between <strong>6 active modules</strong> and <strong>7 pathways/processes</strong>
related to CAKUT (padjusted &lt;= 0.05).</p>
<span id="usecase2amioverlap"></span><table class="docutils align-center" id="id47">
<caption><span class="caption-number">Table 17 </span><span class="caption-text">Overlap analysis between active modules and pathways/processes related to CAKUT</span><a class="headerlink" href="#id47" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"><p>termID</p></th>
<th class="head"><p>termTitle</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p><a class="reference external" href="GO:0001822">GO:0001822</a></p></td>
<td><p>kidney development</p></td>
</tr>
<tr class="row-odd"><td><p><a class="reference external" href="GO:0060993">GO:0060993</a></p></td>
<td><p>kidney morphogenesis</p></td>
</tr>
<tr class="row-even"><td><p><a class="reference external" href="GO:0072001">GO:0072001</a></p></td>
<td><p>renal system development</p></td>
</tr>
<tr class="row-odd"><td><p><strong>REAC:R-HSA-8853659</strong></p></td>
<td><p><strong>RET signaling</strong></p></td>
</tr>
<tr class="row-even"><td><p>WP:WP4823</p></td>
<td><p>Genes controlling nephrogenesis</p></td>
</tr>
<tr class="row-odd"><td><p>WP:WP4830</p></td>
<td><p>GDNF/RET signalling axis</p></td>
</tr>
<tr class="row-even"><td><p>WP:WP5053</p></td>
<td><p>Development of ureteric collection system</p></td>
</tr>
</tbody>
</table>
<p>The <strong>REAC:R-HSA-8853659</strong> Reactome pathway was not identified with the overlap approach used in the Ozisik
<em>et al.</em> <a class="footnote-reference brackets" href="#id33" id="id25" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a> paper.</p>
</section>
<section id="visualisation-of-active-module-identification-results">
<h4>Visualisation of active module identification results<a class="headerlink" href="#visualisation-of-active-module-identification-results" title="Permalink to this heading"></a></h4>
<p>Visualization can help to identify overlap between active modules and pathways/processes related to CAKUT. The
<a class="reference internal" href="#dominousage2fig"><span class="std std-numref">Fig. 23</span></a> presents active modules that have a significant overlap with pathways/processes related to
CAKUT. For better visualization, only 3 over the 6 identified active modules are displayed. You can retrieve the complete
figure in the <a class="reference external" href="https://github.com/MOohTus/ODAMNet/tree/main/useCases/InputData">GitHub page</a>.</p>
<figure class="align-center" id="id48">
<span id="dominousage2fig"></span><img alt="usecase2 AMI" src="../../_images/useCase2_DOMINO_network.png" />
<figcaption>
<p><span class="caption-number">Fig. 23 </span><span class="caption-text">: Visualization of 3 active modules that have a significant overlap with pathways/processes related to CAKUT.
This figure is created using Cytoscape <a class="footnote-reference brackets" href="#id41" id="id26" role="doc-noteref"><span class="fn-bracket">[</span>9<span class="fn-bracket">]</span></a>. Target genes are in grey.</span><a class="headerlink" href="#id48" title="Permalink to this image"></a></p>
</figcaption>
</figure>
<p>As you can see in the <a class="reference internal" href="#dominousage2fig"><span class="std std-numref">Fig. 23</span></a>, topology of modules is different and associated pathways/processes
varies. Target genes are in grey and others are in white.</p>
<p>The first active module (left in the <a class="reference internal" href="#dominousage2fig"><span class="std std-numref">Fig. 23</span></a>) is very connected and contains genes involved in
only one pathway related to CAKUT (<em>RET signaling</em>). Here, the connection between target genes and CAKUT disease is
indirect. Indeed, genes involved in the pathway are not genes targeted by vitamin A. That why we didn’t found this
pathways with the overlap approach (see <a class="reference internal" href="#usecase2-overlap"><span class="std std-ref">Use-case 2 overlap results</span></a>).</p>
<p>The second active module (middle in the <a class="reference internal" href="#dominousage2fig"><span class="std std-numref">Fig. 23</span></a>) is sparse and contains genes involved in several
pathways/processes related to CAKUT (all the 7 pathways/processes identified). Some genes are targeted by vitamin A,
others don’t. Three genes seem to play key roles because they are part of several pathways/processes as <strong>RET</strong>, <strong>STAT1</strong>
or <strong>GDNF</strong> that is not a target gene (white node).</p>
<p>The third active module (right in the <a class="reference internal" href="#dominousage2fig"><span class="std std-numref">Fig. 23</span></a>) contains target genes and genes involved essentially
in kidney development.</p>
<p class="italic">To know how to create this figure, see the <span class="xref std std-ref">networkAMI</span> section.</p>
</section>
</section>
</section>
<section id="random-walk-with-restart-rwr">
<span id="usecase2-rwr"></span><h2>Random Walk with Restart (RWR)<a class="headerlink" href="#random-walk-with-restart-rwr" title="Permalink to this heading"></a></h2>
<div class="admonition note">
<p class="admonition-title">Note</p>
<div class="line-block">
<div class="line"><strong>Multilayer</strong> is a network with several layers where layers contain different nodes types</div>
<div class="line"><strong>Multiplex</strong> is a network with several layers (multilayer) where layers contain same type of nodes</div>
</div>
</div>
<p>The third approach implemented in ODAMNet is a Random Walk with Restart analysis (RWR). RWR is applied using
multiXrank <a class="footnote-reference brackets" href="#id42" id="id27" role="doc-noteref"><span class="fn-bracket">[</span>10<span class="fn-bracket">]</span></a> through a multilayer composed of genes and diseases nodes.</p>
<p>We applied RWR using two different multilayer compositions:</p>
<ul class="simple">
<li><p><strong>Multilayer 1</strong>: Multiplex + pathways/processes related to CAKUT network</p></li>
<li><p><strong>Multilayer 2</strong>: Multiplex + disease-disease similarity network</p></li>
</ul>
<p>Used multilayers will be detailed in corresponding section results.</p>
<p class="italic">For more details about RWR, see <a class="reference internal" href="../approaches/methods_RWR.html"><span class="doc">Random Walk with Restart</span></a>.</p>
<section id="running-random-walk-analysis-with-data-provided-by-users">
<h3>Running Random Walk analysis with data provided by users<a class="headerlink" href="#running-random-walk-analysis-with-data-provided-by-users" title="Permalink to this heading"></a></h3>
<p>Target genes files is given using <code class="docutils literal notranslate"><span class="pre">--targetGenesFile</span></code> parameter. The file contains list of genes targeted by vitamin
A [<a class="reference internal" href="../formats/Input.html#targetgenesfile"><span class="std std-ref">FORMAT</span></a>].</p>
<p>To perform RWR, multiXrank <a class="footnote-reference brackets" href="#id42" id="id28" role="doc-noteref"><span class="fn-bracket">[</span>10<span class="fn-bracket">]</span></a> needs a configuration file given using <code class="docutils literal notranslate"><span class="pre">--configPath</span></code> parameter. This configuration
file contains path of different networks and target genes files used. The configuration file might contains parameters
for RWR analysis. We run the RWR analysis with default parameters.</p>
<p>MultiXrank needs also the networks directory path given using <code class="docutils literal notranslate"><span class="pre">--networksPath</span></code> parameter. MultiXrank defines target
genes as seeds for the walk. Target genes will be saved into a file given using <code class="docutils literal notranslate"><span class="pre">--seedsFile</span></code> parameter.</p>
<p>Two others parameters are required: number to select top nodes in each layer (<code class="docutils literal notranslate"><span class="pre">--top</span></code>) and file name to saved result
network which contains top nodes of each layers and their relationships (<code class="docutils literal notranslate"><span class="pre">--sifFileName</span></code>).</p>
<p>Results files are saved into <code class="docutils literal notranslate"><span class="pre">useCases/OutputResults_useCase2/</span></code> folder.</p>
<div class="admonition tip">
<p class="admonition-title">Tip</p>
<p>Whatever the networks used, the <strong>command line is the same</strong>. But you have to <strong>change</strong> the network name inside the
<strong>configuration file</strong>.</p>
<div class="sphinx-tabs docutils container">
<div aria-label="Tabbed content" class="closeable" role="tablist"><button aria-controls="panel-0-UGF0aHdheXMvcHJvY2Vzc2VzIG9mIGludGVyZXN0IG5ldHdvcms=" aria-selected="true" class="sphinx-tabs-tab group-tab" id="tab-0-UGF0aHdheXMvcHJvY2Vzc2VzIG9mIGludGVyZXN0IG5ldHdvcms=" name="UGF0aHdheXMvcHJvY2Vzc2VzIG9mIGludGVyZXN0IG5ldHdvcms=" role="tab" tabindex="0">Pathways/processes of interest network</button><button aria-controls="panel-0-RGlzZWFzZS1EaXNlYXNlIHNpbWlsYXJpdHkgbmV0d29yaw==" aria-selected="false" class="sphinx-tabs-tab group-tab" id="tab-0-RGlzZWFzZS1EaXNlYXNlIHNpbWlsYXJpdHkgbmV0d29yaw==" name="RGlzZWFzZS1EaXNlYXNlIHNpbWlsYXJpdHkgbmV0d29yaw==" role="tab" tabindex="-1">Disease-Disease similarity network</button></div><div aria-labelledby="tab-0-UGF0aHdheXMvcHJvY2Vzc2VzIG9mIGludGVyZXN0IG5ldHdvcms=" class="sphinx-tabs-panel group-tab" id="panel-0-UGF0aHdheXMvcHJvY2Vzc2VzIG9mIGludGVyZXN0IG5ldHdvcms=" name="UGF0aHdheXMvcHJvY2Vzc2VzIG9mIGludGVyZXN0IG5ldHdvcms=" role="tabpanel" tabindex="0"><div class="highlight-bash notranslate"><div class="highlight"><pre><span></span><span class="w"> </span>multiplex:
<span class="w">     </span><span class="m">1</span>:
<span class="w">         </span>layers:
<span class="w">             </span>-<span class="w"> </span>multiplex/1/Complexes_Nov2020.gr
<span class="w">             </span>-<span class="w"> </span>multiplex/1/PPI_Jan2021.gr
<span class="w">             </span>-<span class="w"> </span>multiplex/1/Reactome_Nov2020.gr
<span class="w">     </span><span class="m">2</span>:
<span class="w">         </span>layers:
<span class="hll"><span class="w">             </span>-<span class="w"> </span>multiplex/2/pathwaysOfInterestNetwork_fromPaper.sif
</span><span class="w"> </span>bipartite:
<span class="hll"><span class="w">     </span>bipartite/Bipartite_pathOfInterest_geneSymbols_fromPaper.tsv:
</span><span class="w">         </span>source:<span class="w"> </span><span class="m">2</span>
<span class="w">         </span>target:<span class="w"> </span><span class="m">1</span>
<span class="w"> </span>seed:
<span class="w">     </span>seeds.txt
</pre></div>
</div>
</div><div aria-labelledby="tab-0-RGlzZWFzZS1EaXNlYXNlIHNpbWlsYXJpdHkgbmV0d29yaw==" class="sphinx-tabs-panel group-tab" hidden="true" id="panel-0-RGlzZWFzZS1EaXNlYXNlIHNpbWlsYXJpdHkgbmV0d29yaw==" name="RGlzZWFzZS1EaXNlYXNlIHNpbWlsYXJpdHkgbmV0d29yaw==" role="tabpanel" tabindex="0"><div class="highlight-bash notranslate"><div class="highlight"><pre><span></span><span class="w"> </span>multiplex:
<span class="w">     </span><span class="m">1</span>:
<span class="w">         </span>layers:
<span class="w">             </span>-<span class="w"> </span>multiplex/1/Complexes_Nov2020.gr
<span class="w">             </span>-<span class="w"> </span>multiplex/1/PPI_Jan2021.gr
<span class="w">             </span>-<span class="w"> </span>multiplex/1/Reactome_Nov2020.gr
<span class="w">     </span><span class="m">2</span>:
<span class="w">         </span>layers:
<span class="hll"><span class="w">             </span>-<span class="w"> </span>multiplex/2/DiseaseSimilarity_network_2022_06_11.txt
</span><span class="w"> </span>bipartite:
<span class="hll"><span class="w">     </span>bipartite/Bipartite_genes_to_OMIM_2022_09_27.txt:
</span><span class="w">         </span>source:<span class="w"> </span><span class="m">2</span>
<span class="w">         </span>target:<span class="w"> </span><span class="m">1</span>
<span class="w"> </span>seed:
<span class="w">     </span>seeds.txt
</pre></div>
</div>
</div></div>
</div>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>odamnet<span class="w"> </span>multixrank<span class="w">  </span>--targetGenesFile<span class="w"> </span>useCases/InputData/VitA-Balmer2002-Genes.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                    </span>--configPath<span class="w"> </span>useCases/InputData/config_minimal_useCase2.yml<span class="w"> </span><span class="se">\</span>
<span class="w">                    </span>--networksPath<span class="w"> </span>useCases/InputData/<span class="w"> </span><span class="se">\</span>
<span class="w">                    </span>--seedsFile<span class="w"> </span>useCases/InputData/seeds.txt<span class="w"> </span><span class="se">\</span>
<span class="w">                    </span>--sifFileName<span class="w"> </span>resultsNetwork_useCase2.sif<span class="w"> </span><span class="se">\</span>
<span class="w">                    </span>--top<span class="w"> </span><span class="m">10</span><span class="w"> </span><span class="se">\</span>
<span class="w">                    </span>--outputPath<span class="w"> </span>useCases/OutputResults_useCase2/
</pre></div>
</div>
<p>Several files are generated into <code class="docutils literal notranslate"><span class="pre">RWR_genesList/</span></code> folder:</p>
<ul class="simple">
<li><p><code class="docutils literal notranslate"><span class="pre">config_minimal_useCase2.yml</span></code> and <code class="docutils literal notranslate"><span class="pre">seeds.txt</span></code>: copies of the input configuration and seed files</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">multiplex_1.tsv</span></code> and <code class="docutils literal notranslate"><span class="pre">multiplex_2.tsv</span></code>: result files that contain RWR score for each node. multiplex_1
corresponds to the gene multiplex and multiplex_2 corresponds to the disease network</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">resultsNetwork_useCase2.sif</span></code>: SIF file name that contains the network result</p></li>
</ul>
<div class="italic line-block">
<div class="line">For more details about the input files, see <a class="reference internal" href="../formats/Input.html#gr"><span class="std std-ref">Random Walk with Restart input network</span></a> section.</div>
<div class="line">For more details about the output files, see <a class="reference internal" href="../formats/Output.html"><span class="doc">Output files</span></a> page.</div>
</div>
</section>
<section id="results-of-random-walk-analysis-with-data-provided-by-users">
<h3>Results of Random Walk analysis with data provided by users<a class="headerlink" href="#results-of-random-walk-analysis-with-data-provided-by-users" title="Permalink to this heading"></a></h3>
<section id="id29">
<h4>Data provided by users overview<a class="headerlink" href="#id29" title="Permalink to this heading"></a></h4>
<p>Target genes are coming from Balmer and Blomhoff <a class="footnote-reference brackets" href="#id35" id="id30" role="doc-noteref"><span class="fn-bracket">[</span>3<span class="fn-bracket">]</span></a>. There are <strong>521 vitamin A target genes</strong>.</p>
<p><strong>Multilayer 1</strong> contains:</p>
<ul>
<li><p>multiplex network</p>
<blockquote>
<div><ul class="simple">
<li><p>PPI (14,703 nodes and 143,653 edges)</p></li>
<li><p>molecular complexes (8,537 nodes and 63,531 edges)</p></li>
<li><p>Reactome pathways (7,926 nodes and 194,500 edges)</p></li>
</ul>
</div></blockquote>
</li>
<li><p>pathways/processes related to CAKUT network (13 nodes and 0 edges)</p></li>
</ul>
<p>These two networks are linked using bipartite that contains 1,655 associations (866 genes and 13 pathways/processes).</p>
<p class="italic">For more details about the pathways/processes of interest network: see <a class="reference internal" href="../network/NetworkUsed.html#pathwaysofinterestnet"><span class="std std-ref">Pathways/processes of interest network</span></a>.</p>
<p><strong>Multilayer 2</strong> contains:</p>
<ul>
<li><p>multiplex network</p>
<blockquote>
<div><ul class="simple">
<li><p>PPI (14,703 nodes and 143,653 edges)</p></li>
<li><p>molecular complexes (8,537 nodes and 63,531 edges)</p></li>
<li><p>Reactome pathways (7,926 nodes and 194,500 edges)</p></li>
</ul>
</div></blockquote>
</li>
<li><p>disease-disease similarity network (8,264 nodes and 33,925 edges)</p></li>
</ul>
<p>These two networks are linked using bipartite that contains 6,534 associations (4,483 genes and 5,878 diseases).</p>
<p class="italic">For more details about the disease-disease similarity network: see <span class="xref std std-ref">DDnet</span>.</p>
</section>
<section id="multilayer-1-pathways-processes-related-to-cakut-network">
<h4>Multilayer 1 : Pathways/processes related to CAKUT network<a class="headerlink" href="#multilayer-1-pathways-processes-related-to-cakut-network" title="Permalink to this heading"></a></h4>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>In this part, we present results found using the Multilayer 1: multiplex + pathways/processed related to CAKUT
network</p>
</div>
<p>MultiXrank <a class="footnote-reference brackets" href="#id42" id="id31" role="doc-noteref"><span class="fn-bracket">[</span>10<span class="fn-bracket">]</span></a> defines <strong>483</strong> target genes over the 521 as seeds to start the walk. For each node (genes or
pathways/processes related to CAKUT), multiXrank calculates a RWR score. Based on this score, we selected the 10 top of
node scores.</p>
<p><em>In this part, we present results found for the first multiplex composition: multiplex + pathways/processes of interest.</em></p>
<p>First, target genes are used as seed to start the walk: <code class="docutils literal notranslate"><span class="pre">483/521</span></code> genes are used.</p>
<p>The gene with the highest score is <code class="docutils literal notranslate"><span class="pre">ASMT</span></code> with <code class="docutils literal notranslate"><span class="pre">score</span> <span class="pre">=</span> <span class="pre">0.0006682735081574565</span></code> (it’s a seed). This score helps
us to select a list of pathways/processes. All pathways/processes with a score bigger than this score are extracted and considered as connected
with target genes (i.e. seeds).</p>
<p>According this highest score, <strong>4 pathways/processes</strong> are selected (<a class="reference internal" href="#usecase2-pathwaysrwr"><span class="std std-ref">Table 18</span></a>).</p>
<span id="usecase2-pathwaysrwr"></span><table class="docutils align-center" id="id49">
<caption><span class="caption-number">Table 18 </span><span class="caption-text">pathways/processes linked to target genes</span><a class="headerlink" href="#id49" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"><p>node</p></th>
<th class="head"><p>pathway</p></th>
<th class="head"><p>score</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p><a class="reference external" href="GO:0072001">GO:0072001</a></p></td>
<td><p>renal system development</p></td>
<td><p>0.002101</p></td>
</tr>
<tr class="row-odd"><td><p><a class="reference external" href="GO:0001822">GO:0001822</a></p></td>
<td><p>kidney development</p></td>
<td><p>0.001847</p></td>
</tr>
<tr class="row-even"><td><p><strong>REAC:R-HSA-195721</strong></p></td>
<td><p><strong>Signaling by WNT</strong></p></td>
<td><p><strong>0.001660</strong></p></td>
</tr>
<tr class="row-odd"><td><p><strong>REAC:R-HSA-157118</strong></p></td>
<td><p><strong>Signaling by NOTCH</strong></p></td>
<td><p><strong>0.001140</strong></p></td>
</tr>
</tbody>
</table>
<p>Two pathways not found with the previous approaches, are link to target genes: <code class="docutils literal notranslate"><span class="pre">REAC:R-HSA-195721</span></code> and <code class="docutils literal notranslate"><span class="pre">REAC:R-HSA-157118</span></code>.
You can visualise the results with a network as shown on the <a class="reference internal" href="#usecase2-pathwaysnetworkrwr"><span class="std std-numref">Fig. 24</span></a>.</p>
<figure class="align-center" id="id50">
<span id="usecase2-pathwaysnetworkrwr"></span><img alt="usecase 2 pathwaysNetworkRWR" src="../../_images/RWR_pathwaysNet_useCase2.png" />
<figcaption>
<p><span class="caption-number">Fig. 24 </span><span class="caption-text">: Results from RWR through the molecular multilayer and pathways/processes of interest network</span><a class="headerlink" href="#id50" title="Permalink to this image"></a></p>
<div class="legend">
<blockquote>
<div><p>Pathways/processes of interest are represented by triangle nodes in pink, genes are represented by white nodes and target genes by grey nodes.</p>
</div></blockquote>
</div>
</figcaption>
</figure>
</section>
<section id="disease-disease-similarity-network">
<h4>Disease-Disease similarity network<a class="headerlink" href="#disease-disease-similarity-network" title="Permalink to this heading"></a></h4>
<p><em>In this part, we present results found for the second multiplex composition: multiplex + disease-disease network.</em></p>
<p>First, target genes are used as seed to start the walk: <code class="docutils literal notranslate"><span class="pre">483/521</span></code> genes are used.</p>
<p>We selected the top 10 of diseases (<a class="reference internal" href="#usecase2-diseasesrwr"><span class="std std-ref">Table 19</span></a>).</p>
<span id="usecase2-diseasesrwr"></span><table class="docutils align-center" id="id51">
<caption><span class="caption-number">Table 19 </span><span class="caption-text">Diseases linked to target genes</span><a class="headerlink" href="#id51" title="Permalink to this table"></a></caption>
<thead>
<tr class="row-odd"><th class="head"><p>node</p></th>
<th class="head"><p>Diseases</p></th>
<th class="head"><p>score</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>OMIM:178500</p></td>
<td><p>Pulmonary fibrosis, idiopathic</p></td>
<td><p>0.000334</p></td>
</tr>
<tr class="row-odd"><td><p>OMIM:125853</p></td>
<td><p>Diabetes mellitus, noninsulin-dependent</p></td>
<td><p>0.000301</p></td>
</tr>
<tr class="row-even"><td><p>OMIM:215600</p></td>
<td><p>Cirrhosis, familial</p></td>
<td><p>0.000255</p></td>
</tr>
<tr class="row-odd"><td><p>OMIM:613659</p></td>
<td><p>Gastric cancer, somatic</p></td>
<td><p>0.000235</p></td>
</tr>
<tr class="row-even"><td><p>OMIM:211980</p></td>
<td><p>Lung cancer, susceptibility to</p></td>
<td><p>0.000230</p></td>
</tr>
<tr class="row-odd"><td><p>OMIM:104300</p></td>
<td><p>Alzheimer disease</p></td>
<td><p>0.000224</p></td>
</tr>
<tr class="row-even"><td><p>OMIM:168600</p></td>
<td><p>Parkinson disease, late-onset</p></td>
<td><p>0.000192</p></td>
</tr>
<tr class="row-odd"><td><p>OMIM:601859</p></td>
<td><p>Autoimmune lymphoproliferative syndrome</p></td>
<td><p>0.000182</p></td>
</tr>
<tr class="row-even"><td><p>OMIM:601665</p></td>
<td><p>OBESITY</p></td>
<td><p>0.000181</p></td>
</tr>
<tr class="row-odd"><td><p>OMIM:171300</p></td>
<td><p>PHEOCHROMOCYTOMA</p></td>
<td><p>0.000145</p></td>
</tr>
</tbody>
</table>
<p>You can represent the results with a network as shown in the <a class="reference internal" href="#usecase2-simnetworkrwr"><span class="std std-numref">Fig. 25</span></a>.</p>
<figure class="align-center" id="id52">
<span id="usecase2-simnetworkrwr"></span><img alt="usecase 2 simNetworkRWR" src="../../_images/RWR_pathwaysNet_useCase2_simNet.png" />
<figcaption>
<p><span class="caption-number">Fig. 25 </span><span class="caption-text">: Results from RWR through the molecular multilayer and disease-disease similarity network</span><a class="headerlink" href="#id52" title="Permalink to this image"></a></p>
<div class="legend">
<blockquote>
<div><p>Diseases are represented by triangle pink nodes, genes are represented by white nodes and target genes by grey nodes.</p>
</div></blockquote>
</div>
</figcaption>
</figure>
</section>
</section>
</section>
<section id="rare-disease-pathways-identified">
<h2>Rare disease pathways identified<a class="headerlink" href="#rare-disease-pathways-identified" title="Permalink to this heading"></a></h2>
<p>Approaches give us a list of CAKUT pathways/processes significantly connected to vitamin A target genes. To easily compare results,
we use orsum <a class="footnote-reference brackets" href="#id34" id="id32" role="doc-noteref"><span class="fn-bracket">[</span>2<span class="fn-bracket">]</span></a>. Results are displayed into a heatmap in the <a class="reference internal" href="#usecase2-orsum"><span class="std std-numref">Fig. 26</span></a>.</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>orsum.py<span class="w">    </span>--gmt<span class="w"> </span>00_Data/hsapiens_background.gmt<span class="w"> </span><span class="se">\</span>
<span class="w">            </span>--files<span class="w"> </span>Overlap_genesList_withpathOfInterest.4Orsum<span class="w"> </span>DOMINO_genesList_signOverlap.4Orsum<span class="w"> </span>pathwaysResults.4Orsum<span class="w"> </span><span class="se">\</span>
<span class="w">            </span>--fileAliases<span class="w"> </span>Overlap<span class="w"> </span>DOMINO<span class="w"> </span>multiXrank<span class="w"> </span><span class="se">\</span>
<span class="w">            </span>--outputFolder<span class="w"> </span>useCase2Comparison/
</pre></div>
</div>
<figure class="align-center" id="id53">
<span id="usecase2-orsum"></span><img alt="usecase2 orsum" src="../../_images/useCase2_orsum.png" />
<figcaption>
<p><span class="caption-number">Fig. 26 </span><span class="caption-text">: Comparison of use-case 2 results using orsum</span><a class="headerlink" href="#id53" title="Permalink to this image"></a></p>
</figcaption>
</figure>
</section>
<section id="references">
<h2>References<a class="headerlink" href="#references" title="Permalink to this heading"></a></h2>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id33" role="note">
<span class="label"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id1">1</a>,<a role="doc-backlink" href="#id2">2</a>,<a role="doc-backlink" href="#id3">3</a>,<a role="doc-backlink" href="#id14">4</a>,<a role="doc-backlink" href="#id25">5</a>)</span>
<p>Ozisik O, Ehrhart F, Evelo C <em>et al.</em>. Overlap of vitamin A and vitamin D target genes with CAKUT-related processes. F1000Research. 2021.</p>
</aside>
<aside class="footnote brackets" id="id34" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id32">2</a><span class="fn-bracket">]</span></span>
<p>Davis AP, Grondin CJ, Johnson RJ <em>et al.</em>. The Comparative Toxicogenomics Database: update 2021. Nucleic acids research. 2021.</p>
</aside>
<aside class="footnote brackets" id="id35" role="note">
<span class="label"><span class="fn-bracket">[</span>3<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id4">1</a>,<a role="doc-backlink" href="#id9">2</a>,<a role="doc-backlink" href="#id17">3</a>,<a role="doc-backlink" href="#id30">4</a>)</span>
<p>Balmer JE &amp; Blomhoff R. Gene expression regulation by retinoic acid. Journal of lipid research. 2002.</p>
</aside>
<aside class="footnote brackets" id="id36" role="note">
<span class="label"><span class="fn-bracket">[</span>4<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id5">1</a>,<a role="doc-backlink" href="#id11">2</a>,<a role="doc-backlink" href="#id19">3</a>)</span>
<p>Martens M, Ammar A, Riutta A <em>et al.</em>. WikiPathways: connecting communities. Nucleic acids research. 2021.</p>
</aside>
<aside class="footnote brackets" id="id37" role="note">
<span class="label"><span class="fn-bracket">[</span>5<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id6">1</a>,<a role="doc-backlink" href="#id10">2</a>,<a role="doc-backlink" href="#id18">3</a>)</span>
<p>Jassal B, Matthews L, Viteri G <em>et al.</em>. The reactome pathway knowledgebase. Nucleic acids research. 2020.</p>
</aside>
<aside class="footnote brackets" id="id38" role="note">
<span class="label"><span class="fn-bracket">[</span>6<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id7">1</a>,<a role="doc-backlink" href="#id12">2</a>,<a role="doc-backlink" href="#id20">3</a>)</span>
<p>Ashburner M, Ball CA, Blake JA <em>et al.</em>. Gene ontology: tool for the unification of biology. Nature Genetics. 2000.</p>
</aside>
<aside class="footnote brackets" id="id39" role="note">
<span class="label"><span class="fn-bracket">[</span>7<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id8">1</a>,<a role="doc-backlink" href="#id13">2</a>,<a role="doc-backlink" href="#id21">3</a>)</span>
<p>The Gene Ontology Consortium. The Gene Ontology resource: enriching a GOld mine. Nucleic acids research. 2021.</p>
</aside>
<aside class="footnote brackets" id="id40" role="note">
<span class="label"><span class="fn-bracket">[</span>8<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id15">1</a>,<a role="doc-backlink" href="#id22">2</a>,<a role="doc-backlink" href="#id23">3</a>)</span>
<p>Levi H, Rahmanian N, Elkon R <em>et al.</em>. The DOMINO web-server for active module identification analysis. Bioinformatics. 2022.</p>
</aside>
<aside class="footnote brackets" id="id41" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id26">9</a><span class="fn-bracket">]</span></span>
<p>Shannon P, Markiel A, Ozier O <em>et al.</em>. Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome research. 2003.</p>
</aside>
<aside class="footnote brackets" id="id42" role="note">
<span class="label"><span class="fn-bracket">[</span>10<span class="fn-bracket">]</span></span>
<span class="backrefs">(<a role="doc-backlink" href="#id27">1</a>,<a role="doc-backlink" href="#id28">2</a>,<a role="doc-backlink" href="#id31">3</a>)</span>
<p>Baptista A, Gonzalez A &amp; Baudot A. Universal multilayer network exploration by random walk with restart. Communications Physics. 2022.</p>
</aside>
</aside>
</section>
</section>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="usecase1.html" class="btn btn-neutral float-left" title="Use-case 1: data are retrieved automatically by queries" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
        <a href="../formats/Input.html" class="btn btn-neutral float-right" title="Input files" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright 2022, Térézol.</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>