<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
  <meta charset="utf-8" /><meta name="generator" content="Docutils 0.18.1: http://docutils.sourceforge.net/" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>Networks used &mdash; ODAMNet 1 documentation</title>
      <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/custom.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/tabs.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/design-style.4045f2051d55cab465a707391d5b2007.min.css" type="text/css" />
      <link rel="stylesheet" href="../../_static/custom.css" type="text/css" />
      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" type="text/css" />
  <!--[if lt IE 9]>
    <script src="../../_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
        <script data-url_root="../../" id="documentation_options" src="../../_static/documentation_options.js"></script>
        <script src="../../_static/jquery.js"></script>
        <script src="../../_static/underscore.js"></script>
        <script src="../../_static/_sphinx_javascript_frameworks_compat.js"></script>
        <script src="../../_static/doctools.js"></script>
        <script src="../../_static/sphinx_highlight.js"></script>
        <script src="../../_static/tabs.js"></script>
        <script src="../../_static/design-tabs.js"></script>
    <script src="../../_static/js/theme.js"></script>
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="Networks visualisation" href="NetworkVisualisation.html" />
    <link rel="prev" title="Networks creation" href="NetworkCreation.html" /> 
</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >

          
          
          <a href="../../index.html" class="icon icon-home">
            ODAMNet
          </a>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <p class="caption" role="heading"><span class="caption-text">Quickstart</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../Installation.html">Installation</a></li>
<li class="toctree-l1"><a class="reference internal" href="../Installation.html#usage">Usage</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Approaches</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_overview.html">Overview of the approaches</a></li>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_overlap.html">Overlap analysis</a></li>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_AMI.html">Active Module Identification</a></li>
<li class="toctree-l1"><a class="reference internal" href="../approaches/methods_RWR.html">Random Walk with Restart</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Networks</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="NetworkDownloading.html">Networks downloading</a></li>
<li class="toctree-l1"><a class="reference internal" href="NetworkCreation.html">Networks creation</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Networks used</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#genes-multilayer-network">Genes multilayer network</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#protein-protein-interaction-ppi-network">Protein-Protein Interaction (PPI) network</a></li>
<li class="toctree-l3"><a class="reference internal" href="#molecular-complexes-network">Molecular complexes network</a></li>
<li class="toctree-l3"><a class="reference internal" href="#reactome-pathways-network">Reactome pathways network</a></li>
<li class="toctree-l3"><a class="reference internal" href="#use-case-command-lines">Use-case command lines</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#pathways-processes-of-interest-network">Pathways/processes of interest network</a></li>
<li class="toctree-l2"><a class="reference internal" href="#disease-disease-similarity-network">Disease-disease similarity network</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#disease-disease-similarity-network-creation">Disease-disease similarity network creation</a></li>
<li class="toctree-l3"><a class="reference internal" href="#gene-disease-bipartite">Gene-disease bipartite</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#references">References</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="NetworkVisualisation.html">Networks visualisation</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Use-cases</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../analysis/usecase1.html">Use-case 1: data are retrieved automatically by queries</a></li>
<li class="toctree-l1"><a class="reference internal" href="../analysis/usecase2.html">Use-case 2: data are provided by the user</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">File formats</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../formats/Input.html">Input files</a></li>
<li class="toctree-l1"><a class="reference internal" href="../formats/Output.html">Output files</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Code description</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../functions/fcts_arguments.html">Input parameters for each approach</a></li>
<li class="toctree-l1"><a class="reference internal" href="../functions/fcts_description.html">Code functions in details</a></li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../index.html">ODAMNet</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="../../index.html" class="icon icon-home" aria-label="Home"></a></li>
      <li class="breadcrumb-item active">Networks used</li>
      <li class="wy-breadcrumbs-aside">
            <a href="../../_sources/pages/network/NetworkUsed.rst.txt" rel="nofollow"> View page source</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section id="networks-used">
<h1>Networks used<a class="headerlink" href="#networks-used" title="Permalink to this heading"></a></h1>
<div class="admonition warning">
<p class="admonition-title">Warning</p>
<ul class="simple">
<li><p>Gene IDs have to be <strong>consistent</strong> between input data <em>(target genes, GMT and networks)</em></p></li>
<li><p>When data are retrieved by queries, <strong>HGNC</strong> IDs are used.</p></li>
</ul>
</div>
<p>In this section, we present networks used in the use-cases.</p>
<p>We propose to apply the random walk with restart (RWR) approach on two different multilayers.</p>
<figure class="align-center" id="id25">
<span id="multilayercompo"></span><a class="reference internal image-reference" href="../../_images/MultilayerComposition.png"><img alt="multilayerCompo" src="../../_images/MultilayerComposition.png" style="width: 678.6px; height: 378.0px;" /></a>
<figcaption>
<p><span class="caption-number">Fig. 6 </span><span class="caption-text">: Multilayers composition: On the left, multilayer is composed of genes multilayer network and pathways/processes
of interest network (disconnected network). On the right, the multilayer is composed of genes multiplayer network
and disease-disease similarity network.</span><a class="headerlink" href="#id25" title="Permalink to this image"></a></p>
</figcaption>
</figure>
<p>Multilayers are composed of:</p>
<ul class="simple">
<li><p>genes multilayer network + pathways/processes of interest network (<a class="reference internal" href="#multilayercompo"><span class="std std-numref">Fig. 6</span></a> - left part)</p></li>
<li><p>genes multilayer network + disease-disease similarity network (<a class="reference internal" href="#multilayercompo"><span class="std std-numref">Fig. 6</span></a> - right part)</p></li>
</ul>
<section id="genes-multilayer-network">
<span id="genesmultilayernet"></span><h2>Genes multilayer network<a class="headerlink" href="#genes-multilayer-network" title="Permalink to this heading"></a></h2>
<p>The genes multilayer network is composed of three different networks. Nodes are identical between networks, but the
relationships between them are coming from different sources. Networks are downloaded from the <a class="reference external" href="https://www.ndexbio.org/">Network Data Exchange</a> (NDEx) <a class="footnote-reference brackets" href="#id13" id="id1" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a>
(see <a class="reference internal" href="NetworkDownloading.html"><span class="doc">Networks downloading</span></a> page).</p>
<section id="protein-protein-interaction-ppi-network">
<span id="ppinet"></span><h3>Protein-Protein Interaction (PPI) network<a class="headerlink" href="#protein-protein-interaction-ppi-network" title="Permalink to this heading"></a></h3>
<p>The Protein-Protein Interaction (PPI) network is obtained from fusion of three datasets : Hi-Union and Lit-BM <a class="footnote-reference brackets" href="#id14" id="id2" role="doc-noteref"><span class="fn-bracket">[</span>2<span class="fn-bracket">]</span></a> and
APID <a class="footnote-reference brackets" href="#id15" id="id3" role="doc-noteref"><span class="fn-bracket">[</span>3<span class="fn-bracket">]</span></a>. It’s composed of:</p>
<ul class="simple">
<li><p>15,390 3 nodes</p></li>
<li><p>131,087 edges</p></li>
<li><p>UUID: <a class="reference external" href="https://www.ndexbio.org/viewer/networks/bfac0486-cefe-11ed-a79c-005056ae23aa">bfac0486-cefe-11ed-a79c-005056ae23aa</a></p></li>
</ul>
<p><em>This network is also used for the active module identification approach.</em></p>
</section>
<section id="molecular-complexes-network">
<h3>Molecular complexes network<a class="headerlink" href="#molecular-complexes-network" title="Permalink to this heading"></a></h3>
<p>Molecular complexes network is constructed from the fusion of Hu.map <a class="footnote-reference brackets" href="#id16" id="id4" role="doc-noteref"><span class="fn-bracket">[</span>4<span class="fn-bracket">]</span></a> and Corum <a class="footnote-reference brackets" href="#id17" id="id5" role="doc-noteref"><span class="fn-bracket">[</span>5<span class="fn-bracket">]</span></a>. It’s composed of:</p>
<ul class="simple">
<li><p>8,497 nodes</p></li>
<li><p>62,073 edges</p></li>
<li><p>UUID: <a class="reference external" href="https://www.ndexbio.org/viewer/networks/419ae651-cf05-11ed-a79c-005056ae23aa">419ae651-cf05-11ed-a79c-005056ae23aa</a></p></li>
</ul>
</section>
<section id="reactome-pathways-network">
<h3>Reactome pathways network<a class="headerlink" href="#reactome-pathways-network" title="Permalink to this heading"></a></h3>
<p>The Reactome pathways network was build using data derived from Reactome protein-protein interaction data <a class="footnote-reference brackets" href="#id18" id="id6" role="doc-noteref"><span class="fn-bracket">[</span>6<span class="fn-bracket">]</span></a>.
It’s composed of:</p>
<ul class="simple">
<li><p>4,598 nodes</p></li>
<li><p>19,292 edges</p></li>
<li><p>UUID: <a class="reference external" href="https://www.ndexbio.org/viewer/networks/b13e9620-cefd-11ed-a79c-005056ae23aa">b13e9620-cefd-11ed-a79c-005056ae23aa</a></p></li>
</ul>
</section>
<section id="use-case-command-lines">
<h3>Use-case command lines<a class="headerlink" href="#use-case-command-lines" title="Permalink to this heading"></a></h3>
<div class="sphinx-tabs docutils container">
<div aria-label="Tabbed content" class="closeable" role="tablist"><button aria-controls="panel-0-UFBJIG5ldHdvcms=" aria-selected="true" class="sphinx-tabs-tab group-tab" id="tab-0-UFBJIG5ldHdvcms=" name="UFBJIG5ldHdvcms=" role="tab" tabindex="0">PPI network</button><button aria-controls="panel-0-TW9sZWN1bGFyIGNvbXBsZXhlcyBuZXR3b3Jr" aria-selected="false" class="sphinx-tabs-tab group-tab" id="tab-0-TW9sZWN1bGFyIGNvbXBsZXhlcyBuZXR3b3Jr" name="TW9sZWN1bGFyIGNvbXBsZXhlcyBuZXR3b3Jr" role="tab" tabindex="-1">Molecular complexes network</button><button aria-controls="panel-0-UmVhY3RvbWUgcGF0aHdheXMgbmV0d29yaw==" aria-selected="false" class="sphinx-tabs-tab group-tab" id="tab-0-UmVhY3RvbWUgcGF0aHdheXMgbmV0d29yaw==" name="UmVhY3RvbWUgcGF0aHdheXMgbmV0d29yaw==" role="tab" tabindex="-1">Reactome pathways network</button></div><div aria-labelledby="tab-0-UFBJIG5ldHdvcms=" class="sphinx-tabs-panel group-tab" id="panel-0-UFBJIG5ldHdvcms=" name="UFBJIG5ldHdvcms=" role="tabpanel" tabindex="0"><div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>odamnet<span class="w"> </span>networkDownloading<span class="w">  </span>--netUUID<span class="w"> </span>bfac0486-cefe-11ed-a79c-005056ae23aa<span class="w"> </span><span class="se">\</span>
<span class="w">                            </span>--networkFile<span class="w"> </span>useCases/InputData/multiplex/1/PPI_HiUnion_LitBM_APID_gene_names_190123.gr<span class="w"> </span><span class="se">\</span>
<span class="w">                            </span>--simple<span class="w"> </span>True
</pre></div>
</div>
</div><div aria-labelledby="tab-0-TW9sZWN1bGFyIGNvbXBsZXhlcyBuZXR3b3Jr" class="sphinx-tabs-panel group-tab" hidden="true" id="panel-0-TW9sZWN1bGFyIGNvbXBsZXhlcyBuZXR3b3Jr" name="TW9sZWN1bGFyIGNvbXBsZXhlcyBuZXR3b3Jr" role="tabpanel" tabindex="0"><div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>odamnet<span class="w"> </span>networkDownloading<span class="w">  </span>--netUUID<span class="w"> </span>419ae651-cf05-11ed-a79c-005056ae23aa<span class="w"> </span><span class="se">\</span>
<span class="w">                            </span>--networkFile<span class="w"> </span>useCases/InputData/multiplex/1/Complexes_gene_names_190123.gr<span class="w"> </span><span class="se">\</span>
<span class="w">                            </span>--simple<span class="w"> </span>True
</pre></div>
</div>
</div><div aria-labelledby="tab-0-UmVhY3RvbWUgcGF0aHdheXMgbmV0d29yaw==" class="sphinx-tabs-panel group-tab" hidden="true" id="panel-0-UmVhY3RvbWUgcGF0aHdheXMgbmV0d29yaw==" name="UmVhY3RvbWUgcGF0aHdheXMgbmV0d29yaw==" role="tabpanel" tabindex="0"><div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>odamnet<span class="w"> </span>networkDownloading<span class="w">  </span>--netUUID<span class="w"> </span>b13e9620-cefd-11ed-a79c-005056ae23aa<span class="w"> </span><span class="se">\</span>
<span class="w">                            </span>--networkFile<span class="w"> </span>useCases/InputData/multiplex/1/Pathways_reactome_gene_names_190123.gr<span class="w"> </span><span class="se">\</span>
<span class="w">                            </span>--simple<span class="w"> </span>True
</pre></div>
</div>
</div></div>
</section>
</section>
<section id="pathways-processes-of-interest-network">
<span id="pathwaysofinterestnet"></span><h2>Pathways/processes of interest network<a class="headerlink" href="#pathways-processes-of-interest-network" title="Permalink to this heading"></a></h2>
<p>In the use-case 1, we are using <strong>data retrieved from databases</strong>. So, we created a rare disease pathways network with
data retrieved from WikiPathways <a class="footnote-reference brackets" href="#id19" id="id7" role="doc-noteref"><span class="fn-bracket">[</span>7<span class="fn-bracket">]</span></a>. The network is composed of 104 nodes and the bipartite contains 4,612 interactions
between genes and rare disease pathways.</p>
<p>In the use-case 2, we are <strong>provided data from a previous study</strong> <a class="footnote-reference brackets" href="#id20" id="id8" role="doc-noteref"><span class="fn-bracket">[</span>8<span class="fn-bracket">]</span></a>. We created a disconnected network with pathways
or processes related to Congenital Anomalies of the Kidney and Urinary Tract (CAKUT) as nodes. The network is composed of
13 nodes and the bipartite network contains 1,655 interactions between genes and pathways and processes related to
CAKUT.</p>
<p>To know how to create these two networks, see the <a class="reference internal" href="NetworkCreation.html"><span class="doc">Networks creation</span></a> page.</p>
</section>
<section id="disease-disease-similarity-network">
<span id="similaritynet"></span><h2>Disease-disease similarity network<a class="headerlink" href="#disease-disease-similarity-network" title="Permalink to this heading"></a></h2>
<section id="disease-disease-similarity-network-creation">
<h3>Disease-disease similarity network creation<a class="headerlink" href="#disease-disease-similarity-network-creation" title="Permalink to this heading"></a></h3>
<div class="line-block">
<div class="line"><em>Data were downloaded on June 11th, 2022</em></div>
<div class="line"><a class="reference external" href="https://hpo.jax.org/app/data/annotation"><em>Annotations</em></a> <em>and</em> <a class="reference external" href="https://hpo.jax.org/app/data/ontology"><em>ontologies</em></a> <em>are coming from HPO</em></div>
</div>
<p>We constructed a disease-disease network based on the phenotype similarity between diseases. A disease is defined as
a set of phenotypes and each phenotype is associated to the Human Ontology Project IDs (HPO) <a class="footnote-reference brackets" href="#id21" id="id9" role="doc-noteref"><span class="fn-bracket">[</span>9<span class="fn-bracket">]</span></a>.</p>
<p>The similarity score is calculated based on the number of shared phenotypes between two diseases (<a class="footnote-reference brackets" href="#id22" id="id10" role="doc-noteref"><span class="fn-bracket">[</span>10<span class="fn-bracket">]</span></a>, <a class="footnote-reference brackets" href="#id23" id="id11" role="doc-noteref"><span class="fn-bracket">[</span>11<span class="fn-bracket">]</span></a>, <a class="footnote-reference brackets" href="#id24" id="id12" role="doc-noteref"><span class="fn-bracket">[</span>12<span class="fn-bracket">]</span></a>).
Every pairs of diseases will have a similarity score. For each disease we selected the 5 most similar diseases to
create the network.</p>
<p>The <strong>disease-disease</strong> similarity network contains 33,925 edges and 8,264 diseases.</p>
</section>
<section id="gene-disease-bipartite">
<h3>Gene-disease bipartite<a class="headerlink" href="#gene-disease-bipartite" title="Permalink to this heading"></a></h3>
<div class="line-block">
<div class="line"><em>Data were downloaded on September 27th, 2022</em></div>
<div class="line"><a class="reference external" href="https://hpo.jax.org/app/data/annotation"><em>Associations</em></a> <em>between genes and diseases file is coming from HPO</em></div>
</div>
<p>The <strong>molecular multilayer network</strong> is connected to the <strong>disease-disease similarity network</strong> with the
<strong>gene-disease bipartite</strong>. The bipartite contains 6,564 associations (4,483 genes and 5,878 diseases).</p>
</section>
</section>
<section id="references">
<h2>References<a class="headerlink" href="#references" title="Permalink to this heading"></a></h2>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id13" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id1">1</a><span class="fn-bracket">]</span></span>
<p>Pratt D, Chen J, Welker <em>et al.</em>. NDEx, the Network Data Exchange. Cell Systems. 2015.</p>
</aside>
<aside class="footnote brackets" id="id14" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id2">2</a><span class="fn-bracket">]</span></span>
<p>Luck K, Kim DK, Lambourne L <em>et al.</em>. A reference map of the human binary protein interactome. Nature. 2020.</p>
</aside>
<aside class="footnote brackets" id="id15" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id3">3</a><span class="fn-bracket">]</span></span>
<p>Alonso-López D, Campos-Laborie FJ, Gutiérrez MA <em>et al.</em>. APID database: redefining protein-protein interaction experimental evidences and binary interactomes. Database (Oxford). 2019.</p>
</aside>
<aside class="footnote brackets" id="id16" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id4">4</a><span class="fn-bracket">]</span></span>
<p>Drew K, Wallingford JB, Marcotte EM. hu.MAP 2.0: integration of over 15,000 proteomic experiments builds a global compendium of human multiprotein assemblies. Molecular Systeme Biology. 2021.</p>
</aside>
<aside class="footnote brackets" id="id17" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id5">5</a><span class="fn-bracket">]</span></span>
<p>Giurgiu M, Reinhard J, Brauner B <em>et al.</em>. CORUM: the comprehensive resource of mammalian protein complexes-2019. Nucleic acids research. 2019.</p>
</aside>
<aside class="footnote brackets" id="id18" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id6">6</a><span class="fn-bracket">]</span></span>
<p>Gillespie M, Jassal B, Stephan R <em>et al.</em>. The reactome pathway knowledgebase 2022. Nucleic acids research. 2022.</p>
</aside>
<aside class="footnote brackets" id="id19" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id7">7</a><span class="fn-bracket">]</span></span>
<p>Martens M, Ammar A, Riutta A <em>et al.</em>. WikiPathways: connecting communities. Nucleic acids research. 2021.</p>
</aside>
<aside class="footnote brackets" id="id20" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id8">8</a><span class="fn-bracket">]</span></span>
<p>Ozisik O, Ehrhart F, Evelo C <em>et al.</em>. Overlap of vitamin A and vitamin D target genes with CAKUT-related processes. F1000Research. 2021.</p>
</aside>
<aside class="footnote brackets" id="id21" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id9">9</a><span class="fn-bracket">]</span></span>
<p>Köhler S, Gargano M, Matentzoglu N <em>et al.</em>. The Human Phenotype Ontology in 2021, Nucleic acids research. 2021.</p>
</aside>
<aside class="footnote brackets" id="id22" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id10">10</a><span class="fn-bracket">]</span></span>
<p>Westbury SK, Turro E, Greene D <em>et al.</em>. Human phenotype ontology annotation and cluster analysis to unravel genetic defects in 707 cases with unexplained bleeding and platelet disorders. Genome Medicine. 2015.</p>
</aside>
<aside class="footnote brackets" id="id23" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id11">11</a><span class="fn-bracket">]</span></span>
<p>Valdeolivas A, Tichit L, Navarro C <em>et al.</em>. Random walk with restart on multiplex and heterogeneous biological networks. Bioinformatics. 2019.</p>
</aside>
<aside class="footnote brackets" id="id24" role="note">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id12">12</a><span class="fn-bracket">]</span></span>
<p>Baptista A, Gonzalez A &amp; Baudot A. Universal multilayer network exploration by random walk with restart. Communications Physics. 2022.</p>
</aside>
</aside>
</section>
</section>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="NetworkCreation.html" class="btn btn-neutral float-left" title="Networks creation" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
        <a href="NetworkVisualisation.html" class="btn btn-neutral float-right" title="Networks visualisation" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright 2022, Térézol.</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>