

<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
  <meta charset="utf-8" />
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  
  <title>Methods overview &mdash; EnvironmentProject 1 documentation</title>
  

  
  <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/custom.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/tabs.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/custom.css" type="text/css" />

  
  

  
  

  

  
  <!--[if lt IE 9]>
    <script src="../../_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="../../" src="../../_static/documentation_options.js"></script>
        <script data-url_root="../../" id="documentation_options" src="../../_static/documentation_options.js"></script>
        <script src="../../_static/jquery.js"></script>
        <script src="../../_static/underscore.js"></script>
        <script src="../../_static/doctools.js"></script>
        <script src="../../_static/tabs.js"></script>
    
    <script type="text/javascript" src="../../_static/js/theme.js"></script>

    
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="Overlap analysis" href="methods_overlap.html" />
    <link rel="prev" title="Installation" href="../Installation.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="../../index.html" class="icon icon-home"> EnvironmentProject
          

          
          </a>

          
            
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        
        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption" role="heading"><span class="caption-text">Quickstart</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../Installation.html">Installation</a></li>
<li class="toctree-l1"><a class="reference internal" href="../Installation.html#requierements">Requierements</a></li>
<li class="toctree-l1"><a class="reference internal" href="../Installation.html#usage">Usage</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Methods</span></p>
<ul class="current">
<li class="toctree-l1 current"><a class="current reference internal" href="#">Methods overview</a></li>
<li class="toctree-l1"><a class="reference internal" href="methods_overlap.html">Overlap analysis</a></li>
<li class="toctree-l1"><a class="reference internal" href="methods_AMI.html">Active Module Identification</a></li>
<li class="toctree-l1"><a class="reference internal" href="methods_RWR.html">Random Walk with Restart</a></li>
<li class="toctree-l1"><a class="reference internal" href="methods_newNetwork.html">Network creation</a></li>
<li class="toctree-l1"><a class="reference internal" href="Cytoscape.html">Visualisation using Cytoscape</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Analysis</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../analysis/example1.html">Example 1 : data source from requests</a></li>
<li class="toctree-l1"><a class="reference internal" href="../analysis/example2.html">Example 2 : data source from database files</a></li>
<li class="toctree-l1"><a class="reference internal" href="../analysis/example3.html">Example 3 : custom data source</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">File formats</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../formats/Input.html">Input files</a></li>
<li class="toctree-l1"><a class="reference internal" href="../formats/Output.html">Output files</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Functions description</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../functions/fcts_arguments.html">Input arguments for each method</a></li>
<li class="toctree-l1"><a class="reference internal" href="../functions/fcts_description.html">Functions</a></li>
</ul>

            
          
        </div>
        
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../index.html">EnvironmentProject</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          

















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="../../index.html" class="icon icon-home"></a> &raquo;</li>
        
      <li>Methods overview</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
          
            <a href="../../_sources/pages/methods/methods_overview.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="methods-overview">
<h1>Methods overview<a class="headerlink" href="#methods-overview" title="Permalink to this headline">¶</a></h1>
<p>As part of the EJP RD project, an overlap analysis was perform between a list of genes targeted by vitamin A and pathways
related to CAKUT disease. The main goal of this analysis was to show if vitamin A could effect on CAKUT disease (Ozisik <em>et al.</em>, 2021 <a class="footnote-reference brackets" href="#id12" id="id1">1</a>).</p>
<p>Data information came from several sources : the <a class="reference external" href="https://ctdbase.org/">Comparative Toxicogenomics Databse (CTD)</a> <a class="footnote-reference brackets" href="#id13" id="id2">2</a>
and the study of Balmer and Blomhoff <a class="footnote-reference brackets" href="#id14" id="id3">3</a> for the target genes and WikiPathways (WP) <a class="footnote-reference brackets" href="#id15" id="id4">4</a>, Reactome <a class="footnote-reference brackets" href="#id16" id="id5">5</a> and Gene Ontology (GO) <a class="footnote-reference brackets" href="#id17" id="id6">6</a><sup>,</sup> <a class="footnote-reference brackets" href="#id18" id="id7">7</a>.</p>
<p>With this project, we want to automatise the analysis and extract data information directly from databases. Users can
study the link between environmental factors and rare diseases easily juts by giving a factors list. Other methods can
be perform as random walk analyse or active subnetwork identification in addition to overlap. Moreover, the tool is open
to external sources provided by users.</p>
<div class="section" id="methods">
<h2>Methods<a class="headerlink" href="#methods" title="Permalink to this headline">¶</a></h2>
<p>Three methods are available to study the link between environmental factors and rare diseases (<a class="reference internal" href="#overviewfig"><span class="std std-numref">Fig. 1</span></a>) :</p>
<blockquote>
<div><ul class="simple">
<li><p><a class="reference internal" href="methods_overlap.html#overlap"><span class="std std-ref">Overlap analysis</span></a></p></li>
<li><p><a class="reference internal" href="methods_AMI.html#ami"><span class="std std-ref">Active Module Identification</span></a></p></li>
<li><p><a class="reference internal" href="methods_RWR.html#rwr"><span class="std std-ref">Random Walk with Restart</span></a></p></li>
</ul>
</div></blockquote>
<div class="figure align-center" id="id20">
<span id="overviewfig"></span><img alt="methods overview" src="../../_images/MethodsOverview.png" />
<p class="caption"><span class="caption-number">Fig. 1 </span><span class="caption-text">: Methods implemented overview</span><a class="headerlink" href="#id20" title="Permalink to this image">¶</a></p>
</div>
</div>
<div class="section" id="data-source">
<h2>Data source<a class="headerlink" href="#data-source" title="Permalink to this headline">¶</a></h2>
<p>Throughout this documentation, examples will be presented to illustrate these different sources of information. By default,
genes data are automatically extracted from CTD <a class="footnote-reference brackets" href="#id13" id="id8">2</a> and pathways from WikiPathways <a class="footnote-reference brackets" href="#id15" id="id9">4</a>.</p>
<div class="section" id="target-genes">
<h3>Target genes<a class="headerlink" href="#target-genes" title="Permalink to this headline">¶</a></h3>
<p>Gene data information may come from three different source :</p>
<div class="sphinx-tabs docutils container">
<div aria-label="Tabbed content" class="closeable" role="tablist"><button aria-controls="panel-0-Q1REIHJlcXVlc3Q=" aria-selected="true" class="sphinx-tabs-tab group-tab" id="tab-0-Q1REIHJlcXVlc3Q=" name="Q1REIHJlcXVlc3Q=" role="tab" tabindex="0">CTD request</button><button aria-controls="panel-0-Q1REIGZpbGU=" aria-selected="false" class="sphinx-tabs-tab group-tab" id="tab-0-Q1REIGZpbGU=" name="Q1REIGZpbGU=" role="tab" tabindex="-1">CTD file</button><button aria-controls="panel-0-R2VuZXMgZmlsZQ==" aria-selected="false" class="sphinx-tabs-tab group-tab" id="tab-0-R2VuZXMgZmlsZQ==" name="R2VuZXMgZmlsZQ==" role="tab" tabindex="-1">Genes file</button></div><div aria-labelledby="tab-0-Q1REIHJlcXVlc3Q=" class="sphinx-tabs-panel group-tab" id="panel-0-Q1REIHJlcXVlc3Q=" name="Q1REIHJlcXVlc3Q=" role="tabpanel" tabindex="0"><div class="admonition warning">
<p class="admonition-title">Warning</p>
<p>The CTD database is updated every month (<a class="reference external" href="https://ctdbase.org/about/changes/">Updates page</a>). May provide is own
version of data is interesting for reproducibility.</p>
</div>
<p>Users give a list of <strong>environmental factors</strong>. CTD is requested an gives a list of target genes.</p>
</div><div aria-labelledby="tab-0-Q1REIGZpbGU=" class="sphinx-tabs-panel group-tab" hidden="true" id="panel-0-Q1REIGZpbGU=" name="Q1REIGZpbGU=" role="tabpanel" tabindex="0"><p>Users give a <strong>tsv file</strong> with the data from <strong>CTD</strong> database. Genes list is extracted from this file.</p>
</div><div aria-labelledby="tab-0-R2VuZXMgZmlsZQ==" class="sphinx-tabs-panel group-tab" hidden="true" id="panel-0-R2VuZXMgZmlsZQ==" name="R2VuZXMgZmlsZQ==" role="tabpanel" tabindex="0"><p>Users give a <strong>list of genes</strong> directly</p>
</div></div>
</div>
<div class="section" id="pathways">
<h3>Pathways<a class="headerlink" href="#pathways" title="Permalink to this headline">¶</a></h3>
<p>Pathways can be extracted from two different sources :</p>
<div class="sphinx-tabs docutils container">
<div aria-label="Tabbed content" class="closeable" role="tablist"><button aria-controls="panel-1-V1AgcmVxdWVzdA==" aria-selected="true" class="sphinx-tabs-tab group-tab" id="tab-1-V1AgcmVxdWVzdA==" name="V1AgcmVxdWVzdA==" role="tab" tabindex="0">WP request</button><button aria-controls="panel-1-R01UIGZpbGU=" aria-selected="false" class="sphinx-tabs-tab group-tab" id="tab-1-R01UIGZpbGU=" name="R01UIGZpbGU=" role="tab" tabindex="-1">GMT file</button></div><div aria-labelledby="tab-1-V1AgcmVxdWVzdA==" class="sphinx-tabs-panel group-tab" id="panel-1-V1AgcmVxdWVzdA==" name="V1AgcmVxdWVzdA==" role="tabpanel" tabindex="0"><div class="admonition warning">
<p class="admonition-title">Warning</p>
<p>WP is updated regularly (<a class="reference external" href="https://www.wikipathways.org/index.php/WikiPathways:Updates">Updates page</a>).
May provide is own version of data is interesting for reproducibility.</p>
</div>
<p>By default, <strong>Rare Disease Pathways</strong> are extracted from <strong>WikiPathways</strong> <a class="footnote-reference brackets" href="#id15" id="id11">4</a>. The corresponding background genes are
extracted in the same time (all human genes in WP).</p>
</div><div aria-labelledby="tab-1-R01UIGZpbGU=" class="sphinx-tabs-panel group-tab" hidden="true" id="panel-1-R01UIGZpbGU=" name="R01UIGZpbGU=" role="tabpanel" tabindex="0"><p>Users give a <strong>GMT file</strong> with pathways. It could be rare diseases pathways from WP for a specific version,
or a custom GMT file with <strong>pathways of interest</strong>. Pathways can come from different sources. Corresponding backgrounds
genes are needed.</p>
</div></div>
</div>
<div class="section" id="examples">
<h3>Examples<a class="headerlink" href="#examples" title="Permalink to this headline">¶</a></h3>
<p>We performed an analysis to study the link between vitamin A and rare diseases. We illustrated the different source extraction
through three examples.</p>
<blockquote>
<div><ul class="simple">
<li><p><a class="reference internal" href="../analysis/example1.html#example1"><span class="std std-ref">Example 1 : data source from requests</span></a></p></li>
<li><p><a class="reference internal" href="../analysis/example2.html#example2"><span class="std std-ref">Example 2 : data source from database files</span></a></p></li>
<li><p><a class="reference internal" href="../analysis/example3.html#example3"><span class="std std-ref">Example 3 : custom data source</span></a></p></li>
</ul>
</div></blockquote>
<div class="admonition tip">
<p class="admonition-title">Tip</p>
<p>You can mix input type. For instance, request CTD and give a custom GMT file of pathways of interest.
Every combination is possible !</p>
</div>
</div>
</div>
<div class="section" id="references">
<h2>References<a class="headerlink" href="#references" title="Permalink to this headline">¶</a></h2>
<dl class="footnote brackets">
<dt class="label" id="id12"><span class="brackets"><a class="fn-backref" href="#id1">1</a></span></dt>
<dd><p>Ozisik, O., Ehrhart, F., Evelo, C. T., Mantovani, A., &amp; Baudot, A. (2021). Overlap of vitamin A and vitamin D target genes with CAKUT-related processes. F1000Research, 10.</p>
</dd>
<dt class="label" id="id13"><span class="brackets">2</span><span class="fn-backref">(<a href="#id2">1</a>,<a href="#id8">2</a>)</span></dt>
<dd><p>Davis AP, Grondin CJ, Johnson RJ, Sciaky D, Wiegers J, Wiegers TC, Mattingly CJ The Comparative Toxicogenomics Database: update 2021. Nucleic Acids Res. 2021.</p>
</dd>
<dt class="label" id="id14"><span class="brackets"><a class="fn-backref" href="#id3">3</a></span></dt>
<dd><p>Balmer, J. E., &amp; Blomhoff, R. (2002). Gene expression regulation by retinoic acid. Journal of lipid research, 43(11), 1773-1808.</p>
</dd>
<dt class="label" id="id15"><span class="brackets">4</span><span class="fn-backref">(<a href="#id4">1</a>,<a href="#id9">2</a>,<a href="#id11">3</a>)</span></dt>
<dd><p>Martens, M., Ammar, A., Riutta, A., Waagmeester, A., Slenter, D. N., Hanspers, K., … &amp; Kutmon, M. (2021). WikiPathways: connecting communities. Nucleic acids research, 49(D1), D613-D621.</p>
</dd>
<dt class="label" id="id16"><span class="brackets"><a class="fn-backref" href="#id5">5</a></span></dt>
<dd><p>Jassal, B., Matthews, L., Viteri, G., Gong, C., Lorente, P., Fabregat, A., … &amp; D’Eustachio, P. (2020). The reactome pathway knowledgebase. Nucleic acids research, 48(D1), D498-D503.</p>
</dd>
<dt class="label" id="id17"><span class="brackets"><a class="fn-backref" href="#id6">6</a></span></dt>
<dd><p>Ashburner et al. Gene ontology: tool for the unification of biology. Nat Genet. May 2000;25(1):25-9</p>
</dd>
<dt class="label" id="id18"><span class="brackets"><a class="fn-backref" href="#id7">7</a></span></dt>
<dd><p>The Gene Ontology resource: enriching a GOld mine. Nucleic Acids Res. Jan 2021;49(D1):D325-D334</p>
</dd>
<dt class="label" id="id19"><span class="brackets">8</span></dt>
<dd><p>Curated chemical–gene interactions data were retrieved from the Comparative Toxicogenomics Database (CTD), MDI Biological Laboratory, Salisbury Cove, Maine, and NC State University, Raleigh, North Carolina. World Wide Web (URL: <a class="reference external" href="http://ctdbase.org/">http://ctdbase.org/</a>). [Month, year of data retrieval].</p>
</dd>
</dl>
</div>
</div>


           </div>
           
          </div>
          <footer>
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
        <a href="methods_overlap.html" class="btn btn-neutral float-right" title="Overlap analysis" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
        <a href="../Installation.html" class="btn btn-neutral float-left" title="Installation" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>
        &#169; Copyright 2022, Térézol.

    </p>
  </div>
    
    
    
    Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    
    provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>
        </div>
      </div>

    </section>

  </div>
  

  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>